Influence of genetic variants on gene expression in human pancreatic islets – implications for type 2 diabetesAna Viñuela1,2,3* ; Martijn van de Bunt4,5,6,*,# ; Arushi Varshney7* ; Rashmi B Prasad8,*; Olof Asplund8; Amanda Bennett4; Michael Boehnke9; Andrew A. Brown1,2,3; Michael R Erdos10; João Fadista11,12; Ola Hansson8; Gad Hatem8; Cédric Howald1,2,3; Apoorva K. Iyengar13; Paul Johnson4; Ulrika Krus8; Patrick E. MacDonald14; Anubha Mahajan4; Jocelyn E. Manning Fox14; Narisu Narisu10; Vibe Nylander5; Peter Orchard15; Nikolay Oskolkov8; Nikolaos I. Panousis1,2,3; Anthony Payne4 ; Michael L Stitzel16, 17; Swarooparani Vadlamudi13, Ryan Welch9; Francis S. Collins10; Karen L. Mohlke13; Anna L. Gloyn4,5,6; Laura J. Scott9; Emmanouil T. Dermitzakis1,2,3; Leif Groop12,18; Stephen C.J. Parker7,15; Mark I. McCarthy4,5,61. Department of Genetic Medicine and Development, University of Geneva Medical School, 1211 Geneva, Switzerland.2. Institute for Genetics and Genomics in Geneva (iGE3), University of Geneva, 1211 Geneva, Switzerland. 3. Swiss Institute of Bioinformatics, 1211 Geneva, Switzerland. 4. Wellcome Centre for Human Genetics, Nuffield Department of Medicine, University of Oxford, Oxford, OX3 7BN, UK.5. Oxford Centre for Diabetes, Endocrinology and Metabolism, Radcliffe Department of Medicine, University of Oxford, Oxford, OX3 7LE, UK 6. Oxford NIHR Biomedical Research Centre, Oxford University Hospitals Trust, Oxford, OX3 7LE, UK.7. Department of Human Genetics, University of Michigan, Ann Arbor, MI 48109. 8. Lund University Diabetes Centre, Department of Clinical Sciences, Lund University,Skåne University Hospital, Malmö, Sweden. 9. Department of Biostatistics and Center for Statistical Genetics, University of Michigan, Ann Arbor, MI 48109. 10. Medical Genomics and Metabolic Genetics Branch, National Human Genome Research Institute, National Institutes of Health, Bethesda, Maryland, 20892, USA.  11. Department of Epidemiology Research, Statens Serum Institut, Copenhagen, DK-2300, Denmark. 12. Department of Clinical Sciences, Lund University Diabetes Centre, SE-214 28 Malmö, Sweden. 13. Department of Genetics, University of North Carolina, Chapel Hill, NC 27599. 14. Department of Pharmacology and Alberta Diabetes Institute, University of Alberta, Edmonton, Alberta, Canada. 15. Department of Computational Medicine & Bioinformatics, University of Michigan, Ann Arbor, MI 48109. 16. The Jackson Laboratory for Genomic Medicine, Farmington, CT 06032. 17. Department of Genetics and Genome Sciences, Institute for Systems Genomics, University of Connecticut, Farmington, CT 06032. 18. Finnish Institute for Molecular Medicine (FIMM), University of Helsinki, Helsinki, Finland.# Current address: Department of Bioinformatics & Data Mining, Novo Nordisk A/S, Novo Nordisk Park, Måløv 2760, Denmark.* These authors contributed equally to this work. 
Introduction	 	 	 	Over the past decade, analysis of genome-wide association study (GWAS) data has generated a growing inventory of genomic regions implicated in type 2 diabetes (T2D) predisposition and variation in diabetes-related glycemic traits. However, progress in defining the mechanisms whereby these associated variants mediate their impact on disease-risk has been relatively slow. One major reason is that at least 90\% of the associated signals map to non-coding sequence. This complicates efforts to connect T2D-associated variants with the transcripts and networks through which they exert their effects1-5.For many common, multifactorial diseases, one valuable approach for addressing this “variant-to-function” challenge is to use expression quantitative trait loci (eQTL) mapping to characterize the impact of disease-associated regulatory variants on the expression of nearby genes6. Demonstrating that a disease-risk variant co-localizes with a cis-eQTL signal is consistent with a causal role for the transcript concerned in disease development. Such hypotheses can then be subject to more direct evaluation, for example, by perturbing the gene in suitable cellular or animal models.However, eQTL signals often show marked tissue-specificity. The power to detect mechanistically-informative expression effects is therefore, at least in part, dependent on assaying expression data from sufficient numbers of samples across the range of disease-relevant tissues6. Appropriate interpretation of GWAS-eQTL signal co-localization analyses also needs to consider physiological and genomic data which may point to the specific tissue most likely to be mediating disease-risk at a given locus5,7.The pathogenesis of T2D involves dysfunction across multiple tissues, most obviously pancreatic islets, adipose, muscle, liver, and brain. Risk variants that influence T2D predisposition through processes active in each of these tissues have been reported (e.g. MC4R in brain [Vaisse,et al., J. Cli. Inv, 2000], KLF14 in adipose [Small, et al., Nat Gene, 2018], TBC1D4 in muscle [Moltke, et al., Nature, 2014], ADCY5 in islets [Thurner, et al.,  eLife, 2018], GCKR in liver [Saxena, DGI, et al, Science, 2007]). However, a range of physiological and genomic analyses consistently point to islet dysfunction as having the greatest contribution to T2D risk5,8,9. For example, genome-wide enrichment analyses highlight the particularly strong relationship between T2D-risk variants and regulatory elements active in human islets10-13.Research access to human pancreatic islet material is largely limited to samples accessible from a subset of cadaveric organ donors, and consequent scarcity has compromised efforts to characterize the regulation of human islet expression. The human pancreatic samples examined in GTEx14 are of limited value, since islets constitute only ~1% of total pancreas. Previous studies have demonstrated the potential of islet gene expression information to characterize T2D effector genes such as MTNR1B and ADCY512,15,16, but the sample sizes examined to date have been modest: the largest published human islet RNA-Seq data includes only 118 samples16.We constituted the InsPIRE (Integrated Network for Systematic analysis of Pancreatic Islet RNA Expression) consortium as a vehicle for the aggregation and joint analysis of available human islet RNA-Seq data15-17. Here, we report on analyses of 420 human islet preparations which provide a detailed landscape of the genetic regulation of gene expression in this key tissue, and its relationship to mechanisms of T2D predisposition.	 	 	 	This research addresses a series of questions with relevance beyond the specific example of T2D. When a disease-relevant tissue is missing from reference datasets such as GTEx, what additional value accrues from dedicated expression profiling from that missing tissue? What is the impact of tissue heterogeneity (cellular heterogeneity within the tissue of interest, and contamination with cells that are not of direct interest) on the interpretation of eQTL data? What does the synthesis of tissue specific epigenomic and expression data tell us about the coordination of upstream transcription factor regulators of gene expression? And, finally, what information do tissue-specific eQTL analyses provide about the regulatory mechanisms mediating disease predisposition? Characterization of genetic regulation of gene expression in isletsWe combined pancreatic islet RNA-Seq and dense genome-wide genotype data from 420 individuals. Data from 196 of these individuals have been reported previously12,15-17. We aggregated, and then jointly mapped and reprocessed, all samples (median sequence-depth per sample, ~60M reads) to generate exon- and gene-level quantifications, using principal component methods to correct for technical and batch variation (see Methods and Supp. Figure S1).To characterize the regulation of gene expression for the 17,914 protein coding and long non-coding RNAs (LncRNAs) genes with quantifiable expression in these samples, we performed eQTL analysis (fastQTL18) on both exon and gene-level expression measures, using all 8.05e+06 variants that pass quality control (QC) (see Methods; Supp. Tables ST1 & ST2). This joint analysis of all 420 individuals identified 4,312 genes (eGenes) with significant cis-eQTLs at the gene level (FDR<1%; cis defined as within 1Mb of the transcription start site [TSS]). Results of this joint analysis were highly-correlated with those obtained from a fixed-effects meta-analysis of the four component studies, indicating appropriate control for the technical differences between the studies (Methods; Supp Figure S2). The complementary exon-level analysis, which can capture the impact of variants influencing splicing as well as expression, detected 6,039 eGenes (FDR<1%, Supp. Figure S3) 20,21. Stepwise regression analysis (after conditioning on the lead variant) identified a further 1,702 independent eQTLs (involving 1,291 eGenes), giving a total of 7,741 islet exon-level eQTLs (Supp. Tables ST1 & ST2). At the 1,291 eGenes with at least two independent exon-eSNPs, although primary eSNPs (that is, the most significant signals for each eGene) tended to be localized closer to the canonical TSS than secondary eSNPs (Wilcoxon test P=6.3x10-30), there were 503 (39.0%) of these genes for which the second eSNP identified during stepwise conditional analysis was more proximal to the TSS (Supp. Figure S3).Tissue specific regulatory variation in islets	For many complex traits of biomedical interest, the cell types considered most relevant to disease development are either entirely absent from large-scale eQTL datasets or represent a minor component of the cellular content of assayed tissues. The value of targeting the specific cell-types of interest for dedicated eQTL discovery -- as opposed to relying on existing eQTL data from more accessible tissues – remains unclear. To examine this, we considered the degree to which the set of 6,039 exon-level islet eGenes overlapped with eQTLs from 44 tissues (for which sample size >70) in the version 6p release of GTEx14. To allow direct comparison with the InsPIRE data, we reprocessed GTEx sequence data to generate exon-level eQTLs (Methods). Approximately 5% (337) of the 6,039 islet eGenes had no significant eQTLs (in exon- or gene-level analyses) in any of the 44 GTEx tissues suggesting the islets eQTL were strong enough to be detected with 420 samples, but maybe not active or strong enough to be significant in other tissues with the current sample size of GTEx (Supp. Table ST3). Therefore, and rather than defining “tissue-shared” effects based on arbitrary thresholds, we estimated the proportion of islet eQTLs active in other GTEx tissues using P-value enrichment analysis (π119): the proportion of islet-eQTLs shared with other GTEx tissues ranged from 40% (brain) to 73% (adipose). As previously reported14, there was a positive linear relationship between these π1 measures and sample sizes for the respective tissues in GTEx (Figure 1A). However, π1 estimates for shared eQTL effects reached only 65% and 57% (respectively) for skeletal muscle (n=361), and whole blood (n=338), the tissues with the largest representation in this version of GTEx. These data demonstrate that there is a substantial component of tissue-specific genetic regulation that could, at these sample sizes, only be detected in islets, illustrating the value of extending current expression profiling efforts to additional tissues and cell-types of particular biomedical importance. Cellular heterogeneity	 	 	 	The human islets analyzed in this, and other, studies include a mixture of cell types, including the hormone-producing ?, ? and ?-cells, and, given the limitations of physical islet isolation, a variable amount of adherent exocrine material. From the perspective of T2D pathogenesis, the transcriptomes of the former (particularly ? and ?-cells) are of most interest. However, the eQTLs identified could have their origins from any of the cellular components. We used a number of approaches to address interpretative challenges resulting from this cellular heterogeneity.First, we performed tissue deconvolution analysis to estimate the proportion of exocrine contamination across the 420 InsPIRE islet samples: these analyses were performed prior to the principal component adjustment used to generate the main eQTL results. We used reference expression signatures for: (a) exocrine tissue (GTEx pancreas data)14; (b) beta-cell; and (c) non-beta cells (the last two from a set of 26 human islet preparations which had been FAC-sorted using the zinc-binding dye Newport Green to separate the beta-cell fraction17). Estimates of the proportion of exocrine pancreas contamination ranged from 1.8% to 91.8% (median 33.5%). These measures of exocrine contamination were significantly correlated (r=0.50, P=2.8x10-15) with independent estimates of exocrine content obtained at the time of islet collection by dithizone staining of the preparations (available for 232 samples) (Supp. Figure S4). Within the endocrine fraction of the islet preparations, median estimates of beta-cell (58.8%, IQR 43.2-66.9%) and non-beta-cell (41.2%, 33.1-56.8%) fractions correspond well to estimates obtained through morphometric assessment22. In 34 samples from donors annotated as having T2D, median estimates of beta-cell composition were lower than those from donors annotated as non-diabetic (n=330) (median 31.8% vs. 35.6%, P=4.5x10-4, Supp. Figure S4). This analysis provides independent confirmation, based on transcriptomic signatures, of evidence from morphometric and physiological studies that the functional mass of beta-cells is reduced in T2D [Meier & Bonadonna, 2013; Butler et al, 2013].Second, we investigated the proportion of eQTL signals from InsPIRE islet RNA profiles also active in GTEx tissues (see above) and confirmed that whole pancreas, often naively-used as a surrogate for the T2D-relevant islet component, represents an imperfect proxy for islet (π1=0.67 with our human islet data). The extent of this eQTL overlap depends on study sample sizes and with GTEx v6p, the whole pancreas overlap is on a par with other tissues such as photo protected skin (π1=0.67) and spleen (π1=0.61) This suggests pancreas and other tissues are equally useful to infer genetic regulatory effects on expression, with the expectation that larger studies will reduce the overlap across tissues while increasing the detection of tissue specific regulatory effects.  The principal component adjustments we used to control for unwanted technical variation during eQTL analysis were designed to account for some of the impact of variation in sample purity. However, by correlating the data-generated PCs with cell proportion estimates, we observed that, even when adjusting using 25 PCs, only ~30% of the variance attributable to variation in exocrine or beta-cell composition was regressed out, requiring more than 107 PCs to remove 50% of the variance. This suggests that some of the eQTLs here attributed to pancreatic islets may, in fact, reflect exocrine pancreatic contamination. To evaluate this further, we compared the sets of eQTLs identified in the InsPIRE islet samples with the highest and lowest proportions of exocrine contamination (n=100 for each) and 100 randomly-selected GTEx whole pancreas samples. Overlap between whole pancreas and islet eQTLs (using π119) was greater in islet samples with the highest exocrine contamination (75% vs 64%) (Supp. Figure S5). Although shared regulatory processes between acinar and islet tissue are to be expected23, these data suggest that apparent overlap in regulatory signals between islet and whole pancreas may partly reflect the consequence of the inadvertent exocrine contamination of islet data.  	 	Of the 420 InsPIRE samples, beta-cell enriched transcriptomes were available for 26 following FAC-sorting. These data allowed us to look for evidence of cell-type-specific eQTL effects and attempt to identify the cellular source of the eQTLs detected in the islet material. With this limited sample size, the only eQTL reaching significance (and then only, at a less stringent threshold of FDR<5%, Supp. Table ST4) was at ADORA2B (P=3.8x10-10, beta = -1.207): this signal was also detected in the InsPIRE islets (P=3.9x10-51, beta = -0.656) and in GTEx pancreas (P=1.6x10-16 , beta = -0.737) (Supp. Figure S6; Supp. Table ST5). Of the 7,741 independent SNP-exon pairs significant in islets, 227 were also significant in beta-cells at FDR<1% (Supp. Table ST6). By comparing the p-value distributions of the eQTLs in islets vs beta-cells19, we estimate that 46% of islet-eQTLs are active in beta-cells (Figure 1B).To identify specific genes with cell-type-specific regulatory effects, we tested for interactions between genotype and the beta-cell or exocrine cellular fraction estimates, controlling for technical variables (Methods). We identified 18 islet cis-eQTLs with a “genotype-by-beta-cell proportion” interaction and 8 with a “genotype-by-exocrine cell proportion” interaction (FDR<1%, Supp. Tables ST7, ST8 & ST9). The former group included ADCY5, a member of the adenylate cyclase family implicated as a T2D GWAS effector transcript by several islet-eQTL studies12,13,24, and CCL2 (also known as MCP-1), a cytokine implicated in type 1 diabetes (T1D) development25. We conclude that a substantial proportion of the regulation of gene expression detected in pancreatic islets is derived from cell-specific effects. Ongoing efforts to develop a single-cell view of islet transcriptional signatures should help to inform these analyses, although the limited sample size of current single-cell transcriptomic studies [Wang and Kaestner 2018] and their lack of genotype information means they offer little direct insight into the relationship between genetic variation and cell-type-specific transcript abundance. Functional properties of islet genetic regulatory signals. 	Using previously-published islet chromatin states derived from histone modification data12, we observed a significant enrichment of islet eSNPs in active islet chromatin states including active TSS (fold enrichment = 3.84, P = 5.5x10-206), active enhancers (fold enrichment > 1.73, P < 4.8x10-04 between two enhancer states) and stretch enhancers (fold enrichment = 1.57, P= 2.7x10-13), with concomitant depletion of eSNPs in repressed and quiescent islet chromatin states (fold enrichment < 0.66)(Supp. Table ST10; Supp. Figure S7). This recapitulates the enrichment observed for T2D GWAS signals within active islet chromatin (Supp. Figure S8) 10-13.To explore further the chromatin context of islet eSNPs, we first asked whether eSNP effect sizes (measured as the slope of the linear regression) were uniform across the different underlying chromatin contexts in which they occur. We found a non-uniform range of distributions (Figure 2A): for example, eSNPs that overlap active TSS chromatin states had significantly larger effects than those that overlap repressed or weak-repressed polycomb chromatin states (Wilcoxon Rank Sum Test P=0.039). Because chromatin states represent integrated histone mark patterns, and transcription factors (TFs) are more likely to bind in open accessible DNA, we next focused on regions of accessible chromatin within each of the chromatin states, using previously-published ATAC-seq (assay for transposase accessible chromatin followed by sequencing) data from human islets12. As expected, a disproportionately high proportion (80%) of islet eQTLs (based on the lead eSNP itself or proxy SNPs with LD r2>0.99) overlap islet ATAC-seq peaks in islet-active TSS chromatin states. More specifically, of the 646 islet eSNPs that overlap islet active TSS chromatin, 522 (80.8%) occur in the (ATAC-defined) open chromatin portion of that chromatin state (Supp. Figure S9). We note that 49.7% of the islet active TSS chromatin state territory is occupied by islet ATAC-seq broad-peaks.When we examined the distribution of absolute effect sizes for eSNPs that occur within islet ATAC-seq peaks within active TSS, islet stretch enhancers, which were defined as enhancer chromatin state segments longer than 3kb and were show to be islet-specific [CITE: parker PNAS 2013] and typical enhancer (enhancer chromatin states smaller than median size of 800bp) annotations, we found that eSNPs in stretch enhancers had significantly lower effect sizes than those in either typical enhancer chromatin (Wilcoxon Rank Sum Test P=0.0088) or active TSS chromatin states (P=0.0099) (Figure 2B). One important corollary of this observation – that eSNPs in different chromatin contexts have different regulatory effect sizes – is that eSNPs in cell-specific stretch enhancers may be less detectable as eQTLs, and that higher sample sizes are needed to ensure better powered discovery of eQTLs within these critical cell identity regions.We next sought to use the combination of islet chromatin state annotation and eQTL data to identify TFs driving islet regulatory networks. For these analyses, we used published TF footprint (in vivo predicted occupied TF motif binding sites) results generated from human islet ATAC-seq data12. We previously reported enrichment of selected TF footprint motifs at islet eSNPs, using a smaller islet expression dataset12: here, the larger eSNP catalog allowed us to determine how eSNP effect size and target gene expression directionality is associated with base-specific TF binding preferences. We partitioned eSNPs into two equally-sized bins representing those with lower (absolute beta from regression <0.254) and higher (≥0.254) effect sizes. Higher-effect size eSNPs were preferentially enriched (<1% FDR) for a subset of footprint motifs, characteristic of islet-relevant TF families including KLF11 (motif=KLF13_1, P=5.3x10-6) and GLIS3 (motif GLIS3_1, P=5.2x10-6). Other sets of footprint motifs, including those related to the RFX and ETS families of TFs, were significantly enriched for low effect size eSNPs (P<2x10-4) (Figure 2C, Supp. Table 11). Collectively, these results further demonstrate a relationship between local chromatin environment at the level of a TF footprint motif and eSNP effect size. 	 	Since TFs can act as activators, repressors, or both 26, we asked if eSNP alleles that match the base preference at TF footprint motifs have a consistent directional impact on gene expression. We defined a motif directionality fraction score for each TF footprint motif by calculating the fraction of overlapping eSNP where the preferred base in the motif was associated with increased expression of the eGene (Methods ‘TF motif directionality’). Directionality fractions indicate if the TF motifs are activating (fraction near 1), repressive (fraction near 0), or show no preference (fraction near 0.5). We found that the motif activity measures generated with this islet eQTL and ATAC-seq footprint-based metric were largely concordant (Spearman’s r=0.64, P=8.1?10-13) with orthogonal motif activity measures derived from massively parallel reporter assays (MPRAs) performed in HepG2 and K562 cell lines27 (Supp. Figure  S10). There were 99 motifs reported as consistently activating or repressive across HepG2 and K562 cell lines present in our study: for these, we tested whether the motif directionality fraction deviated from null expectation (no preference for activator/repressor) using a binomial test. We found that only 8% (n=8) of the motifs showed evidence of skewed activator preference (<10% FDR; Figure 2D, Supp Table ST12). The activator motifs we identified include many ETS family members, which have a known preference for transcriptional activation27. Our analyses demonstrate how integrating diverse epigenomic information with rich eQTL data can reveal characteristics of gene regulation and its regulators. While contrasting tissue-specific stretch enhancers with the more ubiquitous TSS states in the context of eQTL effect sizes delineated the role of underlying chromatin on function; integrating eQTL information with ATAC-seq and high-resolution TF footprinting revealed in vivo activities of these upstream regulators.  Islet eQTLs are enriched among T2D and glycemic GWAS variantsDiverse lines of evidence emphasize the contribution of reduced pancreatic islet function to the development of T2D, and there is evidence, based on patterns of association across diabetes-related quantitative traits, that many T2D GWAS loci act primarily through their impact on insulin secretion5,7,8,13. To examine the relationships between T2D predisposition alleles and the tissue-specific regulation of gene expression, we combined the human islet eQTL data with equivalent exon-level information for 44 tissues available through GTEx (version 6p)14.We examined 122 GWAS lead variants with genome-wide significant associations to T2D (focusing on 78 signals with the most pronounced effects on T2D risk as detected in 3 or 44 continuous glycemic traits relevant to T2D predisposition (including fasting glucose, and beta-cell function (HOMA-B) in non-diabetic individuals) (Supp. Table ST13)28-30. For each of these GWAS lead variants, we extracted the lead eSNP from the 44 GTEx tissues and the InsPIRE pancreatic islets. To determine the extent to which the lead T2D GWAS variant showed tissue-specific enrichment for islet eQTL associations, we compared these observed effect size estimates from the eQTLs to those derived from a null distribution of 15,000 random eSNPs, matched to the GWAS SNPs with respect to the number of SNPs in LD, distance to TSS, number of nearby genes and minor allele frequency (see METHODS). We were particularly focused on the enrichment in eQTL effect sizes at T2D/glycemic GWAS-associated variants for six tissues implicated in T2D pathogenesis (subcutaneous adipose tissue, skeletal muscle, liver, hypothalamus, islets and whole pancreas), plus whole blood for comparison (Figure 3A, Supp. Figure S11, Supp. Table ST14). We included a set of 55 lead variants implicated by GWAS in predisposition to type 1 diabetes (T1D) for comparison31.Across the 45 tissues, we detected significant enrichment for islet eQTLs amongst variants associated with continuous glycemic traits (normalize enrichment score (NES)=1.27; P=3.6x10-3). Apart from a modest signal in ovary (NES=1.13, P=0.02), there was no enrichment in any other GTEx tissue. The enrichment for islet eQTLs for the full set of 78 T2D variants was directionally consistent with the results for continuous glycemic traits but did not reach nominal significance (NES=1.10; P=0.07). However, T2D GWAS signals influence disease risk through physiological effects in multiple tissues. In the subset (n=17) of the 78 T2D GWAS signals with the strongest evidence (from patterns of association to other T2D-related traits) of mediation through reduced insulin secretion (implicating islet dysfunction)7-9, we observed more marked enrichment of islet eQTL signals (NES=1.27; p=0.025). For this subset there was no enrichment for eQTL effect sizes in whole pancreas (NES=0.90, P=0.88). There was no enrichment of islet eQTL effects for the set of T1D association signals, consistent with the consensus that most genetic risk for T1D is mediated through immune mechanisms31.In the subset of 8 T2D GWAS signals with the strongest evidence of mediation through defects in insulin action (n=8), enrichment was seen in insulin target tissues such as liver (NES=1.10; P=0.03), adipose tissue (NES=1.12; P=0.04) and brain cortex (NES=1.10; P=0.03), but not in islets (NES=1.07, P=0.17). Similar patterns of eQTL enrichment were seen for a broader, partly-overlapping, set of 53 lead variants influencing insulin sensitivity derived from a multivariate GWAS32. These data reveal tissue-specific patterns of genetic regulatory impact for variants at T2D- and glycemic-trait loci which mirror the mechanistic inferences generated by physiological analysis of those signals. They also highlight the importance of matching the tissue origin of the transcriptomic data used for mechanistic inference, to the tissue-specific impact of each GWAS signal on disease predisposition.Identifying effector transcripts for T2D and glycemic traitsThis evidence of generalized overlap between islet eQTLs and (selected) T2D and/or glycemic GWAS signals motivates further efforts to characterize these relationships at individual loci. Previous studies have identified GWAS signals displaying apparent overlap between islet eQTLs and the T2D/glycemic GWAS signals12,15,16, but not all of these signals have been evaluated with respect to the statistical evidence for co-localization (i.e. testing whether the eQTL and the GWAS signals are likely to emanate from the same causal variants), and not all coincident signals have replicated despite ostensibly similar designs and power.There are multiple methods for evaluating the evidence for co-localization: these make different assumptions and often lead to discrepant results33. In this analysis, we focused on the co-localization evidence provided by two complementary algorithms: COLOC, which assesses the differences in regression coefficients of variants associated to two traits, and RTC (Regulatory Trait Concordance), which assesses the differences in ranking of SNPs associated to one trait after conditioning on the most associated SNP for the other trait34,35. We detected evidence for co-localization (with either method) of islet eQTLs at 23 GWAS loci, comprising 24 independent signals (the DGKB hosts two signals), 16 of which reflect T2D associations and 8 glycemic traits (Supp. Table ST15). Evidence for co-localization was most compelling for 11 loci (12 signals) at which both RTC and COLOC provided strong support: including extending confirmation of previously observed co-localizations at ADCY5, TCF7L2, HMG20A, IGF2BP2 and DGKB13,36.At other loci, we observe islet cis-eQTL co-localization for the first time. For example, rs7903146, the lead variant at the T2D-risk signal at TCF7L2, co-localizes with islet expression of TCF7L2 (P=1.9x10-7) (Supp. Figure S12), with the T2D-risk allele increasing TCF7L2 expression (eQTL beta = 0.218). The same eQTL signal was also detected in the smaller beta-cell-specific eQTL analysis (n=26; eQTL beta= 0.724; P=1.0x10-3). Previous efforts to characterize the mechanism of action at this signal have demonstrated that the fine-mapped T2D-risk allele at rs7903146 influences chromatin accessibility and enhancer activity in islets37, but evidence linking these events to TCF7L2 expression has been missing. Indeed, recent studies have proposed other nearby genes as possible effectors transcripts, such as ACSL5ref38: however, we found no evidence in any tissue (from GTEx or InsPIRE) to indicate that rs7903146 influences ACSL5 expression. The association between rs7903146 and TCF7L2 expression was restricted to islets, consistent with evidence that non-diabetic carriers of the TCF7L2 risk-allele display markedly reduced insulin secretion [Zhou, et al, HMG, 2014].Several previously-reported co-localizing signals were not observed in our exon-eQTL based analysis. MTNR1B has shown consistent islet cis-eQTL signals across multiple previous studies16 [Tuomi, et al, Cell Metabolism, 2016], but was excluded from our exon-level analysis due to low exonic-read coverage. However, in gene-level expression analyses, we once again observed strong evidence of co-localization between the lead T2D GWAS variant (rs10830963) and MTNR1B expression (P=5.3x10-21; Supp. Table ST2). At ZMIZ1, the previously-reported cis-eQTL was nominally significant (rs185040218; P=3.0x10-5) but this particular signal did not reach the 1% FDR threshold for inclusion in co-localization testing. At other loci, complex, but divergent, patterns of association between the eQTL and T2D GWAS signals (likely reflecting the impact of multiple enhancers active in different tissues to the T2D signal) challenged the assumptions of these co-localization methods. At the ZBED3 locus for example, the association plots highlight two distinct T2D signals (~500kb apart), and two islet eQTL signals for the PDE8B gene, but only the signal at rs7708285 appears coincident (Supp. Figure S13). COLOC detects this as co-localization, but this configuration cannot easily be tested using RTC as it restricts analysis to variants that lie between a single pair of recombination hotspots. Finally, we attempted to further characterize eGenes that overlapped signals from T2D and glycemic trait GWAS studies by assessing the impact of acute changes in glycemic status on their expression in islets. We used data from a recent analysis of human islets obtained from a set of T2D, and non-diabetic donors and focused on transcripts that showed acute changes in expression when exposed to altered glucose levels in culture (that is, islets from diabetic individuals cultured at normal glucose, and islets from non-diabetic subjects cultured in high glucose)39 (Supp. Table ST16). This revealed multiple islet eGenes, including STARD10, WARS, SIX3, NKX6-3 and KLHL42 (Supp. Table ST16) which may be of particular interest given that their expression in islets is regulated both by T2D-associated variation and by acute changes in glucose exposure.Experimental validation at DGKB	 	 The DGKB locus features two independent co-localizing signals: at both of these, the T2D-risk allele is associated with increased islet expression of DGKB (Figure 3A). At the 5’ signal, the lead SNP for both the T2D GWAS and the islet cis-eQTL is rs17168486. At the 3’ signal, the lead eSNP, rs10231021 (Figure 3B), is in high LD (R2=1, D’=1) with the lead GWAS variant rs10231021 (Figure 3C).The pattern of GWAS association signals for diabetes-related traits for both signals is consistent with a primary impact on insulin secretion (implying islet dysfunction)5,7.  We prioritized, for functional analysis, variants that were in high LD (r2>0.8) with the lead SNPs and located in islet ATAC-seq peaks (Figure 3D). At the 3’ signal, seven variants met these criteria: three (rs7798124, rs7798360 and rs7781710, Figure 3D, “element 1”) overlap an ATAC-seq peak shared across islets, skeletal muscle and the lymphoblastoid cell line GM12878 [Buenrostro, Nat Meth, 2013] cell-line, and four others (rs10228796, rs10258074, rs2191348 and rs2191349, Figure 3D “element 2”) lie in a smaller but more islet-specific ATAC-seq peak. We cloned these putative regulatory elements (Figure 3D) into luciferase reporter constructs and performed transcriptional reporter assays in three insulin-34secreting beta-cell models, including human EndoC-?H1, rat INS1-derived 823/13 and mouse MIN6. Element 1 demonstrated consistent enhancer activity across all three beta-cell lines but did not show allelic differences consistent with the eQTL direction of effect (Supp Figure S14). Element 2, when in forward orientation with respect to DGKB, showed reduced luciferase expression in all three beta-cell lines compared to control. The T2D-risk haplotype showed significantly higher expression than the non-risk haplotype in 832/13 (P = 1.9?10-4) and MIN6 cell-lines (P = 1.1?10-6): equivalent experiments in EndoC-?H1 showed a consistent trend, which did not reach significance (Figure 3E). Luciferase assays using element 2 in reverse orientation also showed consistent trends across all three cell lines, reaching significance in 832/13 alone (Supp Figure S14). These data suggest that T2D risk alleles alleviate regulatory element repression and are directionally consistent with the 3’ DGKB eQTL (Figure 3F). In electrophoretic mobility assays using MIN6 nuclear extract, three of the four “element 2” variants (rs10228796, rs2191348, and rs2191349) showed allele-specific binding (Figure 3G), supporting a functional regulatory role. At the 5’ eQTL, we focused attention on rs17168486, which was both the lead cis-expression and GWAS SNP at the 5’ eQTL, and is located in an islet ATAC-seq peak (Supp Figure S14). We cloned an element including this variant into luciferase reporter constructs but observed no consistent allelic effects on transcriptional activity (Supp Figure S14). DiscussionIn this manuscript, we used transcriptome sequencing in 420 human islet preparations to address issues that are of general relevance to the mechanistic interpretation of non-coding association signals detected by GWAS, in addition to their specific importance for T2D. We documented the degree to which RNA-sequencing of a disease-relevant tissue missing from a reference data set (e.g. GTEx) provides access to a more complete survey of eQTLs active in islets. We used this information to extend the number of GWAS signals for T2D and related glycemic traits that have been shown to co-localise with islet eQTLs, providing clues to potential effector transcripts at several of these loci.  We have demonstrated how tissue heterogeneity (cellular heterogeneity within the tissue of interest, and contamination with cells that are not of direct relevance) can complicate the interpretation of eQTLs co-localizing with GWAS signals. We also integrated our eQTL catalogue with islet epigenomic data to reveal effect size heterogeneity based on local chromatin context and to infer in vivo TF directional activities. Finally, we used our results to nominate and experimentally test causal SNPs at the DGKB locus, which displays coordinated regulatory effects at two statistically independent T2D GWAS signals.Several lines of evidence – including analysis of the physiological association patterns of T2D-associated alleles, and genome-wide enrichment analyses – indicate that many, though by no means all, established T2D association signals act through the islet [Wood et al, Diabetes, 2017; Vaisse,et al., J. Cli. Inv, 2000; Small, et al., Nat Gene, 2018; Moltke, et al.,  Nature, 2014; Saxena, DGI, et al, Science, 2007. One of the major motivations behind this study was to bring an enhanced islet eQTL analysis to bear on the challenge of delivering robust mechanistic inference to non-coding GWAS signals, with particular emphasis on the identity of the effector transcripts that may mediate the downstream consequences of the associated variants. At DGKB, evidence that both the T2D signals co-localize with eQTLs with directionally-consistent impacts on DGKB expression in islets lends support to a causal role for DGKB in T2D predisposition.However, it is important to emphasize some of the complexities of accurate inference from the coincidence of eQTLs and GWAS signals. First, the RNA-seq data from which these analyses are derived from human islets maintained in culture in basal glycemic conditions. eQTL signals that are restricted to a subset of the cells within those islets would have been hard to detect, and the same would be true for genes whose expression is dependent on stimulation. Genes that mediate T2D-risk through an impact on islet development may be under different transcriptional control in adult islets: in some circumstances, this may incriminate co-localizing eGenes that are not directly responsible for the phenotype. Similarly, given that not all T2D loci act through the islet, some of the eQTLs detected may reflect tissue-specific regulation that is not germane to the development of the diabetic phenotype. Reassuringly, for the co-localizing loci we detected, we were able to perform analyses that are generally supportive of the idea that their T2D effects are mediated through islet dysfunction. For example, the islet eQTLs we detected were enriched in the subset of T2D and glycemic loci for which the patterns of GWAS association indicate a primary effect on insulin secretion.   Second, the confident assignment of co-localization can be difficult. There are a diversity of algorithms to measure the evidence that two association signals (here, a trait GWAS and an eQTL signal) are likely to reflect the same causal variants, but agreement between them is not complete. An additional challenge arises from the complex architecture of many GWAS signals that feature multiple overlapping signals that require conditional decomposition before co-localization can be accurately assigned. This is likely to be especially important when the sets of GWAS and cis-eQTL signals at a given locus are not completely overlapping, such that clear co-localization at one of the contributing signals, can be masked by differences in the overall shape of the association signals that confounds simplistic analysis.   Third, recent studies have shown that functionally constrained genes – those that are depleted for missense or loss of function variants – are less likely to have eQTLs, suggesting uniform intolerance of both regulatory and coding variation at the same genes40,41. Complementary studies focusing on regulatory elements have shown that large, cell-specific stretch enhancers harbor smaller effect size eQTLs than ubiquitous promoter regions 42 and that genes with more cognate enhancer sequence are depleted for eQTLs43. The results we report are consistent with these observations: we have shown that islet eQTLs that map to the islet stretch enhancers most frequently implicated in GWAS regions have smaller eQTL effect sizes (and therefore may be more difficult to detect). One consequence, for example, is that, at a GWAS variant that has regulatory impact on multiple cis-genes, eQTL signals for bystander genes (those not directly implicated in disease pathogenesis) may be easier to detect than those that are actually mediating the signal.Finally, it is important to emphasize that, even when co-localization has been robustly demonstrated between a GWAS signal and a tissue-appropriate eQTL signal, this does not of itself implicate the eGene concerned as mediating disease predisposition. Causal relationships other than “variant to gene to disease” are possible, including the possibility the variant has separate (horizontally) pleiotropic effects on both. Growing understanding of the extent of shared local regulatory activity and regulatory pleiotropy makes such an alternative explanation all the more credible. In our view, it is best to regard the genes highlighted by coincident GWAS and eQTL signals as “candidate” effector transcripts, and to proceed to experimental approaches that enable direct tests of causality. These may involve perturbing the gene across a range of disease-relevant cell-lines and animal models, and determining the impact on phenotypic readouts that represent reliable surrogates of disease pathophysiology. MethodsPancreatic Islet sample collection and processing	 	 	 	Geneva samples: Islet sample procurement, mRNA processing and sequencing procedure has been described in 17. Briefly, Islets isolated from cadaveric pancreas were provided by the Cell Isolation and Transplant Center, Department of Surgery, Geneva University Hospital (Drs. T. Berney and D. Bosco) through the Juvenile Diabetes Research Foundation (JDRF) award 31-2008-416 (ECIT Islet for Basic Research Program). mRNA was extracted using RLT buffer (RNeasy, Qiagen) and total RNA was prepared according to the standard RNeasy protocol. The original RNA libraries were 49-bp paired-end sequenced however, in order to allow joint analysis with the other available datasets for this study, mRNA samples were re-processed using a 100-bp paired-end sequencing protocol. The library preparation and sequencing followed customary Illumina TruSeq protocols for next generation sequencing as described in the original paper 17. All procedures followed ethical guidelines at the University Hospital in Geneva.Lund Samples: Islet sample procurement, mRNA processing and sequencing procedure has been described in 15. Along with the 89 islet samples previously published in 15, we included 102 islet samples and processed these uniformly following the same protocol. These islet samples were obtained from 191 cadaver donors of European ancestry from the Nordic Islet Transplantation Programme (http://www.nordicislets.org). Purity of islets was assessed by dithizone staining, while measurement of DNA content and estimate of the contribution of exocrine and endocrine tissue were assessed as previously described 44. Total RNA was isolated with the AllPrep DNA/RNA Mini Kit following the manufacturer’s instructions (Qiagen), sample preparation was performed using Illumina’s TruSeq RNA Sample Preparation Kit according to manufacturer's recommendations. The target insert size of 300 bp was sequenced using a paired end 101 bp protocol on the HiSeq2000 platform (Illumina). Illumina Casava v.1.8.2 software was used for base calling. All procedures were approved by the ethics committee at Lund University.Oxford samples: Samples collected in Oxford and Edmonton that were jointly sequenced in Oxford are included in this set of samples. Islet sample procurement, mRNA processing and sequencing procedure has been described in 16. To the 117 samples previously published (78 from Edmonton and 39 from Oxford), 57 samples were added and processed following similar protocols as before (27 from Edmonton and 30 from Oxford). Briefly, freshly isolated human islets were collected at the Oxford Centre for Islet Transplantation (OXCIT) in Oxford, or the Alberta Diabetes Institute IsletCore (www.isletcore.ca) in Edmonton, Canada. Additional islets were obtained from the Alberta Diabetes Institute IsletCore’s long-term cryopreserved biobank. Freshly isolated islets were processed for RNA and DNA extraction after 1–3 days in culture in CMRL media. Cryopreserved samples were thawed as described 45 [Lyon, et al., Endocrinology, 2016]. RNA was extracted from human islets using Trizol (Ambion, UK or Sigma Aldrich, Canada). To clean remaining media from the islets, samples were washed three times with phosphate buffered saline (Sigma Aldrich, UK). After the final cleaning step 1 mL Trizol was added to the cells. The cells were lysed by pipetting immediately to ensure rapid inhibition of RNase activity and incubated at room temperature for ten minutes. Lysates were then transferred to clean 1.5 mL RNase-free centrifuge tubes (Applied Biosystems, UK). RNA quality (RIN score) was determined using an Agilent 2100 Bioanalyser (Agilent, UK), with a RIN score > 6 deemed acceptable for inclusion in the study. Samples were stored at -80°C prior to sequencing. Poly-A selected libraries were prepared from total RNA at the Oxford Genomics Centre using NEBNext ultra directional RNA library prep kit for Illumina with custom 8bp indexes 46. Libraries were multiplexed (3 samples per lane), clustered using TruSeq PE Cluster Kit v3, and paired-end sequenced (100nt) using Illumina TruSeq v3 chemistry on the Illumina HiSeq2000 platform. All procedures were approved by the Human Research Ethics Board at the University of Alberta (Pro00013094), the University of Oxford's Oxford Tropical Research Ethics Committee (OxTREC Reference: 2–15), or the Oxfordshire Regional Ethics Committee B (REC reference: 09/H0605/2). All organ donors provided informed consent for use of pancreatic tissue in research.USA samples: Islet sample procurement, mRNA processing and sequencing has been described in 12. Briefly, 39 Islet samples from organ donors were received from the Integrated Islet Distribution Program, the National Disease Research Interchange (NDRI), and Prodo- Labs. Total RNA from 2000-3000 islet equivalents (IEQ) was extracted and purified using Trizol (Life Technologies). RNA quality was confirmed with Bioanalyzer 2100 (Agilent); samples with RNA integrity number (RIN) > 6.5 were prepared for mRNA sequencing. We added External RNA Control Consortium (ERCC) spike-in controls (Life Technologies) to one microgram of total RNA. PolyA+, stranded mRNA RNA-sequencing libraries were generated for each islet using the TruSeq stranded mRNA kit according to manufacturer’s protocol (Illumina). Each islet RNA-seq library was barcoded, pooled into 12-sample batches, and sequenced over multiple lanes of HiSeq 2000 to obtain an average depth of 100 million 2 x 101 bp sequences. All procedures followed ethical guidelines at the National Institutes of Health (NIH.)Beta-cell sample collection and processingSample collection, mRNA processing and sequencing procedure has been described in 17. To the 11 FAC sorted beta-cells population samples previously published, we added 15 more samples that were processed following the same protocols. Briefly, islets were dispersed into single cells, stained with Newport Green, and sorted into ‘‘beta’’ and ‘‘non-beta’’ populations as described previously 47. The proportion of beta (insulin), alpha (glucagon), and delta (somatostatin) cells in each population (as percentage of total cells) was determined by immunofluorescence. mRNA extractions as well as sequencing followed the same details described for islets samples processing for the Geneva samples.Read-mapping and exon quantificationThe 100-bp sequenced paired-end reads were mapped to the GRCh37 reference genome 48 with GEM 49. Exon quantifications were calculated for all elements annotated in GENCODE v19 50, removing genes with more than 20% zero read count. All overlapping exons of a gene were merged into meta-exons with identifier of type ENSG000001.1_exon.start.pos_exon.end.pos, as described in 21. Read counts over these elements were calculated without using read pair information, except for excluding reads where the pairs mapped to two different genes. We counted a read in an exon if either its start or end coordinates overlapped an exon. For split reads, we counted the exon overlap of each split fragment, and added counts per read as 1/(number of overlapping exons per gene). Gene level quantifications used the sum of all reads mapped to exons from the gene. Genes with more than 20% zero read counts were removed.  	Genotype imputationGenotypes for all islet samples, including 19 beta-cell samples, were available from omniexpress and omni2.5 genotype arrays. Quality of genotyping from the shared SNPs in both arrays was assessed before imputation separately by removing SNPs as follows: 1) SNPs with minor allele frequency (MAF) < 5%; 2) SNP genotype success rate <95%; 3) Palindromic SNPs with MAF > 40%; 4) HWE < 1e-6; 5) Absence from 1000G reference panel; 6) Allele inconsistencies with 1000G reference panel; 7) Probes for same rsID mapping to multiple genomic locations (1000G reference-consistent probe kept). Finally, samples were excluded if they had an overlap genotype success rate lower than 90%; and MAF differences larger than 20% compared to the 1000G reported european MAF.The two panels were separately pre-phased with SHAPEIT v2 51 using the IMPUTE2-supplied genetic maps. After pre-phasing the panels were imputed with IMPUTE2 v2.3.1 52 using the 1000 Genomes Phase I integrated variant set (March 2012) as the reference panel 53. SNPs with INFO score > 0.4 and HWE p > 1e-6 (for chrX this was calculated from female individuals only) from each panel were kept. A combined vcf for each chromosome was generated from the intersection of the checked variants in each panel. Directly genotyped SNPs with a MAF < 1% (including the exome-components of the chips not shared between all centres) were merged into the combined vcfs: i) If SNPs were not imputed they were added and ii) If SNPs had been imputed, the imputed calls for the individual were replaced by the typed genotype. Dosages were calculated from the imputation probabilities (genotyped samples) or genotype calls (WGS samples). For the 22 autosomes the dosage calculation was: 2x( (0.5*heterozygous call) + homozygous alt call). For chromosome X (where every individual should be functionally hemizygous), the calculation was: (0.5*heterozygous call) + homozygous alt call). Genotype calls for males can only be '0/0' and '1/1'. The total number of variants available for analysis after quality assessment was 8,056,952.For the 26 beta cell samples, 19 had genotypes available from omniexpress arrays, whereas 7 had the DNA sequence available. Variant calling from DNA sequence has been previously described in 17. Briefly, the Genome Analysis Toolkit (GATK) 1.5.31 54 was used following the Best Practice Variant Detection v3 to call variants. Reads were aligned to the hg19 reference genome with BWA 55. We used a confidence score threshold of 30 for variant detection and a minimum base quality of 17 for base calling. Good confidence (1% FDR) SNP calls were then imputed on the 1000 Genomes reference panel and phased with BEAGLE 3.3.2 56. Imputation of variants from samples with arrays genotyping were imputed together with genotypes from individuals with islets samples as described before and then merged with genotypes from DNA sequences. SNPs with INFO score > 0.4, HWE p > 1e-6 and MAF > 5%, were kept for further analysis. The total number of variants available for analysis after quality assessment was 6,847,993.RNAseq quality assessment and data normalizationHeterozygous sites per sample were matched with genotype information to confirm the ID of the samples 57. 11 samples did not match with their genotypes, 6 of which would be corrected by identifying a good match. For the remaining samples, no matches were found on the genotypes and they were removed from the dataset, giving a total of 420 samples with genotypes.Raw read counts from exons and genes were scaled to 10 million to allow comparison between samples with different libraries. Scaled raw counts were then quantile normalized. We used principal component analysis (PCA) to evaluate the effects of unwanted technical variation and the expected batch effects due to fact that the islet sample processing mRNA sequencing was performed across four labs. We evaluated a) the optimal number of principal components (PCs) for the discovery of eQTLs and b) the minimum number of PCs necessary to control for laboratories of origin batch effects (Supp. Figure S1). We performed eQTL discovery controlling for 1,5,10, 20 30 40 and 50 PCs for expression, as well as gender, 4 PCs derived from genotype data, and a variable defining the laboratory of origin (coded as: OXF, LUND, GEN and USA). After evaluation of the results, we conclude that controlling for 20 PCs was optimal. To ensure that we controlled for batch effects with these variables, we used a permutation scheme as follows: expression sample labels and expression covariates were permuted within each of the 4 laboratories before performing a standard eQTL analysis against non-permuted genotypes (and matched PCs for genotypes) using different numbers of PCs for expression. Significant eQTLs in any of these analyses are considered a false positive due to technical differences across laboratories of origin of the samples. Our results indicate that 10PCs were sufficient to minimize the number of false positives due to batch effects originating from differences in processing of the islet samples (Supp. Figure S1).	eQTL analysiseQTL analysis for islets and beta-cells were performed using fastQTL 18 on 420 islets samples and 26 beta-cells samples with available genotypes. Cis-eQTL analysis was restricted to SNPs in a 1MB window upstream and downstream the transcription start site (TSS) for each gene and SNPs with MAF>1%. For the analysis of beta-cell samples, we used a filter of MAF>5%. Exon-level eQTLs identified best exons-SNP association per gene (using the –group flag), while gene level eQTLs used gene quantifications and identified the best gene-SNP association. Variables included in the linear models were the first 4 PCs for genotypes, the first 25 PCs for expression, gender and a variable identifying the laboratory of origin of the samples. Significance for the SNP-gene association was assessed using 1000 permutations per gene, correcting P values with a beta approximation distribution 18. Genome-wide multiple testing correction was performed using the q-value correction 19 implemented in largeQvalue 58.To discover multiple independent eQTLs, we applied a stepwise regression procedure as described in 59. Briefly, we started from the set of eGenes discovered in the first pass of association analysis (FDR < 1%). Then, the maximum beta-adjusted P value (correcting for multiple testing across the SNPs and exons) over these genes was taken as the gene-level threshold. The next stage proceeded iteratively for each gene and threshold. A cis-scan of the window was performed in each iteration, using 1,000 permutations and correcting for all previously discovered SNPs. If the beta adjusted P value for the most significant exon-SNP or gene-SNP (best association) was not significant at the gene-level threshold, the forward stage was complete and the procedure moved on to the backward step. If this P value was significant, the best association was added to the list of discovered eQTLs as an independent signal and the forward step proceeded to the next iteration. The exon level cis-eQTL scan identified eQTLs from different exons, but reported only the best exon-SNP in each iteration. Once the forward stage was complete for a given gene, a list of associated SNPs was produced which we refer to as forward signals. The backward stage consisted of testing each forward signal separately, controlling for all other discovered signals. To do this, for each forward signal we ran a cis scan over all variants in the window using fastQTL, fitting all other discovered signals as covariates. If no SNP was significant at the gene-level threshold the signal being tested was dropped, otherwise the best association from the scan was chosen as the variant that represented the signal best in the full model.GTEx eQTLsWe identified exon level eQTLs for 44 GTEx tissues using fastQTL 18 following the same procedure as for the islet eQTLs. Covariates included followed the previously published number of PCs for expression 60 and included 15 PCs for expression for tissues with less than 154 samples; 30 PCs for samples between 155 and 254 samples; and 35 PCs for samples with more than 254 samples. Independent eQTLs from exons were identified as described for islets eQTLs. The proportion of shared eQTLs between islet and beta-cell eQTLs and the eQTLs from GTEx tissues were identified using ?1 19.Tissue de-convolutionTo identify the contribution of the beta-cells, non-beta cells and exocrine components (non-islets cell) expression to the total gene expression measure in islets we performed an expression deconvolution analysis. Expression profiles from GTEx whole pancreas was used as a model for the exocrine component of expression 60, while FAC-sorted expression profiles from beta-cell and non-beta-cells from Nica et al 17 were used to identify the fraction of expression derived from islets cells. First, we performed differential expression analysis of a) exocrine versus whole islet samples; b) beta-cell versus whole islet samples; c) non-beta-cell versus whole islet samples. The top 500 genes from each analysis were combined, and a deconvolution matrix of log2-transformed median expression values was prepared for each cell type. Next, deconvolution was performed using the Bioconductor package DeconRNASeq 61.Deconvolution values per sample are included in the covariates file, together with the expression values in the EGA submission. Enrichment of eQTLs in T2D and glycemic GWASTo perform an enrichment analysis of T2D and glycemic traits GWAS associations among eQTLs across tissues, we examined 78 T2D associated signals 3, and 44 variants from associations with continuous glycemic traits relevant to T2D predisposition (including fasting glucose, and beta-cell function (HOMA-B) in non-diabetic individuals) (Supp Table 13) 28,30,45. For each GWAS lead variant, we extracted the eQTL with the greatest absolute effect size estimate from the results for all GTEx tissues and the InsPIRE pancreatic islets. We then compared their observed effect size estimates to those derived from a null distribution of 15,000 random variants matched in terms of the number of SNPs in LD, distance to TSS, number of nearby genes and minor allele frequency. For comparison with results observed for T2D loci, we also included the set of 50 lead variants implicated by GWAS in predisposition to T1D 31.Co-localization of islet eQTL with T2D GWASCo-localization of GWAS variants and eQTLs were performed using both COLOC [PMID: 24830394] and RTC [ref]. For the analysis using COLOC, all variants within 250 kilobase flanking regions around the index variants were tested for co-localization using default parameters from the software were used on summary statistics from T2D GWAS from 4 and fasting glucose 29. GWAS variants and eSNPs pairs were considered to co-localize if the COLOC score for shared signal was larger than 0.9. RTC analysis was also performed using defaults parameters from the software with a list of 86 lead GWAS variants for T2D and fasting glucose (Supp. Table ST13). Associations between GWAS and gene expression were considered as co-localizing if RTC score was larger than 0.9 (Supp. Table ST15). Chromatin states, Islet ATAC-seq and Transcription factor (TF) footprintsWe used a previously published 13 chromatin state model that included Pancreatic Islets along with 30 other diverse tissues 12. Briefly, these chromatin states were generated from cell/tissue ChIP-seq data for H3K27ac, H3K27me3, H3K36me3, H3K4me1, and H3K4me3, and input control from a diverse set of publically available data 10,62-64 using the ChromHMM program 65. Chromatin states were learned jointly from 33 cell/tissues that passed QC by applying the ChromHMM (version 1.10) hidden Markov model algorithm at 200-bp resolution to five chromatin marks and input 12. We ran ChromHMM with a range of possible states and selected a 13-state model, because it most accurately captured information from higher-state models and provided sufficient resolution to identify biologically meaningful patterns in a reproducible way. As reported previously 12, Stretch Enhancers were defined as contiguous enhancer chromatin state (Active Enhancer 1 and 2, Genic Enhancer and Weak Enhancer) segments longer than 3kb, whereas Typical Enhancers were enhancer state segments smaller than the median length of 800bp 10.We used the union of ATAC-seq peaks previously identified from two human islet samples called using MACS2 v2.1.0 12. We also used previously published TF footprints that were generated in a haplotype-aware manner using ATAC-seq and genotyping data from the phased, imputed genotypes for each of two islet samples using vcf2diploid v0.2.6a 12.Filtering eQTL SNPs for epigenomic analysesSince low MAF SNPs, due to low power, can only be identified as significant eQTL SNP (eSNPs) with high eQTL effect sizes (slope or the beta from the linear regression), we observed that absolute effect size varies inversely with MAF (Supp. Figure S15). To conduct eQTL effect size based analyses in an unbiased manner, we selected significant (FDR 1%) eSNPs with MAF >= 0.2. We then pruned this list to retain the most significant SNPs with pairwise LD(r2)<0.8 for the EUR population using PLINK 66 and 1000 genomes variant call format (vcf) files (downloaded from ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/) for reference (European population). This filtering process resulted in N=3832 islet eSNPs.Enrichment of genetic variants in genomic featuresTo calculate the enrichment of islet eSNPs to overlap with genomic features such as chromatin states and transcription factor (TF) footprint motifs, we used the GREGOR tool 67. For each input SNP, GREGOR selects ~500 control SNPs matched for MAF, distance to the gene, and number of SNPs in LD(r2) ≥ 0.99. A unique overlap is reported if the feature overlaps any input lead SNP or its LD(r2)>0.99 LD SNPs. Fold enrichment is calculated as the number unique overlaps over the mean number of loci at which the matched control SNPs (or their LD(r2>)0.99 SNPs) overlap the same feature. This process accounts for the length of the features, as longer features will have more overlap by chance with control SNP sets. We used the following parameters in GREGOR for eQTL enrichment: r2 threshold (for inclusion of SNPs in linkage disequilibrium (LD) with the lead eSNP) = 0.99, LD window size = 1Mb, and minimum neighbor number = 500.For enrichment of T2D GWAS SNPs in islet chromatin states, we downloaded the list of T2D GWAS  SNPs from [CITE MAHAJAN 2018]. We pruned this list to retain the most significant SNPs with pairwise LD(r2)<0.2 for the EUR population using PLINK 66 and 1000 genomes variant call format (vcf) files (downloaded from ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/) for reference (European population). This filtering process resulted in N=378 T2D GWAS SNPs. We used GREGOR to calculate enrichment using the following specific parameters: r2 threshold (for inclusion of SNPs in linkage disequilibrium (LD) with the lead eSNP) = 0.8, LD window size = 1Mb, and minimum neighbor number = 500We investigated if footprint motifs were more enriched to overlap eQTL of high vs low effect sizes. We sorted the filtered (as described above) eQTL list by absolute effect size values and partitioned into two equally sized bins (N eSNPs = 1,916). Since TF footprints were available for a large number of motifs (N motifs = 1,995), the enrichment analysis had a large multiple testing burden and limited power with 1,916 eSNPs in each bin. Therefore, we only considered footprint motifs that were significantly enriched (FDR <1%, Benjamini & Yekutieli method from R p.adjust function, N motifs = 283) to overlap the bulk set of eSNPs (LD r2<0.8 pruned but not MAF filtered, N eSNPs = 6,468, Supp. Table ST10) for enrichment to overlap the binned set of eSNPs. This helped reduce the multiple testing burden. We then calculated enrichment for the selected footprints to overlap SNPs in each bin using GREGOR with same parameters as described above (Supp. Table ST11).eSNP effect size distribution in chromatin states and ATAC-seq peaks within chromatin statesWe identified the islet eQTL eSNPs (after LD pruning and MAF filtering as described above) occurring in chromatin states or ATAC-seq peaks within chromatin states using BEDtools intersect 68. Similar to the enrichment calculation procedure, we considered a unique eQTL overlap if the lead eSNP or a proxy SNP with LD(r2)>0.99 occurred in these regions. We considered the effect size as the slope or the beta from the linear regression for the eQTL overlapping each region. P values were calculated using the Wilcoxon Rank Sum Test in R.TF motif directionality analysisFor TF footprint motifs that were significantly enriched to overlap the full set of islet eQTLs (after LD pruning to r2<0.8) with (FDR 1%, Benjamini & Yekutieli method from R p.adjust function, N motifs = 283), we determined the overlap position of the eSNP (pruned LD r2<0.8 lead eSNPs and their LD r2>0.99 proxy SNPs) with each TF footprint motif. We considered instances where the eSNP overlapped the TF footprint motif at a position with information content >= 0.7 and either the eSNP effect or the non-effect allele was the most preferred base in the motif. We selected TF footprint motifs that had 10 or more such eSNP overlap instances (N=278). For each TF footprint motif and eSNP overlap, we re-keyed the direction of effect on the target gene being positive or negative with respect to the most preferred base in the motif. For each TF motif, we compiled the fraction of instances where the SNP allele that was most preferred in the TF footprint motif (i.e. base with highest probability in the motif) associated with increased expression of the associated gene. We refer to this metric as the motif directionality fraction where fraction near 1 suggests activating and fraction near 0 suggests repressive preferences towards the target gene expression. Motif directionality fraction near 0.5 suggests no activity preference or context dependence.We compared our results to a previously published study that quantified transcription activating or repressive activities based on massively parallel reported assays in HepG2 and K562 cells 27 (Supp. Figure S10). We then considered 99 motifs from our analyses that were reported to have significant (P<0.01) activating or repressive scores from MPRAs in both HepG2 and K562. With the null expectation of the motif directionality fraction being equal to 0.5, i.e. TF binding equally likely to increase or decrease target gene expression, we used a binomial test to calculate TF that show significant deviation from the null (N = 8 at FDR < 10%, Supp. Table ST11).Cell cultureMIN6 mouse insulinoma beta cells 69 were grown in Dulbecco’s modified Eagle’s Medium (Sigma-Aldrich, St. Louis, Missouri/USA) with 10% fetal bovine serum, 1 mM sodium pyruvate, and 0.1 mM beta-mercaptoethanol. INS-1-derived 832/13 rat insulinoma beta cells (a gift from C. Newgard, Duke University, Durham, North Carolina/USA) were grown in RPMI-1640 medium (Corning, New York/USA) supplemented with 10% fetal bovine serum, 10 mM HEPES, 2 mM L-glutamine, 1 mM sodium pyruvate, and 0.05 mM beta-mercaptoethanol. EndoC-?H1 cells (Endocell) were grown according to (Ravassard et al., 2011) in Dulbecco’s modified Eagle’s medium (DMEM; Sigma-Aldrich), 5.6mmol/L glucose with 2% BSA fraction V fatty acid free (Roche Diagnostics), 50?mol/L 2-mercaptoethanol, 10mmol/L nicotinamide (Calbiochem), 5.5?g/ml transferrin (Sigma-Aldrich), 6.7ng/ml selenite (Sigma-Aldrich), 100U/ml penicillin, and 100?g/ml streptomycin. Cells were grown on coating consisting of 1% matrigel and 2µg/mL fibronectin (Sigma). We maintained cell lines at 37° C and 5% CO2.Transcriptional reporter assaysTo test haplotypes for allele-specific effects on transcriptional activity, we PCR-amplified a 765-bp genomic region (haplotype A) containing variants: rs7798124, rs7798360, and rs7781710, and a second 592-bp genomic region (haplotype B) containing variants: rs10228796, rs10258074, rs2191348, and rs2191349 from DNA of individuals homozygous for each haplotype. The oligonucleotide primer sequences are listed in Supp. Table ST17. We cloned the PCR amplicons into the multiple cloning site of the Firefly luciferase reporter vector pGL4.23 (Promega, Fitchburg, Wisconsin/USA) in both orientations, as described previously 70. Vectors are designated as ‘forward’ or ‘reverse’ based on the PCR-amplicon orientation with respect to DGKB gene. We isolated and verified the sequence of five independent clones for each haplotype in each orientation. For the 5’ eQTL a 250 bp construct containing the rs17168486 SNP (Origene) was subcloned into the Firefly luciferase reporter vector pGL4.23 (Promega) in both orientations.We plated the MIN6 (200,000 cells) or 832/13 (300,000 cells) in 24-well plates 24 hrs before transfections and the EndoC-?H1 cells (140.000 cells) plated 48H prior to transfection. We co-transfected the pGL4.23 constructs with phRL-TK Renilla luciferase reporter vector (Promega) in duplicate into MIN6 or 832/13 cells and in triplicate for EndoC-?H1 cells. For the transfections we used Lipofectamine LTX (ThermoFisher Scientific, Waltham, Massachusetts/USA) with 250 ng of plasmid DNA and 80 ng Renilla for MIN6 cells, Fugene6 (Promega) with 720 ng of plasmid and 80 ng Renilla for 832/13 cells per each welll and Fugene6 with 700 ng plasmid and 10 ng renilla for EndoC-?H1 cells. We incubated the transfected cells at 37° C with 5% CO2 for 48 hours. We measured the luciferase activity with cell lysates using the Dual-Luciferase® Reporter Assay System (Promega). We normalized Firefly luciferase activity to the Renilla luciferase activity. We compared differences between the haplotypes using unpaired two-sided t-tests. All experiments were independently repeated on a second day and yielded comparable results.Electrophoretic Mobility Shift AssaysElectrophoretic mobility shift assays were performed as previously described (CITE). We annealed 17-nucleotide biotinylated complementary oligonucleotides (Integrated DNA Technologies) centered on variants: rs10228796, rs10258074, rs2191348, and rs2191349 (Supp. Table ST18). MIN6 nuclear protein extract was prepared using the NE-PER kit (Thermo Scientific). To conduct the EMSA binding reactions, we used the LightShift Chemiluminescent EMSA kit (Thermo Scientific) following the manufacturer’s protocol. Each reaction consisted of 1 ?g poly(dI-dC), 1x binding buffer, 10 ?g MIN6 nuclear extract, 400 fmol biotinylated oligonucleotide. We resolved DNA-protein complexes on nondenaturing DNA retardation gels (Invitrogen) in 0.5x TBE. We transferred the complexes to Biodyne B Nylon membranes (Pall Corporation), and UV cross-linked (Stratagene) to the membrane. We used chemiluminescence to detect the DNA-protein complexes. EMSAs were repeated on a second day with comparable results.Data accessGenotype and sequence data have been deposited at the European Genome-phenome Archive (EGA; https://www.ebi.ac.uk/ega/) under the accession numbers EGAXXX and EGAXXX. Lead eQTLs per genes can be found in the supplemental tables listed below. Nominal P values for all gene-SNPs and exon-SNPs associations presented in this manuscript can be found using the following link: http://funpopgen.unige.ch/data Supporting informationSupplemental Tables list:Table 1: List of lead eQTLs per gene in islets (exon level quantifications).Table 2: List of lead eQTLs per gene in islets (gene level quantifications).Table 3: List of 337 eQTLs only active in islets. Table 4: List of lead eQTLs per gene in beta-cells (exon level quantifications).Table 5: List of lead eQTLs per gene in GTEx pancreas (exon level quantifications).Table 6: List of 227 significant eQTLs in islets, also active in Beta-cells. Table 7: List of lead GxBeta-cells proportions eQTLs.Table 8: List of lead GxNonBeta-cells proportions eQTLs.Table 9: List of lead GxExocrine-cells proportions eQTLs.Table 10: List of bulk enrichment of footprints Table 11: Binned enrichment of footprints. Table 12: Motif directionality results. Table 13: List of GWAS variants used GWAS enrichment analysis.Table 14: Enrichment analysis results for GWAS-eSNPs across GTEx tissues. Table 15: Joint results for COLOC analysis with ENGAGE GWAS results for fasting glucose (FG) and DIAGRAM GWAS results for T2D.Table  16: Overlap between eQTL signals and glycemic response. Table 17: Reporter assay primers: Oligonucleotide primers used for the transcriptional reporter assaysTable 18: EMSA probes: Oligonucleotide probes used for the electrophoretic mobility shift assaysAcknowledgments Funding A. Varshney was supported by the American Association for University Women International Doctoral Fellowship, Barbour Doctoral Scholarship, and the University of Michigan Rackham Predoctoral Fellowship.F. S. Collins, M. Erdos, and N. Narisu were supported by NHGRI - ZIA HG000024. K. Mohlke was supported by NIH R01 DK072193, NIH U01 DK105561.S. C. J. Parker, L. J. Scott, and M. Boehnke were supported  by U01DK062370.MvdB was supported by a Novo Nordisk postdoctoral fellowship run in partnership with the University of Oxford. M. Stitzel was supported by K99/R00DK092251.P. Orchard was supported by GSTP grant T32 HG00040 from the National Genome Research Institute of the National Institutes of Health.S. C. J. Parker was supported by National Institute of Diabetes and Digestive and Kidney Diseases grants R00 DK-099240 and R01 DK-117960, American Diabetes Association Pathway to Stop Diabetes grant 1–14-INI-7. The Alberta Diabetes Institute IsletCore was supported by the Alberta Diabetes Foundation. We thank the Human Organ Procurement and Exchange (HOPE) program and the Trillium Gift of Life Network (TGLN) for their efforts in obtaining human organs for research.21