Through my dissertation work, I have analyzed large omics datasets to supplement our understanding of gene regulatory mechanisms that underlie the associations between genetic variation and disease traits such as T2D. Several important themes have emerged from my work which are exciting avenues to further augment our understanding of how predisposition to complex disease is encoded in our ‘non-coding’ genome. 


\subsection{Regulatory buffering and the need for molecular context specific studies}
The regulatory nature of a majority of common disease associated variants motivated our investigation of regulatory regions across the genome. We asked asked how the genetics of gene regulation differed across regulatory annotations that had been defined using different sets of epigenomic marks - all of which were shown to marked active regulatory regions. We observed that eQTL occurring in HOT regions that represent mostly promoter-like regions had significantly higher effect sizes than those occurring in more cell-specific stretch enhancers. This effect remained after controlling for distance to the target gene. However, chromatin accessibility QTL in stretch enhancers have significantly larger effect sizes compared to those in HOT regions. These observations were quite robust in that we observed these trends in multiple cell/tissue types such as blood/K652 cell lines, LCL/GM12878 cell lines and islet tissue.


These seemingly conflicting results indicated that the chromatin in the cell-specific stretch enhancers was genetically primed for larger effects on accessibility, however, genetic effects on modulation of gene expression were lower. We noted here that both chromatin and expression QTLs analyzed were identified in the basal state for the cells/tissues. A recent study showed that ~60\% of eQTL identified in stimulated condition in macrophages were identified as chromatin QTL in the basal state \cite{alasooSharedGeneticEffects2018}. Our observations and other supportive evidence suggest that lower effects on gene expression in the basal state despite higher propensity for chromatin effects could be a mechanism to ensure stable expression of critical genes in the basal state while priming these for quick response to patho-physiologic stimuli. Similar inferences about robust gene expression and ‘enhancer redundancy’ have been reported recently. One such study showed that dosage-sensitive genes have evolved robustness to the disruptive effects of genetic variation by expanding their regulatory domains \cite{wangEnhancerRedundancyPredicts2018}. Others have questioned if stretch/super enhancers are really different from other enhancers that just happen to occur close-by \cite{pottWhatAreSuperenhancers2015}. Numerous studies perturbed elements within these enhancers but showed conflicting results - some, where the perturbation affected gene expression and others where there was no observable effect on gene expression, again implying redundancy in gene regulation by individual components of super enhancers \cite{hayGeneticDissectionAglobin2016, shinHierarchyMammarySTAT5driven2016, moorthyEnhancersSuperenhancersHave2017, xieMultiplexedEngineeringAnalysis2017}. 


Considering regulatory annotations from a disease genetics perspective, I and others have observed high enrichment of GWAS loci in trait-relevant stretch enhancers (eg. T2D GWAS loci enriched in islet stretch enhancers, Rheumatoid Arthritis GWAS loci enriched in lymphoblastoid cell line GM12878 stretch enhancers, Fasting Insulin GWAS loci enriched in Adipose and Skeletal Muscle stretch enhancers among several such observations). However, eQTL loci, which have long been touted to help identify the target genes of the GWAS signals through co-localization methods, are highly enriched in promoter regions. The differences in the genetics of gene regulation between these annotations that I have demonstrated could help reconcile why many cis-eQTLs are shared across cell types and infrequently co-localize with GWAS signals \cite{liuFunctionalArchitecturesLocal2017, huangFinemappingInflammatoryBowel2017, gtexconsortiumGeneticEffectsGene2017}. 


We hypothesize that cell-specific stretch enhancers drive critical responses to external stimuli and therefore some genetic associations with gene expression might be evident only under relevant conditions. Stimuli such as nutrient conditions, stress or hormone signaling could modulate TF abundance and localization which could drive context-specific mechanisms. Therefore, one critical and exciting direction is to understand the genetics of gene regulation under carefully selected stimulatory conditions in relevant cell types. Studying the impact of genetic variants under such contexts may be crucial for revealing functional convergence of disease-associated variants. More specifically, we advocate conducting molecular QTL screens under stimulus/treatment induced conditions with carefully considered sample sizes and time points (Fig. \ref{fig:c6_f1} A and B). 


\begin{figure}
            \centering
            \includegraphics[width=1\textwidth]{6_future_directions/figures/fig1.png}
            \caption{Context-specific xQTL mapping to better understand GWAS}
            \label{fig:c6_f1}
\end{figure}


Response studies have been limited in the T2D genomics literature. The potential of such studies however is demonstrated by other work that highlights context specific effects. One such study explored a T2D GWAS variant (rs508419) that lies in a skeletal muscle-specific active promoter region at the ANK1 locus \cite{scottGeneticRegulatorySignature2016, yanNovelTypeDiabetes2016}. Human skeletal muscle eQTL data indicated that the T2D risk allele was associated with higher ANK1 expression \cite{scottGeneticRegulatorySignature2016}. Interestingly, however, increased ANK1 protein only affected glucose uptake when treated with insulin; there was no detectable effect of increased ANK1 protein under basal conditions. The same variant, however, is associated with reduced expression of the transcription factor NKX6-3 in the islet tissue \cite{scottGeneticRegulatorySignature2016, varshneyGeneticRegulatorySignatures2017}, representing a tissue-dependent effect of regulatory variants, and potentially more complicated genetic architecture. Therefore, modelling environmental stimuli in functional T2D genomic studies is both important and challenging.
        
The increased glucose levels in the body resulting from insulin resistance are critical environmental factors for the pancreatic islets. Therefore, glucose specific effects on islet function can be assessed by culturing islets in low vs high glucose conditions and profiling RNA (mRNA-seq) or open chromatin (ATAC-seq) followed by QTL analysis to identify context specific molecular QTL (xQTL) in islets. Patient-derived induced pluripotent stem cell (iPSC) lines differentiated towards the islet lineage could also be useful for this purpose and better suited to culture in the laboratory. 
                        
\subsection{Single-cell molecular profiling approaches to dissect islet heterogeneity}
It is important to note that islets are heterogeneously composed of multiple subtypes (including alpha, beta, gamma and delta cells) that have diverse functions. Recent developments in single cell assays followed by sequencing technologies, both for measuring RNA (scRNA-seq) and open chromatin (scATAC-seq) therefore present exciting avenues. For example, it is now possible to identify regulatory sites that are specific to, say insulin secreting beta cells. scRNA-seq or scATAC-seq can be used to define cell-type proportions, patterns of which when analyzed across individuals can would identify cell-type proportion QTLs, where genetic variation influences proportion of different subtypes. Performing these studies while considering basal or glucose response contexts could again be invaluable in identifying context specific genetic effects on islet cell biology. Such studies could lead to a better understanding of islet biology and potentially at T2D related trait GWAS loci.         
        
\subsection{Linking molecular profiling and xQTL information with GWAS and identifying causal relationships}                
Genome-wide molecular profiling data can be jointly analyzed with GWAS summary data to help fine-map causal variants(s). For example, a hierarchical modeling approach implemented in the fGWAS package \cite{pickrellJointAnalysisFunctional2014} statistically models the prior probabilities that a genomic region contains an association with the trait and that  a SNP in that region is causal, allowing these probabilities to vary based on the underlying functional molecular profiles such as open chromatin. Therefore, statistical integration of molecular profiling data, especially under specific contexts can further enable identification of causal variants (Fig. \ref{fig:c6_f1} C). Next, to establish mechanistic links between genetic effects on these molecular profiles and on the relevant trait such as T2D, it should be determined if the same genetic variant drives the xQTL as well as the GWAS signal. Approaches that test for such ‘co-localization’ between two signals have been established \cite{giambartolomeiBayesianTestColocalisation2014, nicaCandidateCausalRegulatory2010, hormozdiariColocalizationGWASEQTL2016}, (Fig. \ref{fig:c6_f1} C). Furthermore, potential causal relationships between the xQTL and GWAS signals can be assessed using Mendelian randomization \cite{hormozdiariColocalizationGWASEQTL2016}. For loci with ≥2 molecular profile QTLs, potential causal direction between each pair can be assessed using the Causal Inference Test \cite{millsteinDisentanglingMolecularRelationships2009} and MR-Steiger \cite{hemaniOrientingCausalRelationship2017}; these tests provide complementary information. While molecular xQTL data elucidates the genetics of the epigenomic or chromatin landscape, recently developed bayesian strategies \cite{kumasakaHighresolutionGeneticMapping2019} can further help determine causal interactions and the relationships between multiple identified regulatory elements .
                
                        
\subsection{Functional follow up of prioritized variants}
While mapping the epigenomic landscape and identifying genetic associations can inform candidate regulatory regions and potentially causal variants, complementary approaches are needed to functionally validate these effects. Massively parallel reporter assays (MPRAs) are one such tool that can be invaluable in functionally screening regulatory elements and identifying allelic effects. Since enhancers integrate and transduce environmental signals to execute gene expression programs, studying the impact of genetic variants under diverse conditions will be crucial for furthering our understanding of disease-associated variants. Efforts towards more robust, accurate and efficient MPRAs have been ongoing in the lab that enable correcting for PCR duplicates by introducing unique molecular identifier sequences (UMIs), consider effects of different promoter sequences in the assay and also their placement relative to the tested sequence among other advantages.


\subsection{Concluding remarks}
The results presented in this dissertation demonstrate that integrating information from the epigenome, transcriptome and other diverse molecular domains can help understand how complex disease predisposition is encoded in the (mostly non-coding) part of our genome. Moving forward, obtaining molecular profiles in different environmental contexts and probing genetic associations, followed by computational integration with emerging large-scale GWAS data could help partition swathes of GWAS signals into coherent, tissue-specific subsets to shed light on underlying pathophysiologies. Technological advancements will propel the field forward, for example, further reduction in experimental and sequencing cost could enable increased sample sizes in study design; development of more efficient and robust single cell experimental and analysis tools could supplement biological understanding at higher resolution; more exhaustive trans-ethnic studies will enable higher power for signal discovery and delineate a more complete picture of the genetic underpinnings of disease traits. Collectively, such approaches could help reveal additional convergent functional contexts, which could eventually enable higher-resolution patient stratification and determination of individualised risk.