\section{Abstract}
Genome-wide association studies (GWAS) have identified $>$400 independent single nucleotide polymorphisms (SNPs) that modulate the risk of type 2 diabetes (T2D) and related traits. However, the pathogenic mechanisms of most of these SNPs remain elusive. Here, we examined genomic, epigenomic, and transcriptomic profiles in human pancreatic islets to understand the links between genetic variation, chromatin landscape, and gene expression in the context of T2D. We first integrated genome and transcriptome variation across 112 islet samples to produce dense cis-expression quantitative trait loci (cis-eQTL) maps. Further integration with chromatin state maps for islets and other diverse tissue types revealed that cis-eQTLs for islet specific genes are specifically and significantly enriched in islet stretch enhancers (SEs). High resolution chromatin accessibility profiling using ATAC-seq in two islet samples enabled us to identify specific transcription factor (TF) footprints embedded in active regulatory elements, which are highly enriched for islet cis-eQTL. Aggregate allelic bias signatures in TF footprints enabled us de novo to reconstruct TF binding affinities genetically, which support the high-quality nature of the TF footprint predictions. Interestingly, we found that T2D GWAS loci were strikingly and specifically enriched in islet Regulatory Factor X (RFX) footprints. Remarkably, within and across independent loci, T2D risk alleles that overlap with RFX footprints uniformly disrupt the RFX motifs at high information content positions. Together, these results suggest that common regulatory variations have shaped islet TF footprints and the transcriptome, and that a confluent RFX regulatory grammar plays a significant role in the genetic component of T2D predisposition.

\section{Introduction}
Type 2 diabetes (T2D) is a complex disease characterized by pancreatic islet dysfunction and insulin resistance in peripheral tissues. $>$90\% of T2D SNPs identified through genome wide association studies (GWAS) reside in non-protein coding regions and are likely to perturb gene expression rather than alter protein function \cite{mohlkeRecentAdvancesUnderstanding2015}. In support of this, we and others recently showed that T2D GWAS SNPs are significantly enriched in enhancer elements that are specific to pancreatic islets \cite{parkerChromatinStretchEnhancer2013, pasqualiPancreaticIsletEnhancer2014, trynkaChromatinMarksIdentify2013}. The critical next steps to translate these islet enhancer T2D genetic associations into mechanistic biological knowledge are (a) identifying the putative functional SNP(s) from all those that are in tight linkage disequilibrium (LD), (b) localizing their target gene(s), and (c) understanding the direction-of-effect (increased or decreased target gene expression) conferred by the risk allele. Two recent studies performed integrative analyses of genome variation and gene expression variation across human islet samples to identify cis expression quantitative trait loci (cis-eQTL) that linked T2D GWAS SNPs to target genes \cite{fadistaGlobalGenomicTranscriptomic2014, buntTranscriptExpressionData2015}. However, the transcription factor (TF) molecular mediators of the islet cis-eQTL remain poorly understood and represent important links to upstream pathways that will help untangle the regulatory complexity of T2D. 

\section{Results}
\subsection{Integrated analysis of transcriptome and epigenome islet data}
To build links between SNP effects on regulatory element use and gene expression in islets, we performed strand-specific mRNA sequencing of 31 pancreatic islet tissue samples to an average depth of 100M paired end reads. In parallel, we analyzed unstranded mRNA-seq data for 81 islet samples from a previous study \cite{fadistaGlobalGenomicTranscriptomic2014}. We subjected both datasets to the same quality control and processing. We additionally completed dense genotyping of the 31 islet samples and downloaded genotypes for 81 previously described islet samples \cite{fadistaGlobalGenomicTranscriptomic2014}. Phasing and imputation yielded a final set of 6,060,203 autosomal SNPs (Methods) present in both datasets with an overall minor allele count (MAC) $>$10. To identify SNPs affecting gene expression within 1 Mb of the most upstream transcription start site (TSS), we performed separate cis-eQTL analyses for the two sets of islet samples, and combined the cis-eQTL results via meta-analysis. We identified 3,964 unique autosomal cis-eQTL lead SNPs for 3,993 genes at a 5\% false discovery rate (FDR). 

Next, we integrated chromatin immunoprecipitation followed by sequencing (ChIP-seq) data for 5 histone modifications across islets \cite{parkerChromatinStretchEnhancer2013, scottGeneticRegulatorySignature2016} and 30 diverse tissues with publicly available datasets (Table \ref{tbl:c2_t1}) \cite{theroadmapepigenomicsconsortiumIntegrativeAnalysis1112015, mikkelsenComparativeEpigenomicAnalysis2010, buenrostroTranspositionNativeChromatin2013} using ChromHMM \cite{ernstDiscoveryCharacterizationChromatin2010, ernstMappingAnalysisChromatin2011, ernstChromHMMAutomatingChromatin2012}. This produced 13 unique and recurrent chromatin states (Fig. \ref{fig:c2_f1}A, chromatin state tracks; Fig. \ref{fig:c2_sf1}) including promoter, enhancer, transcribed, and repressed regions, which were annotated after analyzing overlap enrichments with the chromatin states reported by the Roadmap Epigenome Consortium \cite{theroadmapepigenomicsconsortiumIntegrativeAnalysis1112015} for matching cell types. To identify specific regulatory element sites within these chromatin states, we profiled open chromatin in two islets using the assay for transposase-accessible chromatin-sequencing (ATAC-seq) \cite{pique-regiAccurateInferenceTranscription2011} (Fig. \ref{fig:c2_f1}A, ATAC-seq tracks). Our high-depth ATAC-seq data ($>$1.4B reads across both islets) allowed us to identify transcription factor (TF) DNA footprints using the CENTIPEDE algorithm (13) (Methods). We assigned regulatory state and TF footprint status to every islet cis-eQTL based on the annotation of SNPs with r2 $>$0.8 with the lead SNP (Fig. \ref{fig:c2_f1}B).  We used iterative conditional analyses \cite{scottGeneticRegulatorySignature2016} to identify 28 T2D and related quantitative trait GWAS SNPs that could be islet cis-eQTL signals (Fig. \ref{fig:c2_f1}C; Dataset S1: Conditional analysis results for GWAS SNP-gene pairs that were significant in the original eQTL analysis; Dataset S2: Epigenetic annotation for SNPs in r2 $\geq$ 0.8 with the GWAS variants included in Dataset S1). Given the modest eQTL signals at most of these loci, conditional analysis in larger islet samples cis-eQTL signals will likely change this list.

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{3_islet_eqtl/figures/mainFig1.png}
    \caption[Integrated genomic, epigenomic, and transcriptomic analyses of human pancreatic islets]{Integrated genomic, epigenomic, and transcriptomic analyses of human pancreatic islets. (A) An overview of diverse molecular profiling data types used in this study. Integrative molecular profiling (open chromatin, ATAC-seq; chromatin states; RNA-seq) highlights islet-specific signatures at the KCNK17 locus. (B) Plot of strength of association (y axis) for significant islet cis-eQTLs colored by chromatin-state annotation (A) by chromosomal location (x axis); diamonds indicate SNPs overlapping ATAC-seq footprints. An interactive version of this plot can be found at theparkerlab.org/tools/isleteqtl/. (C) Plot of strength of islet cis-eQTL association for T2D and related trait GWAS SNPs after conditional analysis to identify variants likely independent of stronger cis- eQTL signals for the same gene by chromosomal position and annotated as in B. The plot includes all GWAS SNP–gene pairs with FDR $<$ 0.05 in original cis- eQTL analysis. The dotted red line represents the P value threshold for FDR $<$ 0.05 based on the conditional analysis. (D) Islet cis-eQTL associated with KCNK17 expression highlighted for comparison with molecular profiling tracks in A. (E) Plot of normalized KCNK17 expression in islet samples and cis-eQTL risk allele dosage. (F) Functional validation of KCNK17 cis-eQTL at its promoter region. The haplotype containing alleles associated with T2D risk and increased KCNK17 expression (rs10947804-C, rs12663159-A, rs146060240-G, and rs34247110-A) shows higher transcriptional activity than the haplotype with nonrisk alleles. The cloned region is indicated at the top of A. Relative luciferase activity is given as mean ± SD of four to five independent clones per haplotype normalized to empty vector. Significance was evaluated using a two-sided t test.}
    \label{fig:c2_f1}
\end{figure}

\begin{table}
  \centering
  \includegraphics[width=\textwidth]{3_islet_eqtl/figures/table_nsc_rsc.png}
  \caption[\ac{NSC} and \ac{RSC} scores for H3K27ac and H3K4me3 datasets used in this study]{\ac{NSC} and \ac{RSC} scores for H3K27ac and H3K4me3 datasets used in this study}
  \label{tbl:c2_t1}
  
\end{table}

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{3_islet_eqtl/figures/sfig1.png}
    \caption[Thirteen-chromatin-state model]{Thirteen-chromatin-state model built from histone modification ChIP-seq data generated using ChromHMM \cite{ernstMappingAnalysisChromatin2011} for 33 cell types. (A) Each graph represents the overlap enrichment for 18 cell types of each of our 13 generated chromatin states with the Roadmap Epigenomics \cite{theroadmapepigenomicsconsortiumIntegrativeAnalysis1112015} reported states. (B) Renaming of generated 13 states (Original State) according to Roadmap Epigenomics overlap enrichments (New State) in A. (C) State numbers, histone mark emission probabilities, state names, and percentage genomic coverage of each chromatin state in human islets.}
    \label{fig:c2_sf1}
\end{figure}

As an example, T2D GWAS index SNP rs1535500 occurs at the KCNK16 locus and the risk allele results in a glutamate substitution at alanine 277 (A277E). This change was implicated in increasing the KCNK16 basal channel activity and cell surface localization when tested in a mouse model \cite{vierraTypeDiabetesAssociated2015}. Our analysis revealed that rs1535500 is not associated with KCNK16 expression (Fig. \ref{fig:c2_sf2}). Interestingly, the rs1535500 risk allele is associated with increased expression of the neighboring potassium channel gene KCNK17 (Fig. \ref{fig:c2_f1}D, E).  We observed that rs1535500 is in high linkage disequilibrium (LD) (r2$>$0.95) with four SNPs (rs10947804, rs12663159, rs146060240 and rs34247110) that are located in an islet promoter chromatin state; all but rs34247110 are located in an ATAC-seq peak (Fig. \ref{fig:c2_f1}A).  Motivated by the overlap with islet regulatory annotations, we cloned two different copies of the 473 bp DNA sequence surrounding these SNPs, one containing the T2D risk alleles for each of the 4 SNPs (risk haplotype), and the other containing the non-risk alleles (non-risk haplotype). We performed luciferase reporter assays in the mouse insulinoma (MIN6) beta cell line to test the transcriptional activity of these two clones. Both clones exhibited promoter activity (~10-fold increased compared to empty vector) but the T2D risk haplotype showed significantly greater (24\%, p=0.03) transcriptional activity than the non-risk haplotype (Fig. \ref{fig:c2_f1}F). This suggests that one or more of these T2D risk variants cause increased regulatory activity in islets. We note these findings highlight a complex functional genetic architecture for a single haplotype that results in regulatory activity linked to one gene (KCNK17) and coding variation in another (KCNK16). Together, these results illustrate how integrated analyses help to identify potential causal SNPs associated with islet expression and T2D risk. To enable easy, in-depth exploration of our integrated genome-wide results, we created an interactive islet cis-eQTL and chromatin state browser (\url{http://theparkerlab.org/tools/isleteqtl/}).

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{3_islet_eqtl/figures/sfig2.png}
    \caption[The \textit{KCNK16} genomic locus]{(A) LocusZoom plot showing that a T2D GWAS SNP (\url{rs1535500/chr6:39284050}, hg19, purple; other variants in LD colored according to r2) is not associated with KCNK16 expression in islets. (B) Plot for normalized KCNK16 expression and rs1535500 risk allele dosage from mRNA-seq and genotyping data in islet samples.}
    \label{fig:c2_sf2}
\end{figure}

\subsection{Common and islet-specific gene eQTLs are enriched in different chromatin states} 
To understand the regulatory architecture of islet eQTLs, we measured their co-occurrence with different classes of chromatin states across diverse tissues, including stretch enhancers (SEs), defined as enhancer chromatin states $\geq$3kb long. These tend to mark cell identity regions and have been shown to harbor tissue-specific GWAS SNPs \cite{parkerChromatinStretchEnhancer2013, quangMotifSignaturesStretch2015}. We calculated genome wide enrichment for cis-eQTL overlaps with these features while controlling for minor allele frequency, distance to TSS, and the number of SNPs in LD \cite{schmidtGREGOREvaluatingGlobal2015}. cis-eQTLs were enriched in active chromatin states such as promoter, and genic enhancer in islets, while inactive states such as polycomb repressed were depleted for such overlaps across multiple tissues (Fig. \ref{fig:c2_sf3}). Reasoning that this common enrichment pattern across diverse tissues may be largely driven by cis-eQTLs of commonly expressed housekeeping genes, we sought to classify cis-eQTLs by the islet expression specificity of their associated genes. To measure gene expression specificity in islets, we analyzed RNA-seq data from 16 additional tissues from the Illumina Human Body Map 2.0 project. We used an information theory approach to define the islet expression specificity index (iESI, \ref{fig:c2_sf4}, also see Methods) as in our previous skeletal muscle study \cite{scottGeneticRegulatorySignature2016}. iESI values near zero represent lowly and/or ubiquitously expressed genes whereas values near one represent genes that are highly and specifically expressed in islets. We divided genes into quintiles based on the ascending iESI (\ref{fig:c2_sf4}). We assigned eQTLs for these genes to their respective iESI quintile and estimated enrichment of each set in chromatin annotations. Interestingly, while eQTLs across iESI quintile bins were similarly enriched in islet promoter states, eQTL enrichment in active and SE states increased concomitantly with the islet specificity (\ref{fig:c2_sf5}). As an example, we found that the cis-eQTL for the KCNA6 gene (\ref{fig:c2_sf6}A) which is expressed in islets with high specificity (KCNA6 iESI = 0.78), overlapped islet-specific enhancer states (\ref{fig:c2_sf6}B). We note that this cis-eQTL locus does not overlap a known T2D GWAS locus. When we restricted our enrichment analysis to ATAC-seq peaks in SE states, we saw a stronger trend toward increasing enrichment by iESI quintile (\ref{fig:c2_sf5}). These results indicate a strong link between active regulatory chromatin architecture and the genetic control of cell-specific gene expression.

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{3_islet_eqtl/figures/sfig3.png}
    \caption{Fold enrichment of islet eQTLs in chromatin states across cells/tissues.}
    \label{fig:c2_sf3}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{3_islet_eqtl/figures/sfig4.png}
    \caption[iESI]{iESI. (A) Heat map showing mean FPKM for genes expressed in different tissues when binned by iESI quintiles. (B) Scatterplots showing FPKM for genes expressed in different tissues vs. the iESI. (C) Distribution of iESI by quintile of expression.}
    \label{fig:c2_sf4}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.6\textwidth]{3_islet_eqtl/figures/sfig5.png}
    \caption[Enrichment of islet cis-eQTLs based on the expression specificity of the target gene]{Enrichment of islet cis-eQTLs binned into quintiles by target gene iESI in islet active TSS and stretch enhancer chromatin states (red) and consensus islet intersect ATAC-seq peaks (present in both islet samples) in these states (blue). $\ast$ P $<$ 0.05 from GREGOR analysis.}
    \label{fig:c2_sf5}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{3_islet_eqtl/figures/sfig6.png}
    \caption[Common and islet-specific gene eQTLs are enriched in different chromatin states]{Common and islet-specific gene eQTLs are enriched in different chromatin states. (A) LocusZoom plot of an islet cis-eQTL in the KCNA6 locus. (B) The cis-eQTL for KCNA6, which is in the top quintile of the iESI (iESI 5), overlaps an islet-specific enhancer state. (C) Active enhancer clustering (y axis) across cell types (x axis) reveals cell-specific enhancer regions. Cluster 13 is islet-specific. (D) Degree of overlap of enhancer clusters with stretch enhancers from four cell types. Islet stretch enhancers show the strongest overlap with islet-specific enhancer cluster 13, whereas GM12878 stretch enhancers show the strongest overlap with GM12878- specific enhancer cluster 1. The Jaccard statistic was normalized per column, so that values range from zero (no overlap) to one (maximum observed overlap). (E) Enrichment of islet eQTLs across enhancer clusters reveals that the full set of eQTLs (column 1) is enriched across multiple enhancer clusters, whereas eQTLs for islet-specific genes (iESI quintile 5; column 5) are enriched in the islet-specific enhancer cluster 13. Gray bars indicate nonsignificant after Bonferroni correction.}
    \label{fig:c2_sf6}
\end{figure}

To further identify and dissect regulatory regions critical for islet-specific gene expression, we sought to distinguish between shared and tissue specific enhancer chromatin states. We performed k-means clustering for active enhancer chromatin states across 31 cells/tissues (Methods). This method segregated enhancer regions based on activity across diverse tissues; for example, cluster 13 is islet specific, while cluster 3 is liver specific (Fig. \ref{fig:c2_sf6}C). We compared these enhancer clusters with SE annotations across tissues and found that tissue specific clusters, such as the islet-specific cluster 13, indeed displayed high enrichment for islet SEs (Fig. \ref{fig:c2_sf6}D). Likewise, in other tissues, tissue-specific enhancer clusters were enriched for the corresponding tissues’ SEs (Fig. \ref{fig:c2_sf6}D). Next, we asked if islet cis-eQTL were enriched in specific enhancer clusters and observed enrichment in multiple clusters (Fig. \ref{fig:c2_sf6}E). We then stratified the cis-eQTLs by iESI quintile and repeated this analysis. Notably, islet cis-eQTLs for genes in iESI quintile 5 only showed significant enrichment in the islet specific enhancer cluster 13 (p-value = 1.2$\times$10\textsuperscript{-8}, fold enrichment = 1.91, Fig. \ref{fig:c2_sf6}E). Together, these results demonstrate that islet tissue-specific genetic regulatory architecture is enriched in islet-specific enhancers and SEs.

\subsection{Islet eQTL are enriched in islet ATAC-seq peaks and DNA footprints}    
Chromatin state maps identify regulatory regions such as promoters and enhancers but lack the resolution to pinpoint specific sites that may be bound and regulated by a TF. To refine the link between genetic variation, TF binding sites, and gene expression, we leveraged the high-resolution ATAC-seq data to identify in vivo putative TF binding sites. We employed CENTIPEDE as previously described to predict whether TF motif occurrences in ATAC-seq peaks are bound or unbound by the corresponding TF \cite{scottGeneticRegulatorySignature2016, pique-regiAccurateInferenceTranscription2011}. This approach detected high-quality footprints for many TFs, including the general CCCTC-binding factor (CTCF) and the more islet specific TF Regulatory Factor X (RFX) (Fig. \ref{fig:c2_f2}A, B). Fig. \ref{fig:c2_f2}A depicts the integrated chromatin and transcriptional landscape at RFX6, a gene that exhibits islet-specific expression (iESI = 0.94, iESI quintile bin = 5). Notably, we detect RFX footprints in islet SEs near RFX6 (Fig. \ref{fig:c2_f2}A), suggesting an autoregulatory mechanism that, based on recent studies \cite{pasqualiPancreaticIsletEnhancer2014, saint-andreModelsHumanCore2016}, may indicate RFX6 is an islet core transcriptional regulatory gene. We detected periodic, nucleosome-sized spikes in the ATAC-seq signal adjacent to the CTCF footprint regions (Fig. \ref{fig:c2_f2}B), consistent with its reported nucleosome-phasing properties \cite{fuInsulatorBindingProtein2008}. Comparing ATAC-seq profiles from islets to those of skeletal muscle tissue \cite{scottGeneticRegulatorySignature2016}, adipose tissue \cite{allumCharacterizationFunctionalMethylomes2015} and a lymphoblastoid cell line (GM12878) \cite{buenrostroTranspositionNativeChromatin2013}, we found that islet ATAC-seq peaks occurred preferentially in islet promoter and enhancer chromatin states (Fig. \ref{fig:c2_sf7}). Islet cis-eQTLs were highly enriched in multiple TF footprint motifs (p-value range: 5.1$\times$10\textsuperscript{-5} - 5.0$\times$10\textsuperscript{-23}; fold enrichment range: 2.02-6.67) but not in non-footprint motifs (Fig. \ref{fig:c2_f2}C, Dataset S3, Methods). These results suggest a strong link between SNPs at TF binding sites in relevant tissues and gene regulation.

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{3_islet_eqtl/figures/mainFig2.png}
    \caption[Nucleotide resolution islet ATAC-seq profiling nominates regulatory mechanisms]{Nucleotide resolution islet ATAC-seq profiling nominates regulatory mechanisms. (A) RFX6 locus with expression (RNA-seq), chromatin states, open chromatin (ATAC-seq), and footprints for CTCF and RFX in islets. (B) Density plots indicating normalized sequence coverage of ATAC-seq from two human islet samples at sites overlapping CTCF (motif = CTCF\_known2) and RFX (motif = RFX2\_4) motifs. (C) Log twofold enrichment of islet cis-eQTLs in TF footprint motifs compared with their enrichment in TF nonfootprint motifs. TFs for which footprint and nonfootprint motifs overlap four or more eQTL SNPs are shown. Blue shows significant enrichment in footprints only (Bonferroni corrected P $<$ 0.05). No significant enrichment was observed in any TF nonfootprint motif. (D) Reconstruction of CTCF (motif = CTCF\_known2) and RFX (motif = RFX2\_4) motifs using ATAC-seq TF footprint allelic bias data. Row 1: original motif PWM. Row 2: PWM genetically reconstructed using the overrepresented alleles (and extent of overrepresentation) for SNPs with significant ATAC-seq allelic bias. Row 3: count of nucleotides in SNPs with significant allelic bias. Row 4: PWM reconstructed using the count of nucleotides for heterozygous SNPs in the TF footprint. Row 5: count of nucleotides in heterozygous SNPs in the TF footprint.}
    \label{fig:c2_f2}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{3_islet_eqtl/figures/sfig7.png}
    \caption[Enrichment of islet, muscle, GM12878, and adipose ATAC-seq peaks in chromatin states across diverse tissues]{Enrichment of islet, muscle, GM12878, and adipose ATAC-seq peaks (columns) in chromatin states across diverse tissues (y axis). Consensus (islet intersection) and individual (islets 1 and 2) islet ATAC-seq peaks show enrichment for active chromatin states in islets, which is more pronounced at TSS-distal ($>$5 kb from TSS) regions. Muscle (column 4), GM12878 (column 5), and adipose (column 6) ATAC-seq peak calls show similar trends with chromatin states from matched tissues. Note that TSS-distal ATAC-seq peaks from the islet intersect dataset overlap islet active enhancers more than any other chromatin state in islets. Note also that the level of islet enhancer overlap is larger than enhancer overlap in any other tissue.}
    \label{fig:c2_sf7}
\end{figure}

To detect motif occurrences that could be altered by the presence of alleles not in the reference genome, we developed a personalized phased SNP-aware genome motif scanning procedure to determine where a motif occurs (Methods). This allowed us to identify motif instances even when multiple non-reference alleles occur within a few base pairs of each other. We observed significant enrichment for islet cis-eQTLs in the set of TF footprint motifs identified only from this haplotype phase-aware scanning approach (that is, the motifs are missed even when a single SNP-aware motif scanning approach is used) in islet samples (p-values: Islet1=4.6$\times$10\textsuperscript{-5}, Islet2=4.0$\times$10\textsuperscript{-7}; Islet-intersect=0.0014; \ref{fig:c2_sf8}). Given the informative chromatin accessibility allelic analyses in recent studies (20, 21), we next asked if we could recreate known TF position weight matrices (PWMs) (Fig \ref{fig:c2_f2}D, row 1) based on the allele-specific bias at heterozygous SNPs within TF footprint motifs. To do this, we identified every heterozygous site in a given TF footprint motif, calculated the allelic bias in ATAC-seq signal at these positions, and retained all SNPs with significant bias (Fig. \ref{fig:c2_f2}D, row 3; Methods). We constructed the PWM using the degree of allelic bias for the over-represented alleles (Fig. 2D, row 2). This allelic bias-based PWM (Fig. \ref{fig:c2_f2}D row 2) closely matched the canonical PWM for the corresponding TF (Fig. \ref{fig:c2_f2}D, row 1), providing an in vivo verification of the cognate PWM. We consider this a genetically reconstructed PWM. There was a larger difference in the PWM score for the two alleles of allelic bias SNPs than for the two alleles of matched 1000G SNPs occurring in the same motif (RFX p-value=0.018; CTCF p-value=0.023) (\ref{fig:c2_sf9}). To further verify that the allelic bias-based genetically reconstructed PWMs were not simply reflecting the allelic composition of SNPs in the motifs, we constructed the PWMs using the allele count for all TF footprint heterozygous SNPs observed at each position (where each observed SNP contributed two alleles) and found the resulting PWMs had little information and little similarity to the cognate motifs used to scan across the genome (Fig. \ref{fig:c2_f2}D, rows 4 and 5). Collectively, these results reinforce the potential of ATAC-seq and allelic footprinting analyses to identify relevant and potentially causal TF binding changes in the genetic control of gene expression.

\begin{figure}
    \centering
    \includegraphics[width=0.5\textwidth]{3_islet_eqtl/figures/sfig8.png}
    \caption[Enrichment of islet cis-eQTLs (5\% FDR) in ATAC-seq TF footprints that are only detected using phased SNP-aware scans]{Enrichment of islet cis-eQTLs (5\% FDR) in ATAC-seq TF footprints that are only detected using phased SNP-aware scans (Materials and Methods). $\ast$ P $<$0.05 from GREGOR analysis.}
    \label{fig:c2_sf8}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{3_islet_eqtl/figures/sfig9.png}
    \caption[SNPs that show allelic bias in ATAC-seq data]{SNPs that show allelic bias in ATAC-seq data (ab; blue box plot) exhibit larger effects on the predicted TF binding site motifs compared with randomly sampled 1000G SNPs (1000G; red box plot) overlapping the same footprint in islet 1. The y axis shows absolute value of the delta score [delta = −log\textsubscript{10}(FIMO P value of alternate sequence) − (−log\textsubscript{10}(FIMO P value of reference sequence))]. P values of the comparisons were determined by the Wilcoxon rank sum of test. (A) Footprints motif = RFX2\_4. (B) Footprint motif = CTCF\_known2.}
    \label{fig:c2_sf9}
\end{figure}

\subsection{T2D GWAS loci are enriched in RFX footprints and T2D risk alleles disrupt the motifs at independent locations}
Given the strong enrichment for islet cis-eQTL in diverse TF footprints, we next sought to identify T2D GWAS SNPs that might regulate gene expression by modulating TF binding. We found that T2D-associated SNPs were significantly enriched in islet RFX TF footprints (p-value range 1.5$\times$10\textsuperscript{-5} - 8.3$\times$10-9 for five RFX TF family members fold enrichment range 7.46 - 30.06 (Fig. \ref{fig:c2_f3}A, Dataset S4)). In contrast, we did not see significant enrichment of T2D associated SNPs in islet non-footprint RFX TF motifs (Methods) or in GM12878 TF footprints (Fig. \ref{fig:c2_f3}A). The RFX family of transcription factors recognize X-box motifs and have highly evolutionarily conserved DNA binding domains (22), which may explain why similar motifs from many RFX family members are enriched. Gaulton et al. (2015) observed enrichment of T2D GWAS SNPs in FOXA2 ChIP-seq binding sites in islet tissues (23). We observed 13.5 fold enrichment of T2D-associated SNPs in islet FOX TF footprints (p=2.8$\times$10\textsuperscript{-5}, slightly less significant than the Bonferroni threshold of 2.5$\times10$\textsuperscript{-5}, Dataset S4). 

Studies of autoimmune disease have found that disease associated variants often occur near but not in TF motifs (24). We therefore asked if T2D-associated SNPs were enriched in regions flanking RFX footprints motifs (excluding footprint itself) (n=22). We found that regions flanking RFX footprint motifs were enriched for T2D associated SNPs and that the enrichment decreased with increasing distance from footprint motif (Fig. \ref{fig:c2_sf10}). The flanking enrichment was lower than in the RFX TF footprints. In contrast, we did not see enrichment of T2D associated SNPs in, non-footprint RFX TF motifs or in the regions flanking the non-footprint RFX TF motifs (Fig. \ref{fig:c2_sf10}).

We next assessed the potential effects of the risk and non-risk alleles for 9 T2D associated SNPs at 5 independent loci on RFX TF binding (Methods) (Fig. \ref{fig:c2_f3}B). For each SNP, the non-risk allele was the highest probability nucleotide in the RFX position weight matrix (PWM), and thus the risk allele was predicted to disrupt the motif (Fig. \ref{fig:c2_f3}B and C black boxes). At two of the five loci, the T2D GWAS risk alleles were associated with significantly increased gene expression in our conditional eQTL analysis: KCNK17 (KCNK16 locus, Fig. \ref{fig:c2_f1}B, C, E) and ABCB9 (PITPNM2 locus, Fig. \ref{fig:c2_f1}C). Other loci might have not been detectable as eQTLs due to state specific regulation or small effect sizes. The observation that T2D risk alleles at multiple loci confluently disrupt RFX motifs provides a hypothesis that could explain the mechanism of a subset of T2D associated variants.

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{3_islet_eqtl/figures/mainFig3.png}
    \caption[T2D GWAS enrichment at islet footprints reveals confluent RFX motif disruption]{T2D GWAS enrichment at islet footprints reveals confluent RFX motif disruption. (A) T2D GWAS SNPs are significantly enriched in RFX motifs in islet footprints but not in control motifs or footprints from a nondisease-relevant cell type (GM12878). TF motifs for which footprints overlap four or more T2D GWAS SNPs are shown. The red line indicates Bonferroni multiple testing threshold. (B) T2D-associated SNPs that overlap high information content ($>$1 bit) positions in RFX motifs. The highest scoring RFX footprints are reported for each T2D GWAS SNP. Act. Enh., active enhancer; Act. TSS, active TSS; Wk. Transc., weak transcribed. $\ast$ Chromatin-state annotation overlapping the SNP. $\dagger$ Because RFX motifs in C are organized by alignment to the longest RFX3\_1 motif, motifs overlapping rs10947804 and rs1716165 correspond to the reverse complement sequence. Therefore, risk and nonrisk alleles are also reported as reverse complement relative to the plus strand sequence. (C) Alignment of highest scoring RFX footprint at each SNP; the boxes indicate the SNP overlap positions. Note that, in every case, the risk allele disrupts that motif.}
    \label{fig:c2_f3}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=0.8\textwidth]{3_islet_eqtl/figures/sfig10.png}
    \caption{Enrichment for T2D GWAS SNPs in regions flanking merged RFX footprint (red) and nonfootprint (blue) motifs.}
    \label{fig:c2_sf10}
  \end{figure}
  
\section{Discussion}
We have integrated genome, epigenome, and transcriptome variation and created maps to better understand the genetic control of islet gene expression. Comparison of these maps with T2D GWAS SNPs has helped identify potential disease mechanisms. For example, the risk allele of the coding SNP rs1535500 has been implicated to increase KCNK16 activity and cell surface localization in a mouse model \cite{vierraTypeDiabetesAssociated2015}. Other risk alleles in SNPs in high LD with rs153550 are associated with increased expression of the neighboring potassium channel gene KCNK17. KCNK16 and KCNK17 are two pore domain ‘background’ K+ channels, members of the TWIK related alkaline pH activated K+ channel (TALK) family \cite{girardGenomicFunctionalCharacteristics2001, lotshawBiophysicalPharmacologicalFunctional2007a}. Both genes are expressed in islets with high specificity (KCNK16 iESI = 0.98; KCNK17 iESI = 0.76). KCNK16 has been implicated in regulating electrical excitability and glucose stimulated insulin secretion (GSIS)  \cite{vierraTypeDiabetesAssociated2015}. The KCNK17 gene is not present in the mouse genome. It is possible that the T2D risk haplotype at this locus may have multiple effects that collectively disrupt beta cell K+ signaling and glucose stimulated insulin secretion by simultaneously over-activating KCNK16 and over-expressing KCNK17.

We find that T2D GWAS associated SNPs are significantly enriched in RFX TF footprint motifs and are almost significantly enriched in FOX TF footprint motifs. We find consistent disruption of islet RFX footprint motifs by T2D risk alleles, including at the KCNK17 locus. Lizio et al. found that knockdown of RFX6 results in increased expression of KCNK17 \cite{lizioMappingMammalianCelltypespecific2015} which is consistent with the T2D risk allele disrupting TF binding and increasing target gene expression. At other T2D GWAS loci, such as the MPHOSPH9 locus, (index SNP rs1727313), two or three T2D GWAS SNPs in high LD are each predicted to have risk alleles that coordinately disrupt independent RFX footprint motifs (Fig. \ref{fig:c2_f3}B, C).  We and others \cite{parkerChromatinStretchEnhancer2013, corradinCombinatorialEffectsMultiple2014, guoCoordinatedRegulatoryVariation2015} previously described the presence of multiple SNPs in enhancers at individual GWAS loci. Our current results build on this concept to include the possibility of multiple confluent disruptions of similar TF motifs in the same locus. Collectively, these results indicate that T2D risk may in part be propagated through genetic modulation of RFX binding in islets. Indeed, our study shortlists only a subset of T2D associated variants as candidates which should be functionally dissected in vivo.

Among the RFX TFs, RFX6 is expressed in islets with high specificity (iESI = 0.94; \ref{fig:c2_sf11}) and is involved in pancreatic progenitor specification, endocrine cell differentiation, maintaining beta cell functional identity, and controlling glucose homeostasis \cite{zhuGenomeEditingLineage2016, smithRfx6DirectsIslet2010, soyerRfx6Ngn3dependentWinged2010}. Beta cell-specific deletion of RFX6 results in impaired insulin secretion \cite{piccandRfx6MaintainsFunctional2014a, chandraRFX6RegulatesInsulin2014}. Individuals that are heterozygous for a frameshift mutation in RFX6 have increased 2-hour glucose levels \cite{huopioClinicalGeneticBiochemical2016}. Importantly, rare autosomal recessive mutations that alter DNA-contacting amino acids in the DNA binding domain of RFX6 result in Mitchell-Riley syndrome, which is characterized by neonatal diabetes \cite{smithRfx6DirectsIslet2010}. Although RFX6 was not in our motif database, a recent report found it to be highly similar to the other RFX family motifs \cite{lizioMappingMammalianCelltypespecific2015}, consistent with the expectation for highly conserved DNA binding domains \cite{aftabIdentificationCharacterizationNovel2008}. Our findings could represent a novel connection between rare coding variation in the islet master TF RFX6 \cite{soyerRfx6Ngn3dependentWinged2010, piccandRfx6MaintainsFunctional2014a} and common non-coding variation in multiple target sites for this TF. The impact of these variations mirror the expected effect on organismal physiology, with coding variants that result in neonatal diabetes and non-coding variants that result in later onset T2D. This study for the first time implicates impaired RFX-dependent transcriptional responses in genetic susceptibility to T2D and nominates new mechanistic hypotheses about the molecular genetic pathogenesis of this complex disease. Following up on the reported loci to functionally validate this hypothesis could help in better understanding T2D mechanisms. Given that most other GWAS SNPs are non-coding, this approach could be used to identify other master TF and multiple target site relationships.

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{3_islet_eqtl/figures/sfig11.png}
    \caption[\textit{RFX} gene expression (FPKM) across islets and 16 Illumina Body map 2.0 tissues]{\textit{RFX} gene expression (FPKM) across islets and 16 Illumina Body map 2.0 tissues. The iESI quintile for each \textit{RFX} gene is labeled in the islet columns. RFX6 has the highest iESI (0.94) among all \textit{RFX} TF genes.}
    \label{fig:c2_sf11}
\end{figure}


\section{Materials and Methods}
Islet procurement and processing 
We procured the islet samples used in this study from the Integrated Islet Distribution Program (IIDP), the National Disease Research Interchange (NDRI), or ProdoLabs. 31 islets (age=45.3 +/- 11.9 years; 52\% male; BMI=27.2+/-4.3 kg/m2) passed mRNA-seq and genotype QC steps (see below). Islets were shipped overnight from the distribution centers.  Upon receipt, we pre-warmed islets to 37 C in shipping media for 1-2 hours prior to harvest.  Approximately 2500-5000 islet equivalents (IEQ) from each organ donor were harvested for RNA isolation. We transferred 500-1000 IEQ to tissue culture-treated flasks and cultured as in (1); genomic DNA isolated from islet explant cultures was used for genotyping.

\subsection{SNP genotyping, sample and genotype QC} 
Genomic DNA was genotyped at the Genetic Resources Core Facility (GRCF) of the Johns Hopkins Institute of Genetic Medicine on the HumanOmni2.5-4v1\_H BeadChip array (Illumina, San Diego, CA, USA): minimum call rate was 97.14\%. We mapped the Illumina array probe sequences to the hg19 genome assembly using BWA. We excluded SNPs with ambiguous probe alignments, SNPs with 1000 Genomes phase 1 variants with minor allele frequency of $\geq$ 1\% or greater within 7bp of the 3' end of probes, or call rates $<$95\%. All alleles were oriented relative to the reference. 

We identified no individuals with $\geq$ 3rd degree relatedness using KING \cite{manichaikulRobustRelationshipInference2010}. We performed principal components analysis using PLINK 1.9(\url{http://www.cog-genomics.org/plink2/general\_usage}) on 60,714 SNPs with minor allele count (MAC) $>$5 and r2 $<$0.2, after excluding SNPs from regions of high LD \cite{priceLongRangeLDCan2008}. 33 self-reported Caucasian samples and one sample of unknown ethnicity clustered together by principal components analysis (PCA). One sample self-reported to be of Caucasian ancestry did not cluster with the others and was excluded for eQTL analyses. 

Fadista et al. graciously provided genotypes of their 89 islet samples for the Illumina HumanOmniExpress 12v1C BeadChip \cite{fadistaGlobalGenomicTranscriptomic2014}. We processed the probes and genotypes as described above.  We identified no individuals with $\geq$ 3rd degree relatedness. We performed PCA as described above using 86,502 LD pruned SNPs. All 89 samples clustered together in the PCA analysis.

\subsection{RNA isolation, mRNA-seq library preparation and mRNA sequencing} 
We extracted and purified total RNA from 2000-3000 islet equivalents (IEQ) using Trizol (Life Technologies). RNA quality was confirmed with Bioanalyzer 2100 (Agilent); samples with RNA integrity number (RIN) $>$6.5 were prepared for mRNA sequencing. We added \ac{ERCC} spike-in controls (Life Technologies) to one microgram of total RNA.  We generated PolyA+, stranded mRNA RNA-sequencing libraries for each islet using the TruSeq stranded mRNA kit according to manufacturer’s protocol (Illumina).  Each islet RNA-seq library was barcoded, pooled into 12-sample batches, and sequenced over multiple lanes of HiSeq 2000 to obtain an average depth of 100 million 2 x 101 bp sequences.

\subsection{mRNA-seq processing and QC}
We retained RNA-seq reads passing the Illumina chastity filter and mapped reads to a reference sequence composed of \ac{ERCC} control fragments and all chromosomes and contigs from hg19, excluding alternate haplotypes, replacing the mitochondrial sequence (chrM) with the Cambridge Reference Sequence, and masking the pseudoautosomal region on chromosome Y. We aligned reads using STAR (version 2.3.1y) \cite{dobinSTARUltrafastUniversal2013a} with default parameters and a splice junction catalog based on Gencode v19 \cite{harrowGENCODEReferenceHuman2012}. Non-uniquely mapping reads and read pairs with unpaired alignments were discarded. Duplicate read pairs, i.e. those mapping to the same coordinates, were retained. 

RNA-seq QC was performed at the level of readgroups (i.e. a library on a lane) using QoRTs \cite{hartleyQoRTsComprehensiveToolset2015a}. We inspected the comprehensive set of QC metrics generated by QoRTs for outlying libraries, lanes, and sequencing runs. We used the 92 ERCC RNA spike-in controls and in-house scripts to assess library quality and batch effects, and to check the accuracy of the strand-specific protocol \cite{jiangSyntheticSpikeinStandards2011}. We also performed a principal components analysis (PCA) on the matrix of expression data. The QoRTs and PCA processes revealed two outlying sample libraries.  One sample showed extreme 3’ bias in gene body coverage, and the other showed low gene diversity and was a strong outlier by PCA. In addition, we excluded one sample that was reportedly Caucasian, but was an outlier in the genotype PCA (see above). These three libraries were removed, leaving 31 islet samples for analysis.

To confirm sample identity and check for contamination, we compared SNP chip genotypes to RNA-seq alignments in annotated exonic regions using verifyBamID \cite{junDetectingEstimatingContamination2012}, using the --maxDepth 100 option to avoid having highly-expressed genes bias the estimate of contamination. No sample showed contamination $>$0.78\%.

We aligned the non-strand-specific RNA-seq reads from Fadista et al. \cite{fadistaGlobalGenomicTranscriptomic2014} with the same version of STAR to the same hybrid reference genome. Again, we discarded non-uniquely mapped reads and read pairs with unpaired alignment, and we retained duplicate pairs. We performed QC using QoRTs, and PCA of the expression data, as described above, and identified one outlier library.  In addition, comparison of SNP chip genotypes to RNA-seq alignments with verifyBamID identified two swapped samples, and five samples that had greater than 2\% estimated contamination in the RNA-Seq sample. We removed all eight samples, leaving 81 samples to be analyzed.

\subsection{Expression quantification}
To study regulatory variation, we performed analyses at the gene level. Definitions for all transcriptome features were based on GENCODE v19 \cite{harrowGENCODEReferenceHuman2012}, which annotates a total of 57,820 genes: 20,345 protein-coding, 13,870 long non-coding RNAs, and 14,206 pseudogenes. We ignored pseudogenes for all downstream analyses. We counted fragments mapping to genes using htseq-count v0.5.4 \cite{andersHTSeqPythonFramework2015} (\url{http://www-huber.embl.de/users/anders/HTSeq/doc/count.html}) and calculated FPKM values for each gene. 

We processed data from Fadista et al. in the same way, except that counts of genes were performed in a non-strand-specific manner, consistent with the RNA-seq libraries.

\subsection{Imputation} 
We excluded SNPs with: MAC $<$1, Hardy-Weinberg equilibrium p-value $<$ 10\textsuperscript{-6}, absolute alternate allele frequency difference $>$0.2 compared to the 1000G EUR sample and A/T or C/G SNPs with MAF $>$0.2.  This left 2,057,703 autosomal SNPs for subsequent imputation. We performed autosomal SNP imputation using a two-step strategy \cite{howieFastAccurateGenotype2012} with the haplotypes from 1000G phase3 v5 \cite{the1000genomesprojectconsortiumGlobalReferenceHuman2015a} as the reference panel. To improve phasing quality given the small number of islet samples, we pre-phased our islet samples together with the 2,504 reference panel samples using ShapeIT version 2 \cite{delaneauImprovedWholechromosomePhasing2013}. We then imputed genotypes with Minimac2 \cite{fuchsbergerMinimac2FasterGenotype2015}. We retained 8,377,422 imputed variants with a MAC $\geq$ 1 and r2 $\geq$ 0.3.

For the 81 Fadista et al. islet samples, we removed SNPs, and prephased and imputed genotypes as described above. We used 692,118 SNPs for imputation. We retained 9,758,857 imputed variants with a MAC $\geq$ 1 and r2 $\geq$ 0.3.

\subsection{cis-eQTL meta-analysis}
We performed separate cis expression quantitative trait (cis-eQTL) analysis for our islets (n=31) and Fadista et al. islets (n=81) and combined the results using meta-analysis. We performed PCA (using the same procedure described in the genotype QC section) separately on these two sets of samples.  We considered for analysis 6,060,203 SNPs that were present in both studies and had a combined MAC $\geq$ 10. We tested SNPs within 1 Mb of the most upstream TSS of each gene using Matrix eQTL \cite{shabalinMatrixEQTLUltra2012}. We included in the analysis 19,360 genes present in both sets of samples (out of 26,845 genes present in our islets and 19,650 present in Fadista et al. islets). For individual i and gene j, to generate the gene expression value Y\textsubscript{ij}, we inverse normalized FPKM\textsubscript{ji} for each gene j. We then performed factor analysis via PEER \cite{stegleBayesianFrameworkAccount2010, stegleUsingProbabilisticEstimation2012} on the inverse normalized FPKM (specifying from 1 to 60 factors to optimize the detection of cis-eQTLs (below) and including age, sex, the top 2 genotype-based principal components and and for our islet samples only experimental batch , as covariates in the model), and inverse normalized the resulting residuals. We used the linear regression model with an additive genetic effect: 

\[Y_{ij} = \alpha + \beta_{js}G_{is} + \epsilon_{ij} \]

where α is the intercept, G\textsubscript{is} is the imputed allele count for SNP s for individuals i, β\textsubscript{js} is the regression coefficient of the imputed allele count for SNP s on transformed gene expression Y\textsubscript{ij} and ε\textsubscript{ij} is a normally distributed error term with mean 0 and variance $\sigma$\textsuperscript{2}.
 
We used false discovery rate (FDR) \cite{storeyStatisticalSignificanceGenomewide2003} to account for multiple testing and considered as significant associations with FDR $\leq$ 5\%. We expect that removing technical and biological variation via PEER will increase power to detect cis-eQTLs \cite{scottGeneticRegulatorySignature2016}. For each study we report results using number of PEER factors that maximized the number of eQTLs on chromosome 20 at FDR $\leq$ 5\%: 30 for our islets and 32 for Fadista et al.

For each SNP-gene pair we combined the results from our islet samples and Fadista et al. using a sample-sized weighted meta-analysis \cite{willerMETALFastEfficient2010} and report p-values based on this analysis. In addition, we performed a fixed-effects inverse variance-weighted meta-analysis \cite{willerMETALFastEfficient2010} and report eQTL effect sizes from this analysis. We do not report p-values from this analysis as we found the p-values were consistently inflated. We present results for SNPs present in both studies (MAC$\geq$1) and with MAC $\geq$ 10 in the combined study.
 
Gene-based cis-eQTLs for GWAS variants for T2D and related traits
We compiled a list of 225 SNPs with p-value $<$5 x 10-8 in GWAS (GWAS SNPs) for T2D, fasting glucose, fasting glucose adjusted for BMI, fasting insulin, fasting insulin adjusted for BMI, 2-hr glucose, 2-hr glucose adjusted for BMI, and fasting proinsulin from the NHGRI GWAS catalog \cite{welterNHGRIGWASCatalog2014} and carried out manual curation of the literature to create a comprehensive list that was up-to-date as of May 2014. Of these 225 GWAS SNPs, 214 were tested in our cis-eQTL analysis for a total of 3,995 GWAS SNP-gene pairs. To identify GWAS variant cis-eQTLs that may be independent of other stronger cis-eQTLs for the same gene, we performed iterative conditional analysis on each of the 3,995 GWAS SNP-gene pairs. For each GWAS SNP-gene pair and study we used the linear regression model with an additive genetic effect:

\[Y_{ij} = \alpha + \beta_{jGWAS}G_{iGWAS} + \epsilon_{ij} \]


where G\textsubscript{iGWAS} is the imputed allele count for the GWAS SNP for individuals i, β\textsubscript{jGWAS}, is the regression coefficient of the imputed allele count for the GWAS SNP, and G\textsubscript{is} is the set of SNPs within 1Mb of the most upstream TSS. We combined the results from our samples and Fadista et al. islets using meta-analysis as described above.  If $\geq$ 1 SNP had a meta-analysis p-value $<$ 1.2 x 10-4 (corresponding to the p-value threshold for gene-based cis-eQTLs with FDR $<$5\%) we retained the SNP with the most significant p-value in the model and repeated the procedure until no added SNP had a p-value $<$1.2 $\times$ 10\textsuperscript{-4}. This procedure corresponds to performing step-wise forward selection of SNPs within 1Mb of the most upstream TSS based on the results of the meta-analysis at each step (using a stopping threshold p-value of 1.2 $\times$ 10\textsuperscript{-4}). The conditional p-value for a given GWAS SNP is the p-value for $\beta$textsubscript{jGWAS} from the final model. We considered as significant conditional associations with FDR $\leq$ 5\% based on the 3,995 GWAS SNP-gene pairs.

\subsection{Functional validation of eQTL variant activity and direction of effect}
We maintained the MIN6 mouse insulinoma beta cell line \cite{miyazakiEstablishmentPancreaticBeta1990} as previously described \cite{kulzerCommonFunctionalRegulatory2014}. We amplified a 473-bp genomic region containing rs10947804, rs12663159, rs146060240 and rs34247110 from human DNA (primers: 5'-GCCAGGTAAGCCAGGTA-3' and 5'-GAGTGCGGTTTCCAGAAGTC-3') and cloned it into the pGL4.10 promoterless vector (Promega) as previously described \cite{kulzerCommonFunctionalRegulatory2014}. The region was cloned in the forward orientation with respect to KCNK17 transcription and includes the promoter, 5’UTR, and the first 34 codons of KCNK17. We performed site-directed mutagenesis with the QuikChange Lightning Site-Directed Mutagenesis Kit (Agilent) to change the KCNK17 start codon from ATG to AGG to prevent interference with translation and function of the luciferase protein. We amplified the KCNK17-increasing and decreasing haplotypes. The haplotype of alleles associated with higher KCNK17 expression (risk haplotype) includes rs10947804-C, rs12663159-A, rs146060240-G, and rs34247110-A. The haplotype associated with lower KCNK17 expression (non-risk haplotype) includes rs10947804-T, rs12663159-C, rs146060240-deletion, and rs34247110-G. We performed luciferase assays as previously described \cite{kulzerCommonFunctionalRegulatory2014}. We plated 200,000 cells per well in a 24-well plate and transfected after 24 hours. We co-transfected 250 ng of haplotype plasmid and 80 ng Renilla plasmid in duplicate wells using Lipofectamine LTX (Life Technologies) and assayed luciferase activity 48 hours post-transfection. For the KCNK17-increasing haplotype, we transfected 4 independent clones and for the KCNK17-decreasing haplotype we transfected 5 independent clones. Each independent clone was transfected in duplicate. Quantified luciferase activity was normalized to empty vector (EV) and we tested for difference in luciferase activity between haplotypes using a two-sided t-test. We observed the same transcriptional effect in three separate experiments. 

\subsection{Analysis of islet-specific expression}
We used an information theory approach \cite{schugPromoterFeaturesRelated2005, heGlobalViewEnhancer2014} to score genes based on islet expression level and specificity relative to the panel of 16 diverse Illumina Human Body Map 2.0 tissues. We first calculated expression (x) in FPKM values for all Gencode v19 genes across a representative islet sample and each of the 16 tissues in the Body Map 2.0 data. We calculated the relative expression of each gene (g) in islets compared to all 17 tissues (t) as p:

\[ p_{g,islet} = \frac{x_{g,islet}}{\sum_{n=1}^{17}x_{g,t}}  \]

We next calculated the entropy for expression of each gene across all 17 tissues as H:

\[ H_g = -\sum_{n=1}^{17}p_{g,t} log_2(p_{g,t}) \]

Following previous studies (24, 25), we defined islet tissue expression specificity (Q) for each gene as:

\[ Q_{g,islet} = H_g - log_2(p_{g,islet}) \]

To aid in interpretability, we divided Q for each gene by the maximum observed Q and subtracted this value from 1 and refer to this new score as the islet expression specificity index (iESI):

\[ iESI_g = 1 - \frac{Q_{g,islet}}{Q_{max,islet}} \]

iESI scores near zero represent lowly and/or ubiquitously expressed genes and scores near 1 represent genes that are highly and specifically expressed in islets. We divided genes and subsequently the associated eQTL variants associated into quintiles as shown in Fig. \ref{fig:c2_sf4}, \ref{fig:c2_sf5}, \ref{fig:c2_sf6}E) based on the iESI score. The division by quintile provided an average sample size of 618 lead eQTL variants in each bin, which we then used to compute enrichments in genomic features. In our previous analysis of skeletal muscle, we used a decile approach as we detected lead eQTLs in $>$90\% of testable genes \cite{scottGeneticRegulatorySignature2016}, whereas here we detected a lower number of eQTLs, 3,964, and thus used a quintile approach. To depict a higher-resolution partitioning of genes based on the iESI score, we used a deciles in the interactive eQTL browser (\url{http://theparkerlab.org/tools/isleteqtl/}).

\subsection{Chromatin state analyses}
We collected cell/tissue ChIP-seq reads for H3K27ac, H3K27me3, H3K36me3, H3K4me1, and H3K4me3, and input from a diverse set of publically available data \cite{theroadmapepigenomicsconsortiumIntegrativeAnalysis1112015, ernstMappingAnalysisChromatin2011, mikkelsenComparativeEpigenomicAnalysis2010, parkerChromatinStretchEnhancer2013}. Collectively, these data represent 31 cells/tissues (shown in Fig. S6C), as well as 8 additional human and rodent datasets included for other ongoing projects. We performed read mapping and integrative chromatin state analyses in a manner similar to our previous reports (20, 29), and followed quality control procedures reported by the Roadmap Epigenomics study \cite{theroadmapepigenomicsconsortiumIntegrativeAnalysis1112015}. Briefly, we mapped reads using BWA \cite{liFastAccurateShort2009a} (version 0.5.8c), removed duplicates using samtools, and filtered for mapping quality score of at least 30. To assess the quality of each dataset, we performed strand cross correlation analysis using phantompeakqualtools  (v2.0, \url{http://code.google.com/p/phantompeakqualtools}) \cite{theroadmapepigenomicsconsortiumIntegrativeAnalysis1112015}. To select cells/tissues for ChromHMM to learn chromatin states, and following the Roadmap Epigenomics practices, for each tissue we performed QC on the most well-defined peak datasets, H3K27ac and H3K4me3. We required each of these two marks within a tissue/cell to have normalized strand cross-correlation (NSC) $>$ 0.8 and relative strand cross-correlation (RSC) score $>$1.1. Islets and 32 other cell/tissue types out of 39 passed this criteria.  The failed samples are consistent with the Roadmap Epigenomics study analyses: the 5 brain tissues and ES-HUES64 did not pass this criteria. To more uniformly represent data sets with different sequencing depths, we randomly subsampled each data set containing $>$20M mapped reads to a depth of 20M. Chromatin states were learned jointly from the 33 cell/tissues that passed QC by applying the ChromHMM (version 1.10) hidden Markov model (HMM) algorithm at 200 bp resolution to the 5 chromatin marks and input \cite{ernstMappingAnalysisChromatin2011, ernstChromHMMAutomatingChromatin2012, ernstDiscoveryCharacterizationChromatin2010}. We ran ChromHMM with a range of possible states, and selected a 13 state model as it most accurately captured information from higher state models and provided sufficient resolution to identify biologically meaningful patterns in a reproducible way. We have used this state selection procedure in previous analyses \cite{scottGeneticRegulatorySignature2016, parkerChromatinStretchEnhancer2013}. To assign biological function names to our states that are consistent with previously published states, we performed enrichment analyses in ChromHMM comparing our states to the states reported by Roadmap Epigenomics (in their ‘extended’ 18 state model) \cite{theroadmapepigenomicsconsortiumIntegrativeAnalysis1112015} for 18 matched cells/tissues (Fig. S1). We assigned the name of the Roadmap state that was most strongly enriched in each of our states. We then applied our chromatin state model to obtain chromatin state segmentations for the 6 cell/tissue types which were not used to learn the model using ChromHMM MakeSegmentation. 

\subsection{Clustering by enhancer states across tissues} 
To identify patterns of active enhancer chromatin state calls across cell and tissues, we performed k-means clustering using 200bp genomic windows where ChromHMM posterior probability for active enhancer state 1 or 2 was greater than 0.95 in at least one cell/tissue type used in this study. We identified an optimal number of clusters by plotting the within group sum of squares versus number of clusters for a range of k, and selected k=60 which corresponded to the “elbow” in the plot. We performed k-means clustering using the Hartigan-Wong algorithm with 10,000 iterations and 50 random starts.

\subsection{Overlap of enhancer clusters with stretch enhancers}
We called stretch enhancers for all cells/tissues in our chromatin state segmentations as in our previous work  \cite{scottGeneticRegulatorySignature2016, parkerChromatinStretchEnhancer2013} by merging adjacent enhancer states (Active Enhancer 1 and 2, Weak Enhancer and Genic Enhancer) in a given tissue and identifying contiguous regions $\geq$ 3kb. We quantified the overlap between each enhancer cluster and stretch enhancers for islets, liver, H1 and GM12878 using the Jaccard statistic in BEDtools \cite{quinlanBEDToolsFlexibleSuite2010}. In Fig. \ref{fig:c2_sf6}C, we normalized the Jaccard statistic within each column such that the maximum is set to 1.

Enrichment of genetic variants in genomic features We calculated the enrichment of lead islet cis-eQTL or lead T2D GWAS SNPs (including SNPs in r2 $\geq$ 0.8 with the lead SNP (SNPs in LD)) in features such as chromatin states, stretch enhancers, enhancer clusters, transcription factor footprint or non-footprint motifs using GREGOR \cite{schmidtGREGOREvaluatingGlobal2015}. TF non-footprint motifs (shown in Fig. \ref{fig:c2_f2}C and \ref{fig:c2_f3}A and B), are defined as TF motifs that are not called as footprints in either of the islet samples. For eQTL enrichment, we included the lead cis-eQTL SNP for genes significant at a given FDR threshold. The enrichment trends were consistent across different FDR thresholds (5\%, 1\%, and 0.1\%), with more stringent sets having slightly more pronounced trends. We report here the results for the FDR $\leq$ 5\% set. For T2D GWAS SNP enrichment, we aimed to use independent T2D association signals, i.e. reported lead T2D SNPs that were not in LD with each other. We sorted the list of lead GWAS SNPs (defined in section Gene-based cis-eQTLs for GWAS variants for T2D and related traits) by p-value of association with T2D and sequentially removed SNPs with r2 $>$ 0.2 with a higher ranked SNP. 

For each input SNP, ~500 control SNPs were selected that matched the input SNP for MAF, distance to the gene, and number of SNPs in r2 $\geq$ 0.8. Fold enrichment is calculated as the number of loci at which the index SNP (or SNP in LD) overlaps the feature over the mean number of loci at which the matched control SNPs (or SNPs in LD) overlap the same feature. This process accounts for the length of the features as longer features will have more overlap by chance with control SNP sets. We used the following parameters in GREGOR: r2 threshold (for inclusion of SNPs in linkage disequilibrium (LD) with the lead eQTL or T2D GWAS SNP) = 0.8, LD window size = 1Mb, and minimum neighbor number = 500. For both eQTL and GWAS SNP enrichment of TF footprint and non-footprint motifs, we report results for SNP-feature overlaps $\geq$ 4 to avoid artifacts due to low overlaps.

\subsection{Open chromatin profiling (ATAC-seq)}
We profiled chromatin accessibility in islets from 2 human organ donor samples, which were genotyped using methods identical to the other samples (see above), using the assay for transposase-accessible chromatin-sequencing (ATAC-seq).  Approximately 50-100 islet equivalents from each sample were transposed in triplicate following the methods in \cite{buenrostroTranspositionNativeChromatin2013}.  ATAC-seq replicates were barcoded and sequenced 2 x 125 bp on a HiSeq 2000 to combined total depths of $>$831M reads for islet 1 and $>$585M reads for islet 2.

For each library, we performed read alignment, duplicate removal, and filtering as described in our previous study \cite{scottGeneticRegulatorySignature2016}. We next pooled all replicates for each sample and called peaks using MACS2 (\url{https://github.com/taoliu/MACS}), version 2.1.0, with flags “-g hs --nomodel --shift -100 --extsize 200 -B --broad --keep-dup all”, retaining all peaks that satisfied a 5\% FDR.

\subsection{Haplotype-aware PWM scans}
To detect potential TFBSs in a haplotype-aware manner, we generated personalized diploid genomes from the phased, imputed genotypes for each of two islet samples, using vcf2diploid (v0.2.6a, \cite{rozowskyAlleleSeqAnalysisAllelespecific2011}). We scanned each haplotype using FIMO with PWMs from ENCODE \cite{kheradpourSystematicDiscoveryCharacterization2014}, JASPAR \cite{mathelierJASPAR2014Extensively2014}, and Jolma et al. \cite{jolmaDNABindingSpecificitiesHuman2013}. We ran FIMO using the observed nucleotide frequencies from the hg19 reference (40.9\% GC) and the default p-value cutoff (1x10-4). We converted the resulting hits to reference coordinates using chainSwap and liftOver with -minMatch=0.1, and merged the results from the two haplotypes into a single set of results per motif per sample. As an example, for islet 1, this procedure produced a total of 2.16B motif matches from our motif database. Of these, 610,544 (0.0283\%) are not detected in a single-SNP aware motif scanning procedure.

\subsection{ATAC-seq footprints}
We used CENTIPEDE \cite{pique-regiAccurateInferenceTranscription2011} to call footprints in the islets ATAC-seq data. Briefly, for each PWM scan result, we built a matrix encoding the number of TN5 integration events at a region ± 100 bp from each motif occurrence. To increase the amount of information given as input for the algorithm, we split the ATAC-seq signal into three different categories based on the diverse fragment length distribution: 36-149bp, 150-324bp, and 325-400bp. We considered any given motif occurrence bound if both the CENTIPEDE posterior probability was $\geq$ 0.99 and its coordinates were fully contained within an ATAC-seq peak. 

\subsection{Genetic reconstruction of position weight matrices using ATAC-seq footprint allelic bias data}
Previous studies have identified signatures of allelic bias in chromatin accessibility data at TF footprints \cite{mauranoLargescaleIdentificationSequence2015, moyerbraileanWhichGeneticsVariants2016}. Motivated by this, we used the heterozygous genotype calls from our islet ATAC-seq samples and the alleles observed in the reads to quantify allelic bias in regions of open chromatin (ATAC-seq TF footprints). To diminish reference allele mapping bias of our mapped ATAC-seq reads, we used the WASP mapping pipeline and duplicate removal tool \cite{geijnWASPAllelespecificSoftware2015} (downloaded from GitHub on Feb. 19, 2016). To avoid double-counting alleles that may be covered by each read in a pair as a result of occurring on a short fragment, we clipped overlapping read pairs using the ClipOverlap function of BamUtil. We included properly paired and mapped reads with mapping quality of $\geq$ 30 and base quality of $\geq$ 20. We restricted our analyses to the set of heterozygous SNPs calls within each sample (see above for genotype information). For each SNP we counted the number of reads containing each allele.  Because we did not have sufficient statistical power to call allelic bias at SNPs with low coverage, we included only SNPs with $\geq$ 20x coverage to reduce the multiple testing burden.  To help protect against mapping artifacts, we excluded SNPs with $\leq$ 2x coverage for either allele. 

We used a two-tailed binomial test that accounted for reference allele bias to evaluate the significance of the allelic bias at each SNP in each sample.  We estimated the allelic bias expected under the null for each sample and reference-alternate allele pair as previously described \cite{lappalainenTranscriptomeGenomeSequencing2013}. Briefly, for each sample and for each reference-alternate allele pair (e.g. AG and GA are separate ref-alt allele pairs) we calculated the expected fraction of reference alleles (fracRef) as the sum of the reference allele count divided by the sum of the total allele count for SNPs of a given reference-alternate allele pair. To prevent SNPs of high coverage from biasing the fracRef, we downsampled SNPs with coverage in the top 25th percentile to 30x coverage and used the downsampled reference allele and total count. To prevent SNPs of low coverage from biasing the mean fracRef, only SNPs with a total read coverage $\geq$ 30 were used. We used the observed sample- and allele-pair-specific fracRef as the true fracRef under the null hypothesis of no ASE in the binomial test. We did not test SNPs in regions blacklisted by the ENCODE Consortium due to poor mappability (wgEncodeDacMapabilityConsensusExcludable.bed and wgEncodeDukeMapabilityRegionsExcludable.bed) \cite{battleCharacterizingGeneticBasis2013}. We performed the binomial test using R’s binom.test and multiple testing correction using the 'qvalue' command in Bioconductor’s qvalue R package (version 2.2.2, \url{http://github.com/jdstorey/qvalue}; \cite{storeyStatisticalSignificanceGenomewide2003}. We considered SNPs with q-value $<$ 0.05 as having significant allelic bias.

For each motif, we reconstructed the PWM using variants with significant ATAC-seq footprint allelic bias. To create the PWM for each motif, we took all significant allelic bias SNPs at position j with over-represented nucleotide i, and summed their absolute allelic deviations from the adjusted expected fracRef (sample-specific and allele pair-specific, as calculated in the previous section). The resulting matrix of values for nucleotide i at position j is a reflection of the number of allelic biased SNPs of nucleotide i at position j and the unevenness of their imbalance toward nucleotide i. We summed the values in the matrices for the two islet samples and used them to create a PWM, so that the genetically reconstructed motifs represent the combined data from both samples. 

As a control, at each motif, we also reconstructed the PWM by summing the counts of nucleotide i at position j for all SNPs (biased + unbiased) in the motif.

\subsection{Effect of ATAC-seq footprint SNPs with allelic bias on predicted TFBS strength for CTCF and RFX motifs}
Given that we were able to reconstruct PWMs with ATAC-seq allelic bias results, we sought to address whether the two alleles from SNPs with significant allelic bias had larger differences in their PWM score than the alleles of randomly chosen SNPs occurring within the same footprints. We calculated PWM scores for the reference and alternate allele version of each sequence using the FIMO tool as described above. For each SNP, we used the FIMO p-value to calculate a SNP effect score (delta) as follows:  delta = -log\textsubscript{10}(p-value of alternate sequence) - (-log\textsubscript{10}(p-value of reference sequence)). We then measured the delta score for all allelic bias SNPs overlapping a TF footprint for CTCF\_known2 and RFX2\_4 motifs. We constructed a null set of SNPs by choosing a random set of 1000G SNPs with matching MAF and TSS-distance that also overlap the same footprints. We evaluated the difference in the absolute (delta score) distributions with a Wilcoxon rank sum test. These results are shown in Supplementary figure S9.

\subsection{T2D GWAS loci overlap with RFX footprints}
We performed enrichment analysis for T2D associated SNPs (T2D GWAS SNP and SNPs in r2 $\geq$ 0.8 with the GWAS SNP) to overlap with TF footprint and non-footprint motifs as described above. We selected TF motifs with less than 100,000 footprint occurrences genome wide in either of the Islet samples or GM12878 to help ensure specificity of binding; 1995 out of 2,870 TF motifs passed this criteria.  In Fig. \ref{fig:c2_f3}C, we show T2D associated SNPs that occur at high information content ($>$ 1) positions in their respective RFX PWM. For each shown T2D associated SNP, we used our phased genotype calls to determine the T2D associated SNP risk allele (given the T2D GWAS SNP risk allele). Multiple RFX footprints can be called at the same SNP due to motif similarity; we report the motif from the highest scoring PWM. In Fig. \ref{fig:c2_f3}D we used TOMTOM \cite{guptaQuantifyingSimilarityMotifs2007} to align the different RFX motifs using the longest (RFX3\_1) as the seed.


\section{Acknowledgements and publication}
The results presented in this chapter have been published in \cite{varshneyGeneticRegulatorySignatures2017}. My specific contributions towards this project were analyzing the epigenomic data including chromatin state, ATAC-seq and TF footprint enrichments for eQTL and GWAS loci and the analyses of T2D GWAS loci overlapping TF footprint motifs. This work was a large team effort, and I am lucky to have had the opportunity to work with such smart, motivated and helpful group of people. I thank Drs. Laura Scott, Mike Erdos and Ryan Welch for their significant contributions towards this work. I also thank my lab mates Ricardo Albanus, Peter Orchard and John Hensley for their contributions for this work. Many thanks to the members of the FUSION team for their feedback during this project. I especially thank Prof. Steve Parker for his guidance, insight and mentorship throughout this project.


