\section{Abstract}
Epigenomic signatures from histone marks and \ac{TF} binding sites have been used to annotate putative gene regulatory regions. However, a direct comparison of these diverse annotations is missing, and it is unclear how genetic variation within these annotations affects gene expression. Here, we compare five widely-used annotations of active regulatory elements that represent high densities of one or more relevant epigenomic marks: super and typical (non-super) enhancers, stretch enhancers, high-occupancy target (HOT) regions, and broad domains, across the four matched human cell types for which they are available. We observe that stretch and super enhancers cover cell type-specific enhancer chromatin states whereas HOT regions and broad domains comprise more ubiquitous promoter states. \ac{eQTL} in stretch enhancers have significantly smaller effect sizes compared to those in HOT regions. Strikingly, chromatin accessibility QTL in stretch enhancers have significantly larger effect sizes compared to those in HOT regions. These observations suggest that stretch enhancers could harbor genetically primed chromatin to enable changes in TF binding, possibly to drive cell type-specific response to environmental stimuli. Our results suggest that current eQTL studies are relatively underpowered or could lack the appropriate environmental context to detect genetic effects in the most cell type-specific regulatory annotations, which likely contributes to infrequent co-localization of eQTL with genome-wide association study (GWAS) signals.

\section{Introduction}
Genome-wide association studies (GWAS) have shown that most of the genetic variants associated with disease related traits lie in non protein-coding regions \cite{hindorffPotentialEtiologicFunctional2009}. More importantly, these loci are specifically enriched in enhancer elements of disease-relevant cell types \cite{theencodeprojectconsortiumIntegratedEncyclopediaDNA2012, mauranoSystematicLocalizationCommon2012, trynkaChromatinMarksIdentify2013, parkerChromatinStretchEnhancer2013, corradinCombinatorialEffectsMultiple2014,pasqualiPancreaticIsletEnhancer2014, quangMotifSignaturesStretch2015}. This suggests that the majority of disease associated genetic variants modulate regulatory elements that can influence gene expression. Therefore, it is essential to identify and understand the genetic signatures and molecular function(s) of gene regulatory regions. \\

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig1.png}
    \caption[Description of the regulatory annotation calling procedures]{Description of the regulatory annotation calling procedures. A: Super/typical enhancers are called by using the H3K27ac mark ChIP-seq to assign enhancer elements, stitching elements within 12.5 kb and ranking the stitched segments based on H3K27ac levels. B: Stretch enhancer calling procedure involves analyzing patterns of multiple histone marks, assigning chromatin state segmentations using ChromHMM, followed by identifying contiguous enhancer chromatin state segments longer than 3 kb. C: HOT regions are defined as regions with higher transcription factor binding occupancies than expected. D: Broad domains are defined as the top 5\% of the H3K4me3 ChIP-seq peaks by length.}
    \label{fig:c1_f1}
\end{figure}

Epigenomic profiling such as \ac{ChIP-seq} of histone modifications or \ac{TF} that can indicate regulatory activity in vivo have been effectively used to predict the regulatory function of genomic regions. For example, ‘super enhancers’ have been defined in multiple cell types as regions with high levels of the \ac{H3K27ac} mark \cite{hniszSuperEnhancersControlCell2013}. Putative enhancer elements were identified from ChIP-seq peaks, and elements within 12.5 kb of each other were stitched together. After ranking these stitched regions based on the enhancer associated ChIP-seq signal (Fig. \ref{fig:c1_f1}A), a small number (~3\%) of identified regions that contained a large fraction ($>$40\%) of the ChIP-seq signal, observable as a steep rise in the ChIP-seq signal curve (geometrical inflection point, Fig. \ref{fig:c1_f1}A) \cite{whyteMasterTranscriptionFactors2013}, were termed super enhancers. These elements were at least an order of magnitude larger in size than the remaining non-super enhancer elements (i.e., ‘typical enhancers’). This signal-based approach has been generalized as the \ac{ROSE} algorithm \cite{lovenSelectiveInhibitionTumor2013}; \cite{whyteMasterTranscriptionFactors2013} (Fig. \ref{fig:c1_f1}A). Super enhancers are thought to encompass multiple constituent enhancer elements that collectively have high regulatory potential and drive high expression of cell identity regions \cite{whyteMasterTranscriptionFactors2013, hniszSuperEnhancersControlCell2013}. \\

In another approach, ChIP-seq data for multiple histone modifications were used to annotate the genome. A \ac{HMM} based approach identified distinct and recurrent patterns in the ChIP-seq data and segmented the genome into ‘chromatin states’ \cite{ernstMappingAnalysisChromatin2011, ernstChromHMMAutomatingChromatin2012}. Analyzing chromatin states across diverse cell types and tissues, the authors identified that the longest 10\% of contiguous enhancer chromatin states (enhancers $\geq$ 3 kb) were highly cell type-specific, occurred nearby genes with highly cell type-specific \ac{GO} terms, and were enriched for cell type relevant disease and trait associated variants \cite{parkerChromatinStretchEnhancer2013}. These regions were referred to as stretch enhancers \cite{parkerChromatinStretchEnhancer2013} (Fig. \ref{fig:c1_f1}B) and represent substantially large regions of enhancer associated chromatin. \\

Regulatory annotations have also been defined from TF ChIP-seq profiling. Analysis of such datasets across cell types revealed that more than 50\% of TF bound sites occurred in highly occupied clusters that were not randomly distributed across the genome \cite{moormanHotspotsTranscriptionFactor2006, themodencodeconsortiumIdentificationFunctionalElements2010, theencodeprojectconsortiumIntegratedEncyclopediaDNA2012, boyleComparativeAnalysisRegulatory2014}. To identify regions where TF occupancies were higher than expected by chance, one study first collapsed ChIP-seq peaks for multiple TFs as observed binding regions (Fig.  \ref{fig:c1_f1}C, blue bar). The expected regions of TF binding or “target regions” (Fig.  \ref{fig:c1_f1}C, gray bars) and individual TF binding sites within these regions (Fig.  \ref{fig:c1_f1}C, colored triangles), were then randomly sampled 1000 times, while keeping the number and size distributions equivalent to those observed. Occupancies were scored based on observed and expected collapsed binding sites (Fig.  \ref{fig:c1_f1}C, blue and green blocks, respectively); regions with the top 5\% occupancies were classified as \ac{HOT} regions (Fig. \ref{fig:c1_f1}C). \\

The \ac{H3K4me3} mark is associated with active and poised promoters \cite{bernsteinBivalentChromatinStructure2006, mikkelsenGenomewideMapsChromatin2007,adliGenomewideChromatinMaps2010}. Unusually large regions of the H3K4me3 mark have been observed in multiple cell types across humans, mice and other species, often spanning up to ~60 kb \cite{adliGenomewideChromatinMaps2010,benayounH3K4me3BreadthLinked2014,chenBroadH3K4me3Associated2015}. Importantly, the broadest 5\% of H3K4me3 domains were found to mark genes with cell type-specific functions \cite{benayounH3K4me3BreadthLinked2014,thibodeauChromatinInteractionNetworks2017}. These regions have been termed broad domains (Fig. \ref{fig:c1_f1}D). \\

These diverse methodologies identify genomic regions with substantially high densities of epigenomic marks known to be associated with gene regulation. These regions denote important classes of regulatory elements, which show cell type-specificity, transcriptional activity in reporter assays, and disease relevance based on GWAS SNP enrichments \cite{kvonHOTRegionsFunction2012, parkerChromatinStretchEnhancer2013, hniszSuperEnhancersControlCell2013, benayounH3K4me3BreadthLinked2014, boyleComparativeAnalysisRegulatory2014, linActiveMedulloblastomaEnhancers2016, blinkaSuperEnhancersNanogLocus2016, daveMiceDeficientMyc2017}. Few studies have compared the characteristics for subsets of these annotations, showing some degree of overlap between HOT regions and super enhancers \cite{liGenomewideIdentificationCharacterisation2016} and chromatin interactions between broad domains and super enhancers \cite{thibodeauChromatinInteractionNetworks2017}. However, the functional differences among these annotations, especially how genetic variation in these elements affects target gene expression, are unclear. To fill this gap, we compared diverse characteristics of super-, typical-, stretch-enhancers, HOT regions and broad domains (hereafter collectively referred to as ‘regulatory annotations’) in the only four matched human cell types for which they are available: the lymphoblastoid cell line (LCL) GM12878, embryonic stem cell line H1, leukemia cell line K562, and hepatic carcinoma cell line HepG2. We used previously published annotations as these were rigorously generated by respective authors and are widely used. Collectively, these regulatory annotations represent the computational and statistical integration of 245 ChIP-seq data sets (an average of 61 ChIP-seq data sets per cell type). We report annotation summary statistics and the proportion of overlap with diverse chromatin states in these regions. We measure enrichment for proximity to genes that are expressed in a cell type-specific manner, and integrate genetic regulatory data to measure enrichment for expression quantitative trait loci (eQTL). Finally, as measures of strength of gene and chromatin accessibility regulation, we compare the effect sizes of loci associated with gene expression (eQTL), DNase hypersensitivity (dsQTL), and allelic bias in ATAC-seq data. Comparisons using these metrics allow us to quantify biological properties of these regulatory annotations. \\

\section{Results}
\subsection{Genomic distribution, coverage, and overlap of diverse regulatory annotations}
To catalogue super, typical, stretch enhancers, HOT regions and broad domains regulatory annotations, we computed the number of distinct segments marked by each annotation, the length distribution of these segments, and the percentage of the genome that is covered by each annotation across the four cell types (Fig. \ref{fig:c1_f2}A-C). Across all cell types, HOT regions comprised the greatest number of segments (Fig. \ref{fig:c1_f2}A). However, they were smaller in size (Fig. \ref{fig:c1_f2}B). Super enhancers comprised the longest segments among all annotations across the studied cell types (Fig. \ref{fig:c1_f2}B), likely due to stitching together H3K27ac peaks that are separated $\leq$ 12.5 kb. All pairwise comparisons between segment lengths for annotations were significant (adjusted p $<$ 2.2$\times$10\textsuperscript{-06}) in each of the cell types from Wilcoxon Rank Sum Test followed by Bonferroni correction, highlighting the differences across annotations. While the percent genome covered by each annotation varied across cell types, these regions consistently covered less than 2\% of the genome (Fig. \ref{fig:c1_f2}C). \\

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig2.png}
    \caption[Summary statistics and overlaps demonstrate differences in characteristics of regulatory annotations]{Summary statistics and overlaps demonstrate differences in characteristics of regulatory annotations. For each annotation in each cell type considered, shown are number of annotation segments (A), length distribution of segment annotations (B) and percent genomic coverage (C). Jaccard statistic (base-pair level intersection/union) between each pair of annotations is shown within a cell type (D) and across cell types (E).}
    \label{fig:c1_f2}
\end{figure}

Next, we calculated the fraction of overlap between all pairs of regulatory annotations. We report the Jaccard statistic (base-pair level intersection/union) for overlap between two annotations (Fig. \ref{fig:c1_f2}D, E). We compare overlaps between different annotations within a cell type (Fig. \ref{fig:c1_f2}D) and between a single annotation (e.g., broad domains) across cell types (Fig. \ref{fig:c1_f2}E). Despite their relatively low genomic coverage (0.5\% of the genome), super enhancer segments show considerable overlaps with stretch enhancer segments in the same cell type (Fig. \ref{fig:c1_f2}D), which are significantly enriched (p=0.0001, Fig. \ref{fig:c1_fs1}). This is in agreement with both of these annotations representing large domains of active enhancers marked with H3K27ac. HOT regions show extensive overlaps across cell types (Fig. \ref{fig:c1_f2}E), indicating that these regions are less cell type-specific. Broad domains display a similar pattern, though to a less pronounced degree (Fig. \ref{fig:c1_f2}E). Conversely, stretch, super and typical enhancers show low overlaps across cell types, which indicates a higher degree of cell type-specificity (Fig. \ref{fig:c1_f2}E). \\

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig_s1.png}
    \caption[Overlap enrichment between pairs of regulatory annotations]{Log\textsubscript{2}(Fold enrichment) for overlap between each pair of regulatory annotations is shown. Enrichments calculated using GAT [46]. Gray=Not significant after Bonferroni correction. Super and typical enhancers in the same cell type are strongly depleted for overlap since these are disjoint sets. Black tiles on the diagonal represent same cell type and regulatory annotation in the pair.}
    \label{fig:c1_fs1}
\end{figure}


\subsection{Regulatory annotations comprise distinct chromatin states}
Most regulatory annotations are defined using histone modification ChIP-seq profiles. However, the differences in their underlying chromatin landscape are unclear. We compared each regulatory annotation with previously reported chromatin state segmentations across all four cell types \cite{varshneyGeneticRegulatorySignatures2017} (Fig. \ref{fig:c1_fs2}). Such comparisons are informative because the chromatin states (ChromHMM states) have been generated from an integrative analysis of ChIP-seq data for five diverse histone marks (H3K4me1, H3K4me3, H3K27ac, H3K36me3 and H3K27me3) resulting in 13 chromatin states encompassing active promoter (regions enriched for H3K4me3, H3K27ac marks), enhancer (regions enriched for H3K4me1 and H3K27ac marks), transcribed (regions enriched for H3K36me3), repressed (regions enriched for H3K27me3 marks), and quiescent states (regions lacking marks) \cite{varshneyGeneticRegulatorySignatures2017}. Different enhancer states, such as active enhancer 1 and 2 represent states with different levels of H3K4me1 and H3K27ac mark enrichment and have different genomic coverage \cite{varshneyGeneticRegulatorySignatures2017}. For each regulatory annotation in a particular cell type, we computed the fraction of overlap with chromatin states in the corresponding cell type and across the other three cell types (Fig. \ref{fig:c1_fs2}). Generally, HOT regions and broad domains overlap with promoter-related chromatin states consistently across all four cell types, irrespective of which cell type they were called in (Fig. \ref{fig:c1_fs2}, facets a1-4, b1-4). In contrast, stretch, super and typical enhancers show a higher fraction of overlap with enhancer-related chromatin states in the corresponding cell type. Notably, stretch/super/typical enhancer regions defined in one cell type constitute mostly non-enhancer chromatin states in other cell types (Fig. \ref{fig:c1_fs2}, facets c1-4, d1-4, e1-4), which further reinforces the cell type-specific nature of these annotations. \\

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig_s2.png}
    \caption[Fraction of annotations overlapped by chromatin states]{Fraction of annotations overlapped by chromatin states. Overlap fractions of each annotation (facet columns) defined in each cell type (facet rows) with chromatin states defined in each cell type (X- axis) is shown. Stretch enhancers were defined using the same chromatin state model for the corresponding cell types.}
    \label{fig:c1_fs2}
\end{figure}

We then sought to quantify the cell type-specificity of enhancer and promoter chromatin states in each regulatory annotation. For each segment of a regulatory annotation, we computed the ChromHMM posterior probabilities of being called an enhancer or active promoter state averaged over 200bp intervals, denoting chromatin state preference of that segment in each cell of the four cell types. We then computed the information content encoded by these probabilities across cell types (see methods). High information content indicates high specificities of chromatin state. We observe that stretch enhancers constitute high information and high probability enhancer chromatin state (Fig. \ref{fig:c1_f3}A showing GM12878 annotations, Fig. \ref{fig:c1_fs3} showing annotations in all cell types) whereas HOT regions constitute low information and high probability promoter state (Fig. \ref{fig:c1_f3}B showing GM12878 annotations, Fig. \ref{fig:c1_fs4} showing annotations in all cell types). These analyses highlight the differences in the underlying chromatin context and cell type-specificities for these annotations.

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig_s3.png}
    \caption[Enhancer chromatin state information content for annotations]{Enhancer chromatin state information content for annotations. Average posterior probability for an annotation segment to be called an enhancer chromatin state vs the information content of that feature in the each cell type (facet rows).}
    \label{fig:c1_fs3}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig3.png}
    \caption[Enhancer and promoter chromatin state information content shows cell type-specificity of regulatory annotations]{Enhancer and promoter chromatin state information content shows cell type-specificity of regulatory annotations. Average posterior probability for an annotation segment to be called an enhancer (A) or promoter (B) chromatin state vs the information content of that feature in the GM12878 cell type calculated by comparing average posterior probabilities across the four cell types.}
    \label{fig:c1_f3}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig_s4.png}
    \caption[Promoter chromatin state information content for annotations]{Promoter chromatin state information content for annotations. Average posterior probability for an annotation segment to be called an promoter chromatin state vs the information content of that feature in the each cell type (facet rows).}
    \label{fig:c1_fs4}
\end{figure}

\subsection{Regulatory annotations exhibit distinct cell type-specificity of gene regulatory function}
Regulatory annotations have been linked to common diseases based on their enrichment to overlap GWAS variants. We directly compared GWAS SNP enrichments for diseases that are relevant to the cell types represented here, such as Crohn’s disease, rheumatoid arthritis and other autoimmune traits (relevant for lymphoblastoid cell line GM12878), and metabolic traits such as \ac{BMI} and type 2 diabetes (T2D) (relevant for liver hepatocyte cell line HepG2) – in each regulatory annotation. Super and stretch enhancers in GM12878 (\ac{LCL}) were generally the most enriched for autoimmune related trait GWAS SNPs (Fig. \ref{fig:c1_fs5}), whereas stretch enhancers and broad domains in HepG2 were enriched for BMI and T2D GWAS SNPs (Fig. \ref{fig:c1_fs5}). \\
We next assessed the gene regulatory potential for these annotations using several diverse comparisons. We first measured the distance to nearest protein-coding gene from the ends of each annotation segment and found that broad domain and super enhancer segments tend to occur in closer proximity of gene transcription start sites (TSSs) relative to other annotations (Fig. \ref{fig:c1_fs6}). Because a regulatory element does not always target the nearest gene, we next utilized cis-expression quantitative trait loci (cis-eQTL), which unambiguously identify target genes by associating genetic variation (SNPs) with gene expression. We asked if regulatory annotations overlapped cis-eQTL which were previously identified in \ac{LCL}s in the \ac{GTEx} project \cite{gtexconsortiumGeneticEffectsGene2017}. HOT regions in the LCL GM12878 showed the highest enrichment to overlap LCL eQTLs (Fig. \ref{fig:c1_fs7}), likely because these represent active promoter regions with high TF binding activity and lie close to protein coding genes (Fig. \ref{fig:c1_fs6}). However, HOT regions in control cell types (i.e., non-LCL) were similarly enriched to overlap LCL eQTLs, which highlights the similarity of HOT regions across cell types. \\

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig_s5.png}
    \caption[Enrichment for annotations in GM12878 and HepG2 to overlap GWAS loci for different traits]{Enrichment for annotations in GM12878 and HepG2 to overlap GWAS loci for different traits. Red line = Bonferroni multiple testing correction threshold. Gray = not significant after Bonferroni correction. Annotations overlapping at least 3 GWAS loci for a trait are shown in each panel.}
    \label{fig:c1_fs5}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig_s6.png}
    \caption[Cumulative distribution for distance to nearest TSS (all Gencode V19 protein coding genes) for segments in each regulatory annotation in each cell type]{Cumulative distribution for distance to nearest TSS (all Gencode V19 protein coding genes) for segments in each regulatory annotation in each cell type.}
    \label{fig:c1_fs6}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig_s7.png}
    \caption[Enrichment of regulatory annotations in four cell types to overlap with LCL eQTL (GTEx v7)]{Enrichment of regulatory annotations in four cell types to overlap with LCL eQTL (GTEx v7). Fold enrichments are shown in A, -log\textsubscript{10}(p values) are shown in B. Enrichment p values significant after a Bonferroni correction for 20 tests are marked with ‘*’.}
    \label{fig:c1_fs7}
\end{figure}

We hypothesized that significant enrichment of LCL eQTLs in regulatory annotations of unrelated cell types is largely driven by eQTLs for more ubiquitously expressed genes. To test this hypothesis, we classified protein-coding genes by their specificity of expression in LCLs using RNA-seq data for 50 diverse tissues from the GTEx project \cite{gtexconsortiumGeneticEffectsGene2017} and an information theory approach \cite{schugPromoterFeaturesRelated2005, heGlobalViewEnhancer2014, scottGeneticRegulatorySignature2016, varshneyGeneticRegulatorySignatures2017}. We calculated the expression specificities of genes by comparing the relative expression of each gene in LCLs with the entropy of the gene across all 50 tissues in the panel. We defined the LCL expression specificity index (LCL-ESI), which ranges from 0 (i.e., low or ubiquitously expressed genes) to 1 (i.e., highly and specifically expressed genes in LCL). We binned the genes into quintiles based on this LCL-ESI measure; such that bin five represents genes with the highest LCL-ESI scores (Fig. \ref{fig:c1_fs8}). We then asked which regulatory annotations occurred closer to cell type-specific genes. We calculated the distance to the nearest TSS for genes in each LCL-ESI bin, which revealed that annotation segments occur closer to genes with higher LCL-ESI (Fig. \ref{fig:c1_fs9}, colored lines). To control for the different number of segments in each annotation, we constructed a null expectation by randomly sampling genes from across the five LCL-ESI bins and calculating the distribution of distances to nearest gene TSS (Fig. \ref{fig:c1_fs9}, black). We then normalized the observed distance distribution for each LCL-ESI bin gene set with that from the null set and used this as a controlled measure of TSS proximity enrichment (Fig. \ref{fig:c1_f4}A). We observed that all regulatory annotations are depleted from occurring close to non-specific genes (LCL-ESI bin 1) and enriched to occur closer to highly specific genes (LCL-ESI bin 5). Notably, super, stretch, typical enhancers and broad domains were more enriched to occur near the most cell type-specific genes than HOT regions (Fig. \ref{fig:c1_f4}A). As expected, enrichments for all annotations to occur within larger distances to TSS (order of mega bases) converge to 1 (Fig. \ref{fig:c1_f4}A), indicating a properly controlled proximity enrichment test. \\

\begin{figure}
    \centering
    \includegraphics[width=0.7\textwidth]{2_regulatory_elements/figures/fig_s8.png}
    \caption[Gene expression specificity index in lymphoblastoid cell line (LCL-ESI)]{Gene expression specificity index in lymphoblastoid cell line (LCL-ESI). A: Distribution of LCL-ESI for protein coding genes with median transcripts per million (TPM) $>=$ 0.15 in LCL. Colors indicate equal sized binning of the genes into quintiles by LCL-ESI. Each bin contained 2753 protein coding genes. B: Median TPM for genes in each LCL-ESI quintile bin across the 50 GTEx tissues analyzed. Lymphoblastoid cell line (LCL) is named as ‘Cells-EBV-transformed lymphocytes’ in the GTEx dataset.}
    \label{fig:c1_fs8}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig_s9.png}
    \caption[Cumulative distribution for distance to nearest TSS for regulatory annotations in GM12878]{Cumulative distribution for distance to nearest TSS (Gencode V19 protein coding genes binned by LCL-ESI, 2753 genes in each bin) for regulatory annotations in GM12878. Black curves represent 10,000 random sub-samplings of 2753 genes from across the five bins.}
    \label{fig:c1_fs9}
\end{figure}


\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig4.png}
    \caption[Proximity to protein coding genes and enrichment for eQTL highlight functions of regulatory annotations]{Proximity to protein coding genes and enrichment for eQTL highlight functions of regulatory annotations. A: Enrichment for regulatory annotation elements in GM12878 to lie within distances (x-axis) of transcription start site (TSS) of protein coding genes binned by gene expression specificity in lymphoblast cell lines (LCL-ESI). Enrichment calculated in comparison to 10,000 random samplings, 95\% confidence intervals shown. B: Pearson correlation of LCL-ESI gene quintile bin numbers (increasing LCL specificity) with the fold enrichment of eQTLs of these genes in regulatory annotations. Positive correlation shows that the eQTLs for more LCL specific genes are more enriched in annotations. Significant (p $<$ 0.05) correlations are marked with a ‘*’.}
    \label{fig:c1_f4}
\end{figure}

We next asked which regulatory annotations were more enriched to overlap eQTL of more cell type-specific genes. We obtained sets of LCL eQTL \cite{gtexconsortiumGeneticEffectsGene2017} for genes in each LCL-ESI bin and calculated the enrichment of each eQTL set in the regulatory annotations. Indeed, we observe that GM12878 regulatory annotations were increasingly enriched to overlap eQTLs for highly LCL specific genes (Fig. \ref{fig:c1_fs10}) and the fold enrichment for eQTLs in a bin is positively correlated with the LCL-ESI bin number (Fig. \ref{fig:c1_f4}B, GM12878 facet). Notably, stretch enhancers, and in some instances typical enhancers, in non-LCL cell types showed strong negative correlations of LCL eQTL fold enrichment with LCL-ESI bin number (Fig. \ref{fig:c1_f4}B), indicating higher cell type-specificity for stretch enhancers. This is consistent with the previous histone modification based chromatin state analyses (Fig. \ref{fig:c1_f3}, Figs. \ref{fig:c1_fs2}, \ref{fig:c1_fs3}, \ref{fig:c1_fs4}, \ref{fig:c1_fs5}), which also highlight the cell type-specificity of stretch enhancers. HOT regions in non-LCL cell types show high enrichments for eQTLs in less cell type-specific LCL-ESI bins 1-3 (Fig. \ref{fig:c1_fs10}). This analysis shows that high enrichments of LCL eQTLs in non-LCL annotations (Fig. \ref{fig:c1_fs7}) was driven by eQTLs for more ubiquitously expressed genes. These analyses further emphasize the differences in the cell type-specificities of these regulatory annotations.\\

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig_s10.png}
    \caption{Enrichment for regulatory annotations to overlap LCL eQTL (GTEx v7, 10\% FDR) binned by LCL-ESI or the eQTL eGene.}
    \label{fig:c1_fs10}
\end{figure}

\subsection{Patterns of expression and chromatin QTL effect sizes in annotations suggest regulatory buffering}
While enriched overlap with eQTLs demonstrates genetic regulatory potential for each annotation (Figs. \ref{fig:c1_fs7}, \ref{fig:c1_fs10}, \ref{fig:c1_f4}B), this analysis does not distinguish the strength of these genetic effects on gene expression. To understand this, we compared the absolute effect sizes (beta values from the linear regression models) of LCL eQTLs overlapping different GM12878 regulatory annotations. We excluded SNPs with \ac{MAF} $<$ 0.2, since these SNPs have substantially reduced statistical power and are therefore biased to be detected as eQTL only with higher effect sizes (Fig. \ref{fig:c1_fs11}). We observed that LCL eQTLs in GM12878 stretch enhancers have nominally significantly lower (p=0.032) effect sizes than GM12878 HOT regions, however this comparison does not survive a Bonferroni correction accounting for 10 pairwise tests (Fig. \ref{fig:c1_fs12}A). To achieve higher power for such an analysis, we utilized the larger GTEx blood eQTL dataset and compared effect sizes in annotations of the blood relevant leukemia cell line K562. Consistent with the LCL analysis, we observed that effect sizes of blood eQTL in K562 stretch enhancers were significantly lower than that of HOT regions (Bonferroni corrected p = 0.0082, Fig. \ref{fig:c1_f5}A). We note that the differences in effect sizes for LCL and blood eQTL are largely due to different sample sizes and therefore power to detect eQTL. To further control for potential sources of bias in this analysis, we next asked if this effect size difference was driven by distance to the eQTL target gene’s TSS or the number of SNPs in high \ac{LD} with the index eQTL SNP. We modeled the eQTL absolute effect size using linear regression including these additional two covariates along with an indicator variable encoding stretch enhancer or HOT region annotation (eQTL overlapping both annotations were not considered). We observed a significant effect on the indicator variable (p = 0.005, regression coefficient = -0.0521, Table \ref{tbl:c1_t1}), which confirms the smaller effect size of eQTL in stretch enhancers, independent of TSS distance and LD structure. \\

\begin{figure}
    \centering
    \includegraphics[width=.7\textwidth]{2_regulatory_elements/figures/fig_s11.png}
    \caption[Lower minor allele frequency (MAF) variants have higher eQTL effect sizes]{Lower minor allele frequency (MAF) variants have higher eQTL effect sizes. A: Distribution of MAF for LCL eQTL (GTEx v7, 10\% FDR). B: LCL eQTL absolute effect size (slope of the linear regression) vs minor allele frequency (MAF).}
    \label{fig:c1_fs11}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig_s12.png}
    \caption[Gene expression and chromatin QTL effect size differences in regulatory annotations suggest regulatory buffering]{Gene expression and chromatin QTL effect size differences in regulatory annotations suggest regulatory buffering. A: Distribution of eQTL effect sizes for LCL eQTL (GTEx v7, 10\% FDR) in GM12878 regulatory annotations are shown. Nominal P values $<$ 0.05 are shown. B: Power to detect eQTL after Bonferroni correction at effect sizes corresponding the 10th through 90th percentiles observed for each annotation (shown in A). Other constant parameters for the power calculation are shown in box.
}
    \label{fig:c1_fs12}
\end{figure}

\begin{table}
  \centering
  \includegraphics[width=\textwidth]{2_regulatory_elements/figures/table_ols.png}
  \caption[Regression results modeling eQTL effect size and regulatory annotation, distance to gene TSS and number of SNPs in LD]{Ordinary least squares regression results modeling blood eQTL absolute effect size dependent on K562 HOT regions or stretch enhancer annotation, distance of the eQTL to eGene TSS and number of SNPs in LD r2$>$0.99}
  \label{tbl:c1_t1}
  
\end{table}  

Differences in effect sizes of eQTLs in stretch enhancers compared to HOT regions directly translates to differences in the statistical power to detect eQTL residing in these regulatory annotations, which have remarkably distinct cell type-specificities. To quantify this, we performed a power calculation for the 10th through 90th percentiles of the eQTL effect size distribution observed in each annotation, keeping other parameters such as sample size, MAF, type 1 error rate, number of tests and the standard deviation of the error term constant. We show that variants in stretch enhancers have nearly uniform lower power to be detected as eQTL across the effect size distribution (Fig. \ref{fig:c1_f5}B, S12B). Indeed, stretch enhancers showed lower enrichment to overlap eQTLs than HOT regions (Fig. \ref{fig:c1_fs7}). Therefore, identifying eQTLs in cell type-specific stretch enhancers will require larger sample sizes. \\

Among other mechanisms, eQTL SNPs can influence gene expression in vivo by modulating TF binding. TFs can either bind in nucleosome-depleted regions or bind and displace nucleosomes (pioneer factors) \cite{grossNucleaseHypersensitiveSites1988, wangSequenceFeaturesChromatin2012, buenrostroTranspositionNativeChromatin2013}. Therefore, QTL analysis of chromatin accessibility using DNase I hypersensitivity (dsQTL) can assess variant effects on regulatory element activity. Interestingly, we found that LCL dsQTLs \cite{degnerDNaseSensitivityQTLs2012} in stretch enhancers have significantly higher effect sizes than those in HOT regions (Bonferroni corrected p=6.2$\times$10\textsuperscript{-08}, Fig. \ref{fig:c1_f5}C), which is the opposite of what we observed for eQTL effects (Fig. \ref{fig:c1_f5}A). dsQTL in super enhancers and typical enhancers also have higher effect sizes than those in HOT regions (Bonferroni adjusted p = 0.013, 2.2$\times$10\textsuperscript{-05} respectively). To examine the effect of genetic variation on open chromatin at the resolution of an individual sample, we quantified allelic bias in the assay for transposase accessible chromatin followed by sequencing (ATAC-seq) data available in GM12878 \cite{buenrostroTranspositionNativeChromatin2013}. Allelic bias measured by quantifying the ATAC-seq signal over each of the two alleles at a heterozygous site is an indicator of allelic differences in chromatin accessibility at a specific locus. To control for different power to detect allelic bias, we uniformly down-sampled all SNPs to 30x coverage. We included all SNPs from the full range of MAFs with nominally significant allelic bias (p $<$ 0.05) since the SNP MAF does not affect the power to detect allelic bias in an individual sample. Consistent with the dsQTL results, we observed that SNPs in stretch enhancers show a significantly larger allelic bias effect size (see Methods) compared to HOT regions (Bonferroni corrected p = 0.0051, Fig. \ref{fig:c1_f5}D). This trend remains after removing SNPs with MAF $<$ 0.2, similar to the dsQTL analyses above (Fig. \ref{fig:c1_fs13}), indicating that SNP MAF does not confound this analysis. No other pairwise tests were significant. Collectively, these observations show that stretch enhancers harbor variants that have strong genetic effects on chromatin changes but these are buffered at the level of transcription.

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig5.png}
    \caption[Gene expression and chromatin QTL effect size differences in regulatory annotations suggest regulatory buffering]{Gene expression and chromatin QTL effect size differences in regulatory annotations suggest regulatory buffering. A: Distribution of eQTL effect sizes for blood eQTL (GTEx v7, 10\% FDR) in K562 regulatory annotations. B: Power to detect eQTL after Bonferroni correction at effect sizes corresponding the 10th through 90th observed for each annotation (shown in A). Other constant parameters for the power calculation are shown in box. C: Distribution of effect sizes for LCL DNase QTLs in GM12878 regulatory annotations. D: Distribution of effect sizes (deviation from expectation) for SNPs with significant allelic bias in GM12878 ATAC-seq (p $<$ 0.05, minimum coverage at SNP=30, reads down-sampled to 30, see methods) in GM12878 regulatory annotations. P values from Wilcoxon rank sum tests, after a Bonferroni correction accounting for 10 pairwise tests. Number of QTLs/allelic biased SNPs overlapping each regulatory annotation is shown in parentheses in A, C and D.}
    \label{fig:c1_f5}
\end{figure}

\begin{figure}
    \centering
    \includegraphics[width=1\textwidth]{2_regulatory_elements/figures/fig_s13.png}
    \caption[Effect sizes for Allelic Bias in GM12878 ATAC-seq]{Effect sizes for Allelic Bias in GM12878 ATAC-seq after removing low MAF SNPs (consistent with eQTL and dsQTL effect size analyses). SNPs with MAF $>$ 0.2 and allelic bias p value $<$ 0.05 were included for this analysis.}
    \label{fig:c1_fs13}
\end{figure}

\section{Discussion}
We performed a comparative analysis of five regulatory annotations, all based on diverse epigenomic signatures, to better understand their regulatory capacity and downstream transcriptional effects. We observed that stretch, super and typical enhancers overlap enhancer chromatin states in the corresponding cell type, but overlap non-enhancer chromatin states in unrelated cell types, supporting the cell type-specificity of these regulatory elements. These observations highlight H3K27ac as a good proxy for cell type-specific regulatory function. Annotations based on the H3K4me3 mark (Broad domains) and TF binding (HOT regions) show a large fraction ($>$40\%) of overlaps with promoter chromatin states across different cell types. Consistent with our observations, a recent study in the fly reported that regions bound by large numbers of TFs (such as HOT regions) are less cell type-specific \cite{kudronModERNResourceGenomeWide2017}. While the diverse ChIP-seq data used to define regulatory annotations comes from different individuals, we note that future studies using ChIP-seq data from the same individual might have even higher power to detect cell type-specific differences.\\

Analysis of genetic effects on gene regulatory function of annotations revealed that blood eQTLs in K562 stretch enhancers have significantly lower effect sizes compared to HOT regions. Stretch/super enhancers are known to regulate more cell type-specific genes for which the expression levels may be tightly controlled under basal conditions. Multiple studies have observed redundancy in gene regulation by individual components of super enhancers \cite{hayGeneticDissectionAglobin2016, shinHierarchyMammarySTAT5driven2016, moorthyEnhancersSuperenhancersHave2017, xieMultiplexedEngineeringAnalysis2017}. Such studies then contested the notion of super/stretch enhancers as a distinct entity, arguing that these annotations are no different than other enhancers. However, here we offer an alternative explanation - that enhancer buffering which results from functional redundancy could be a mechanism for tighter control of gene expression under basal conditions and would explain the low observed eQTL effect sizes. These regions could encode regulatory plasticity, allowing critical genes to respond to multiple (patho)physiologic stimuli. This would lead to smaller effects in the steady state, whereas each component could contribute to tight but pliable regulation by different signaling pathways. Therefore, the outcome of perturbing enhancer components might be different in response to different environmental stimuli and existing studies that probe basal conditions would not detect such effects.\\

In contrast, genetic variants associated with open chromatin in stretch enhancers show significantly higher effects than those in HOT regions, both within a single sample (allelic bias in ATAC-seq) and across multiple samples (dsQTL). Our results present an apparent discrepancy in that genetic variants in stretch enhancers display higher chromatin QTL effect sizes and slightly but significantly lower basal expression QTL effect sizes when compared to HOT regions. It is possible that the large constellation of TFs bound in HOT regions \cite{theencodeprojectconsortiumIntegratedEncyclopediaDNA2012, kudronModERNResourceGenomeWide2017} maintain more constitutively open chromatin, which would be less susceptible to effects of individual genetic variants. This concept of buffering has been demonstrated previously where a smaller fraction of SNPs in strong DNase peaks showed significant allelic bias compared to those in weak DNase peaks \cite{mauranoLargescaleIdentificationSequence2015}. We reason that chromatin accessibility, which influences TF binding could be a molecular feature of the initial response cascade to propagate gene expression changes under stimulatory conditions. We hypothesize that the larger genetic effects on stretch enhancer chromatin accessibility will propagate to gene expression effects under specific environmental conditions. Under this hypothesis, we expect that many dsQTL will be associated with gene expression under specific stimuli (or response-specific eQTL) rather than steady state (basal eQTL). In support of this, a recent study in the macrophage model system \cite{alasooSharedGeneticEffects2018} showed that ~60\% of eQTLs that manifest upon stimulation are chromatin QTL in the basal state. Unfortunately, currently available response expression or chromatin QTL datasets are underpowered for a comparison of effect sizes in the regulatory annotations analyzed here owing to low sample sizes. \\

Our observations could help reconcile why many cis-eQTLs are shared across cell types and infrequently co-localize with GWAS signals \cite{liuFunctionalArchitecturesLocal2017, huangFinemappingInflammatoryBowel2017, gtexconsortiumGeneticEffectsGene2017}. We have shown that while stretch enhancers are enriched to overlap GWAS loci for cell type-relevant traits, variants in these regions are underpowered to be identified as eQTL. Current eQTL studies are biased to identify eQTLs for more broadly expressed genes. Our results suggest that larger sample sizes will be needed to identify cell type-specific eQTLs. Additionally, our results suggest the need to perform response eQTL studies under carefully selected environmental conditions.


\section{Materials and Methods}

\subsection{Regulatory annotation sources}
Regulatory annotations for GM12878, H1 hESCs, HepG2 cell types were downloaded from previously published studies for HOT regions \cite{boyleComparativeAnalysisRegulatory2014}, Broad domains \cite{benayounH3K4me3BreadthLinked2014}, Stretch Enhancers \cite{varshneyGeneticRegulatorySignatures2017}, Super and Typical Enhancers \cite{hniszSuperEnhancersControlCell2013}.

\subsection{Summary statistics and overlaps between annotations, chromatin states and ATAC-seq peaks}
Summary statistics such as the number of features in each annotation, segment size distribution and percent genome coverage (Fig. \ref{fig:c1_f2}A-C) were calculated using custom scripts (see GitHub). To compute overlap fractions between all pairs of annotations shown in Fig. \ref{fig:c1_f2}D,E, we calculated the base pair level overlap between each pair using BEDtools intersect \cite{quinlanBEDToolsFlexibleSuite2010}. For each pair of annotation sets, we then calculated the Jaccard statistic by dividing the total length of the intersection region with the total length of the union region. To calculate the fraction of regulatory annotation overlap with chromatin states in Fig. \ref{fig:c1_fs2}, we used chromatin states previously defined in the four cell types considered \cite{varshneyGeneticRegulatorySignatures2017} and used BEDtools intersect. Stretch enhancer annotations were also obtained from this previous study \cite{varshneyGeneticRegulatorySignatures2017}. \\

Enrichment for overlap between each pair of regulatory annotations in Fig. \ref{fig:c1_fs1} was calculated using the Genomic Association Tester (GAT) tool \cite{hegerGATSimulationFramework2013}. To ask if two sets of regulatory annotations overlap more than that expected by chance, GAT randomly samples segments of one regulatory annotation set from the genomic workspace (hg19 chromosomes) and computes the expected overlaps with the second regulatory annotation set. We used 10,000 GAT samplings for each regulatory annotation. The observed overlap between segments and annotation is divided by the expected overlap and an empirical p-value is obtained.

\subsection{Chromatin state information content analysis}
We first compiled the average posterior probabilities of a regulatory annotation segment to be called an enhancer or promoter chromatin state. We utilized the previously published 13-chromatin state ChromHMM model (also used to define stretch enhancers) \cite{varshneyGeneticRegulatorySignatures2017}, which also outputs posterior probabilities for each 200bp genomic segment to be called each of the 13 states in each of the four cell types. We considered the sum of Active enhancer 1 and 2, weak enhancer and genic enhancer posterior probabilities to represent enhancer states, and averaged these values over all the 200bp tiles overlapping each annotation segment. We considered Active TSS, Weak TSS and Flanking TSS states to denote promoter chromatin states. For example, for a segment in GM12878 broad domains, we obtained the average posterior probabilities for the region being an enhancer or promoter state in a cell \(x_{segment, cell}\) for cell \( \in \) {GM12878, H1, HepG2, and K562}. To calculate the information content, we first calculated the relative average posterior probabilities, \(p_{segment, cell}\)

\[ p_{segment, cell} = \frac{x_{segment, cell}} { \sum_{cell=1}^{4}x_{segment, cell}} \]

Next, we calculated entropy of the segment as:

\[ Entropy_{segment}= -\sum_{cell=1}^{4}p_{segment, cell} \times log_2(p_{segment, cell}) \]

We know that entropy is maximized with all segments have equal relative probabilities, or \( p_{segment, cell} = \frac{1}{4} \) for cell \( \in \) {GM12878, H1, HepG2, and K562}

\[ Max. Entropy_{segment} = -\sum_{cell=1}^{4}\frac{1}{2} \times log_2(\frac{1}{4}) = 2 \]

\[ Information\, content_{segment,cell} = p_{segment, cell} \times (Max. Entropy_{segment} - Entropy_{segment}) \]

We then compared \(x_{segment, cell}\) with \(Information content_{segment, cell}\).

While high posterior probabilities for enhancer or promoter states indicate preference for that state, high information content indicates cell type-specificity of that chromatin state preference. For plotting Fig. \ref{fig:c1_f3}, to have the same x axes ranges for all facets for easier comparison (stretch enhancers only show high mean posterior probabilities for enhancer states and low posterior probabilities for promoter states due to their definition), we added one pseudo-count in each corner for all facets.

\subsection{Distance to nearest gene}
We downloaded the Gencode V19 gene annotations from \url{ftp://ftp.sanger.ac.uk/pub/gencode/Gencode_human/release_19/gencode.v19.annotation.gtf.gz} and obtained the transcription start site (TSS) coordinates for protein coding genes. For each segment in each annotation, we computed the distance to nearest protein coding gene TSS using BEDtools closest \cite{quinlanBEDToolsFlexibleSuite2010}.

\subsection{Enrichment of genetic variants in genomic features}
Enrichment for genome wide association study (GWAS) variants for different traits and expression quantitative trait loci (eQTL) identified in the lymphoblastoid cell line (LCL) in regulatory annotations was calculated using GREGOR (version 1.2.1) \cite{schmidtGREGOREvaluatingGlobal2015}. Since the causal SNP(s) for the traits are not known, GREGOR allows considering the input lead SNP along with SNPs in high linkage disequilibrium (LD) (based on the provided R2THRESHOLD parameter) while computing overlaps with genomic features (regulatory annotations). Therefore, as input to GREGOR, we supplied SNPs that were not in high linkage disequilibrium with each other. We pruned the list of SNPs using the PLINK (v1.9) tool \cite{purcellPLINKToolSet2007}; \cite{changSecondgenerationPLINKRising2015} –clump option and 1000 genomes phase 3 vcf files (downloaded from \url{ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502} ) as reference. For each input SNP, GREGOR selects ~500 control SNPs that match the input SNP for minor allele frequency (MAF), distance to the nearest gene, and number of SNPs in LD. Fold enrichment is calculated as the number of loci at which an input SNP (either lead SNP or SNP in high LD) overlaps the feature over the mean number of loci at which the matched control SNPs (or SNPs in high LD) overlap the same features. This process accounts for the length of the features, as longer features will have more overlap by chance with control SNP sets.\\

Specific parameters for the GWAS enrichment were: GWAS variants for Rheumatoid Arthritis, type 1 diabetes (T1D) and type 2 diabetes (T2D) were obtained from the NHGRI-EBI catalog (\url{https://www.ebi.ac.uk/gwas/}). We used the following parameters - Pruning to remove SNPs with r2 $>$ 0.2 for European population; GREGOR: r2 threshold = 0.8. LD window size = 1Mb; minimum neighbor number = 500, population = European.\\

Specific parameters for the LCL eQTL enrichment were: LCL eQTL data from the genotype tissue expression (GTEx V7) study was downloaded from the GTEx website \url{https://www.gtexportal.org/home/datasets}. We used the following parameters – Pruning to remove SNPs with r2 $>$ 0.8 for European population; GREGOR: r2 threshold = 0.99. LD window size = 1Mb; minimum neighbor number = 500, population = European. \\

We used different r2 threshold for GWAS (r2=0.8) vs eQTL (r2=0.99) enrichment analyses because eQTL analyses measure a molecular feature instead of a complex phenotype and therefore have higher resolution to identify the more likely causal variants.

\subsection{Analysis of LCL-specific expression (LCL-ESI)}
We used an information theory approach \cite{schugPromoterFeaturesRelated2005, heGlobalViewEnhancer2014} to score genes based on LCL expression level and specificity relative to the panel of 50 diverse GTEx tissues, each of which had RNA-seq data for more than 25 samples. We downloaded RNA-seq data from GTEx V7 study from the website \url{https://www.gtexportal.org/home/datasets} filename \url{GTEx\_Analysis\_2016-01-15\_v7\_RNASeQCv1.1.8\_gene\_median\_tpm.gct.gz}. This data was in the form of median transcripts per million (TPM) for each gene in each tissue. We considered protein-coding genes and removed those that were lowly expressed in LCL (median TPM $>$ 0.15) to avoid potential artifacts. We calculated the relative expression of each gene (g) in LCL compared to all 50 tissues (t) as p:

\[ p_{g,LCL} = \frac{x_{g, LCL}}{\sum_{n=1}^{50}x_{g,t}}  \]

We next calculated the entropy for expression of each gene across all 50 tissues as H:

\[ H_g = -\sum_{n=1}^{50}p_{g,t} log_2(p_{g,t}) \]

Following previous studies \cite{schugPromoterFeaturesRelated2005, heGlobalViewEnhancer2014}, we defined LCL tissue expression specificity (Q) for each gene as:

\[ Q_{g, LCL} = H_g - log_2(p_{g, LCL}) \]

To aid in interpretability, we divided Q for each gene by the maximum observed Q and subtracted this value from 1 and refer to this new score as the LCL expression specificity index (LCL-ESI):

\[ LCL-ESI_g = 1 - \frac{Q_{g, LCL}}{Q_{max, LCL}} \]
 
LCL-ESI scores near zero represent lowly and/or ubiquitously expressed genes and scores near 1 represent genes that are highly and specifically expressed in LCL. 

Enrichment for distance to genes based on gene expression specificity in LCL
We binned the protein coding genes into quintiles based on LCL-ESI, such that bin 5 included the most LCL specific genes. Each quintile bin contained N=2,753 protein coding genes. We then used BEDtools closest to calculate distance to nearest protein coding gene TSS for each bin, obtaining empirical cumulative distribution functions (ECDFs) for each regulatory annotation in each cell type. Since the regulatory annotations vary in the number of segments, and will therefore have different probabilities to occur nearby TSS, the distance to nearest protein coding gene TSS ECDFs cannot be directly compared. We therefore obtained the expected distance to nearest protein coding gene TSS ECDF for each annotation by randomly sampling N=2,753 genes from across the five bins 10,000 times and calculating the distance to nearest gene. We then calculated the TSS proximity enrichment for each annotation by dividing the observed with the mean expected ECDF. Enrichment therefore denotes the fold change in the observed fraction of annotation segments within a certain distance of protein coding gene TSSs in a specific LCL-ESI bin over the mean fraction of segments at the same distance from the randomly sampled genes. The 95\% confidence intervals for the enrichment values were calculated as observed/(mean ± 1.96*SE), where SE = standard error of the mean expected fraction. 

Enrichment to overlap eQTL based on expression specificity of gene
We sorted the eQTL SNPs into quintiles based on LCL-ESI of the associated genes (eGene) and grouped them into five equally sized bins, resulting in 585 eQTLs in each bin. Bin numbers represent eQTLs that correspond to increasingly LCL-specific genes where bin number 1 represents the least LCL-specific and bin number 5 represents the most LCL-specific genes. We calculated the enrichment for each eQTL set to overlap regulatory annotations using GREGOR with the same parameters as described above for the bulk set of LCL eQTL. To quantify the trend of LCL eQTL enrichment with LCL eGene expression specificity, we calculated the Spearman correlation of the enrichment effect size expressed as log\textsubscript{2}(fold enrichment) with the eQTL bin number using the cor() function from the stats package (v3.5.1) in R(R Core Team).

\subsection{Gene expression and chromatin accessibility QTL effect sizes in regulatory annotations}
We used the beta values or the slope of the linear regression as the effect size of LCL and blood eQTL (GTEx V7) and DNase hypersentivity site QTL (dsQTL) \cite{degnerDNaseSensitivityQTLs2012}. All of these QTL studies used inverse rank based normalization steps on the molecular features, which enables direct comparison of the effect sizes across the genome. Because low MAF SNPs have low statistical power to be detected as significant QTL at low effect sizes, these SNPs are biased to have large QTL effect sizes. We therefore removed QTL SNPs with MAF $<$ 0.2. We pruned the QTL SNPs to retain SNPs with r2 $<$ 0.8 after sorting by p value of association as described above using PLINK \cite{purcellPLINKToolSet2007}; \cite{changSecondgenerationPLINKRising2015}. Since the causal SNP for the QTL signal is unknown, we also considered SNPs in high linkage disequilibrium at r2 $>$ 0.99 with the lead QTL SNPs which were obtained using vcftools\cite{danecekVariantCallFormat2011} and 1000 genomes phase 3 reference vcf specified above. We observed higher eQTL enrichment in annotations with increasing the r2 thresholds, which is indicative of a higher signal to noise ratio. A previous study analyzing LCL eQTL also showed that functional enrichment decreased rapidly from the best eQTL towards lower ranked eQTL \cite{lappalainenTranscriptomeGenomeSequencing2013}. We compared the absolute QTL effect sizes of loci (QTL index SNP or SNP with r2 $>$ 0.99 with the index SNP) that overlapped each GM12878 annotation. We used the Wilcoxon rank sum test to identify significant differences between effect sizes of eQTL overlapping each annotation. \\
To test if there may be confounding from other genomic properties such as the distance between the eQTL eSNP to eGene and number of SNPs in high LD with the lead SNP could also influence the eQTL effect size, we calculated the contribution of the underlying regulatory annotation on the effect size while accounting for these factors. We modeled the eQTL effect size in a linear regression using the Python statsmodels library where we included a regulatory annotation indicator variable encoding eQTL overlap by a stretch enhancer or HOT region annotation and the following two covariates: (1) absolute distance of the eQTL lead SNP to its corresponding eGene TSS and (2) total number of SNPs in high LD (r2 $>$ 0.99 with the lead SNP) that overlapped the annotation. eQTLs that overlapped both annotations were not considered. Summary statistics of this regression model are presented in \ref{tbl:c1_t1}. \\

To calculate the statistical power for eQTL analysis after Bonferroni correction based on a linear regression, we used the powerEQTL.SLR function from the ‘powerEQTL’ R package \cite{dongPowerEQTLPowerSample2017} (v0.1.3; \url{https://rdrr.io/cran/powerEQTL/} ). For eQTLs overlapping each annotation, we used the eQTL effect sizes representing the 10th to 90th percentile values and calculated power by using the following parameters: MAF = 0.2, type I error rate = 0.0005, total number of tests = 1000,000, standard deviation of the error term = 0.4 and sample size N = 250.

\subsection{Comparison of allelic bias effect sizes in annotations}
To determine SNP allelic bias in GM12878 ATAC-seq data, we used the publicly available data \cite{buenrostroTranspositionNativeChromatin2013}. Adapters were trimmed using cta (v. 0.1.2; \url{https://github.com/ParkerLab/cta}), and reads mapped to hg19 using bwa mem \cite{liAligningSequenceReads2013} (default options except for the -M flag; v. 0.7.15-r1140). Bam files were filtered for high-quality autosomal read pairs using samtools \cite{liSequenceAlignmentMap2009} view (-f 3 -F 4 -F 8 -F 256 -F 2048 -q 30; v. 1.3.1). WASP \cite{geijnWASPAllelespecificSoftware2015} (version 0.2.1, commit 5a52185; using python version 2.7.13) was used to adjust for reference mapping bias; for remapping the reads as part of the WASP pipeline, we used the same mapping and filtering parameters described above for the initial mapping and filtering. Duplicates were removed using WASP’s rmdup\_pe.py script. We used the phased GM12878 VCF file downloaded from \url{ftp://ftp-trace.ncbi.nlm.nih.gov/giab/ftp/release/NA12878\_HG001/NISTv3.3.1/GRCh37/HG001\_GRCh37\_GIAB\_highconf\_CG-IllFB-IllGATKHC-Ion-10X-SOLID\_CHROM1-X\_v.3.3.1\_highconf_phased.vcf.gz}. To avoid potential artifacts associated with double-counting alleles, overlapping read pairs were clipped using bamUtil clipOverlap (v. 1.0.14; \url{http://genome.sph.umich.edu/wiki/BamUtil}: clipOverlap). The bam files from the samples in Table \ref{tbl:c1_t2} were then merged to create a single GM12878 bam file using samtools merge. We filtered for heterozygous autosomal SNPs with minimum coverage of 30. Since the power to detect allelic bias depends upon the read coverage at the SNP, SNPs with lower coverage are biased toward having higher effect sizes at any given level of statistical significance. To prevent this type of bias, we randomly down-sampled reads at each heterozygous SNP to a total of 30 reads with base quality of at least 20. We then counted the number of reads containing each allele. We used a two-tailed binomial test that accounted for reference allele bias to evaluate the significance of the allelic bias at each SNP (as described previously \cite{varshneyGeneticRegulatorySignatures2017}; implemented in a custom perl script). We did not test SNPs in regions blacklisted by the ENCODE Consortium because of poor mappability (wgEncodeDacMapabilityConsensusExcludable.bed and wgEncodeDukeMapabilityRegionsExcludable.bed). We then selected SNPs that show significant allelic bias at a nominal threshold of binomial test p value $<$ 0.05 and used BEDtools intersect to identify the set of nominally significant SNPs overlapping each annotation. We defined the effect size of allelic bias as the absolute deviation from expectation given by the absolute difference between the observed and expected fraction of reads mapping to the reference allele. We also compared the allelic bias effect sizes while only considering SNPs with MAF $>$ 0.2.

\begin{table}
  \caption[GM12878 ATAC-seq sample information]{GM12878 ATAC-seq sample information}
  \label{tbl:c1_t2}
\begin{center}
 \begin{tabular}{||c c c c c||} 
   \hline
   GEO accession & Run accession & Cell type & Cell count & Replicate \\ [0.5ex] 
 \hline\hline
   GSM1155957 & SRR891268 & GM12878 & 50000 & rep1\\
   GSM1155958 & SRR891269 & GM12878 & 50000 & rep2\\
   GSM1155959 &  SRR891270 & GM12878 & 50000 & rep3\\
   GSM1155960 & SRR891271 & GM12878 & 50000 & rep4\\
   GSM1155961 & SRR891272 & GM12878 & 500 & rep1\\
   GSM1155962 & SRR891273 & GM12878 & 500 & rep2\\
   GSM1155963 &  SRR891274 & GM12878 & 500 & rep3\\
 \hline
\end{tabular}
\end{center}
\end{table}  



\section{Data Availability}
Supplementary material is available at FigShare. Workflows for analyses as described below were run using Snakemake \cite{kosterSnakemakeScalableBioinformatics2012}. All analysis steps and code to facilitate reproducibility of this work are openly shared at GitHub \url{https://github.com/ParkerLab/regulatoryAnnotations_comparisons}. Static version of scripts and all processed data are deposited to Zenodo \url{https://zenodo.org/record/1413623#.W8f2x1JRfpB}


\section{Acknowledgements and publication}
The results presented in this chapter have been published in \cite{varshneyCellSpecificityHuman2018}. I thank Steve for his ideas, insight and contributions for this work. I also thank all the other authors for their contributions.





