Type 2 diabetes (T2D) is a complex, life-altering and chronic disease, which affects an estimated 30 million Americans and 415 million people worldwide. Large-scale genetic studies have identified numerous independent genetic signals that encode predisposition to this disease and related traits. However, the underlying biological mechanisms driving this predisposition are largely unknown. Currently, our lack of knowledge about mechanisms in T2D is a serious impediment in designing precision therapeutic strategies. The focus of my research is to untangle the genetic complexity of T2D to better understand the biological mechanisms of how disease predisposition is encoded in our DNA.


\subsection{T2D pathophysiology}
T2D is a heterogeneous syndrome characterized by hyperglycemia (increased plasma glucose levels) and abnormalities in carbohydrate and fat metabolism. The beta cells of the pancreatic islets of langerhans secrete the hormone insulin, which is essential to maintain normal levels of glucose in the body. Insulin secretion from the pancreas normally reduces glucose output by the liver, enhances glucose uptake by skeletal muscle, and suppresses fatty acid release from fat tissue. A combination of factors including resistance to insulin, inadequate insulin secretion, and excessive or inappropriate glucagon secretion contribute towards development of T2D. It has been established that insulin resistance in peripheral tissues such as skeletal muscle, adipose (fat) and liver, which can arise partly due to obesity, results in an increased insulin demand to achieve glucose homeostasis in the body \cite{stumvollTypeDiabetesPrinciples2005} (Fig. \ref{fig:c1_f0}, adapted from \cite{stumvollTypeDiabetesPrinciples2005}). The pancreas can usually compensate for this increased demand with increased insulin levels through an expansion of beta cell mass and/or insulin secretion by the islet beta cells. However, over time due to glucose toxicity and other factors, islets can begin to dysfunction and the failure to compensate for insulin resistance results in the development of T2D \cite{kellerGeneExpressionNetwork2008}. Conditions such as impaired fasting glucose and impaired glucose tolerance are known to predispose to the development of overt diabetes \cite{stumvollTypeDiabetesPrinciples2005}.


\begin{figure}
        \centering
        \includegraphics[width=.9\textwidth]{1_introduction/figures/fig0.png}
        \caption{Pathophysiology of type 2 diabetes. Insulin resistance in peripheral tissues leads to increased circulating fatty acids and the hyperglycemia. In turn, the raised concentrations of glucose and fatty acids in the bloodstream will feed back to worsen both insulin secretion and insulin resistance. Reprinted from \cite{stumvollTypeDiabetesPrinciples2005}}
        \label{fig:ci_f0}
\end{figure}


\subsection{Genetic studies to understand T2D predisposition}
T2D is caused due to a complex interplay between genetic, epigenetic and environmental factors. While obesity, diet and lifestyle are strong predictors of T2D, T2D also has a strong genetic component. Individuals with one parent who have T2D have a 40\% estimated lifetime risk of developing the disease whereas the risk increases to 70\% if  both parents are affected. Therefore, identifying these genetic bases can provide crucial insights into T2D pathogenesis. 


Numerous studies to date have aimed to identify genetic signatures of T2D. Early family-based linkage studies discovered that variants in the \textit{TCF7L2} gene were associated with T2D \cite{duggiralaLinkageTypeDiabetes1999}. Followed fine-mapping efforts indicated that an intronic variant rs7903146 contributed to the original linkage signal  \cite{duggiralaLinkageTypeDiabetes1999, grantVariantTranscriptionFactor2006}. The association has been confirmed in European, African and Asian populations and it is one of strongest and most consistently replicated genetic association with T2D with an odd ratio of ~1.4. Interestingly, a recent large scale (genome wide) study identified seven independent signals at the TCF7L2 locus \cite{mahajanFinemappingTypeDiabetes2018}, highlighting the complexity of the locus. Candidate gene studies elucidated that variants in genes including \textit{KCNJ11}, \textit{PPAR-g}, \textit{ABCC8} among others were associated with T2D. While family-based linkage and candidate gene studies supplemented our understanding of the T2D genetic architecture, these approaches were found to be ultimately limited in the context of T2D. This is because these studies generally had smaller sample sizes and tested a select group of variants based on somewhat imperfect understanding of candidate biological pathways. Smaller scale and more focussed approaches have indeed been successful for many ‘mendelian’ diseases that involve higher effect, highly penetrant variants, however, emerging evidence posited that T2D had a more complex genetic architecture driven by more commonly occurring variants that would have more modest effects. This concept is also known as the ‘common disease, common variant’ hypothesis (Fig. \ref{fig:ci_f1}), adapted from \cite{manolioFindingMissingHeritability2009}. Therefore, it was imperative to cast a wider net to identify the multiple genetic loci associated with the disease. Developments in the genotyping technology and enhanced cataloging of haplotypes (regions of the genomes that are inherited together) of common variants through efforts such as the HapMap \cite{theinteranationalhapmapconsortiumInternationalHapMapProject2003}, 1000 genomes \cite{the1000genomesprojectconsortiumGlobalReferenceHuman2015a} projects enabled testing millions of single nucleotide polymorphisms (SNPs) for association with T2D. These genome wide association study (GWAS) approaches have now identified numerous loci ($>$240) associated with T2D and related traits \cite{fuchsbergerGeneticArchitectureType2016a, mahajanRefiningAccuracyValidated2018, mahajanFinemappingTypeDiabetes2018}. Multiple GWAS studies have also be combined through meta analyses that can result in increased statistical power and add to the list of new loci. Trans-ethnic GWAS studies which can leverage differences in the LD structure across populations while considering differences in allele frequencies, are also highly effective in identifying loci \cite{prasadGeneticsTypeDiabetes2015, diabetesgeneticsreplicationandmeta-analysisdiagramconsortiumGenomewideTransancestryMetaanalysis2014, liTransethnicGenomewideAssociation2014}. 


\begin{figure}
        \centering
        \includegraphics[width=.7\textwidth]{1_introduction/figures/fig1.png}
        \caption{Relationship between strengths of effects (effect sizes) and risk variant frequencies. Reprinted from \cite{manolioFindingMissingHeritability2009}}
        \label{fig:ci_f1}
\end{figure}


The heritability estimates for T2D have varied from 20-70\% across various studies \cite{poulsenHeritabilityTypeII1999, almgrenHeritabilityFamilialityType2011, prasadGeneticsTypeDiabetes2015, mahajanFinemappingTypeDiabetes2018}. It has been suggested that since chip based GWA studies largely profiled common SNPs, rare variants (freq<??) that could not be profiled initially could explain the missing heritability. However, recent studies with substantially increased sample sizes and more complete coverage of low frequency variation have not bolstered this hypothesis for T2D \cite{mahajanRefiningAccuracyValidated2018}.


While multitude of studies have demonstrated the potential of GWAS in identifying loci, it is important to note that GWA studies essentially report associations between genomic regions and the disease trait. GWAS, however, do not inform about the underlying causal molecular mechanisms; understanding these necessitate several exhaustive follow up studies. Mechanistic insights from a refined view of T2D genetics are essential to realize the translational value of GWA studies; such efforts may then allow for personalised risk scores \cite{kheraGenomewidePolygenicScores2018}, stratification of patients by different underlying pathophysiology \cite{udlerTypeDiabetesGenetic2018} or towards identifying therapeutic targets.
                                
Understanding the causal molecular mechanisms underlying T2D GWAS associations is quite challenging due to multiple factors. Firstly, T2D and related trait GWA studies have largely implicated common variants that individually have modest effect sizes (odds ratios 1.1-1.5). Moreover, most of the variants occur in non-protein coding regions, suggesting that these do not directly affect protein structure or function. GWAS loci are commonly referred to by the names of genes located close to them for simplicity, however, only a few are close to strong biological candidates. Only occasionally one might find causal SNP candidates with particularly strong biological credentials such those causing a non-synonymous coding change.  Secondly, the lead GWAS SNP might not always be the causal SNP. This is because our genome is inherited in blocks such that multiple variants are highly correlated with each other and are said to in linkage disequilibrium (LD). Thirdly, the target genes and how the GWAS SNP risk alleles affect their expression level (increasing/decreasing) often unknown, as these may be distant. For example, an obesity-associated variant located in the intronic region of the \textit{FTO} gene, do not affect the expression of this gene, however, influence the expression of the genes \textit{IRX3} and \textit{IRX5}, which are located over a megabase away from the variant \cite{claussnitzerFTOObesityVariant2015}. These factors culminate in scenarios where GWAS signals tag multiple, mostly non-coding variants where the causal SNP(s) and their target genes are difficult to identify using genetic information alone.    
                                                                             
\subsection{Flow of biological information from genetic variation to phenotype} 
In understanding how genetic variation can influences phenotypic traits, it is critical to consider several layers molecular domains over which genetic information can propagate. The DNA encodes information which can affect the chromatin landscape and influence gene expression, effects of which can then relay to influence protein and metabolomic networks in eventually affecting phenotypic changes (Fig. \ref{fig:ci_f2}, adapted from \cite{civelekSystemsGeneticsApproaches2014}). As initial molecular control layers, understanding the epigenomic and transcriptomic effects of genetic variation can be highly informative in piecing together molecular mechanisms \cite{kyonoGenomicAnnotationDiseaseassociated2019}. For example, genetic variants occurring in regulatory elements may confer risk by altering transcription factor binding sites that propagate signals from upstream transcription factors to influence downstream target gene expression. One of the first steps towards understanding the molecular impact of genetic variation is therefore using epigenomic information to identify the regulatory elements through which these act. 


\begin{figure}
        \centering
        \includegraphics[width=.7\textwidth]{1_introduction/figures/fig2.pdf}
        \caption{Molecular domains propagating genetic information towards phenotype. Adapted from \cite{civelekSystemsGeneticsApproaches2014}}
        \label{fig:ci_f2}
\end{figure}


\subsection{Investigating the epigenomic domain to identify gene regulatory elements}
DNA wraps around histone proteins and forms nucleosomes; covalent modifications on these histones and other patterns in this landscape helps establishing and maintaining relevant cell-specific and cell-identity gene expression programs. Different modifications on the histone tails have been observed to be associated with distinct functions. For example, promoters are marked by tri-methylation of histone H3 lysine 4 (H3K4me3) \cite{bernsteinBivalentChromatinStructure2006, mikkelsenGenomewideMapsChromatin2007,adliGenomewideChromatinMaps2010}, enhancers are marked by mono-methylation of H3K4 (H3K4me1) \cite{heintzmanDistinctPredictiveChromatin2007} the acetylation of H3K27 (H3K27ac) mark is associated with both active promoter and enhancer activity \cite{mikkelsenGenomewideMapsChromatin2007}. Also, trimethylation of H3K36 (H3K36me3) is associated with transcribed regions; and trimethylation of H3K27 (H3K27me3) is associated with Polycomb repressed regions \cite{heintzmanDistinctPredictiveChromatin2007, zhouChartingHistoneModifications2011}. These molecular modifications among others have been thoroughly profiled across a multitude of cell and tissue types using chromatin immunoprecipitation followed by sequencing (ChIP-seq) \cite{theencodeprojectconsortiumIntegratedEncyclopediaDNA2012, consortiumIntegrativeAnalysis1112015}. Patterns of these diverse and informative signals have been distilled using hidden markov model (HMM) method implemented in the ChromHMM tool \cite{ernstMappingAnalysisChromatin2011, ernstChromHMMAutomatingChromatin2012} to segment the genome into ‘chromatin states’. Parker, Stitzel and colleagues constructed chromatin state maps for pancreatic islets, and identified islet-specific stretch enhancers (SEs), which are long (≥3 kb) segments of the genome that are continuously decorated with enhancer-associated histone marks. Importantly, this study revealed that T2D GWAS loci are highly and specifically enriched to occur in islet stretch enhancers. Similar observations were also made by others and these studies collectively represent the first level of functional convergence in which disease-relevant variants across the genome are enriched in a set of large enhancers active in specific tissues \cite{theencodeprojectconsortiumIntegratedEncyclopediaDNA2012,mauranoSystematicLocalizationCommon2012,trynkaChromatinMarksIdentify2013,parkerChromatinStretchEnhancer2013,pasqualiPancreaticIsletEnhancer2014, quangMotifSignaturesStretch2015}. However, while chromatin state analysis is useful for narrowing down the regions of interest to a small subset of regulatory regions (Fig. \ref{fig:ci_f3}, reprinted from \cite{kyonoGenomicAnnotationDiseaseassociated2019}), the resolution of analysis is approximately 200 bp (a consequence of the fact that each nucleosome contains about 147 bp of DNA wrapped around the histones), which is still too coarse to pinpoint the underlying sequence motif(s) that could be mediating a genetic regulatory effect. 


\begin{figure}
        \centering
        \includegraphics[width=.7\textwidth]{1_introduction/figures/fig3.pdf}
        \caption{Functional mapping of diabetes-associated variants using tissue-specific regulatory maps. GWAS have identified loci associated with risk for type 2 diabetes, with strength of association (-log\textsubscript{10} p value) shown throughout the genome in a Manhattan plot (top, data from \cite{thediabetesgeneticsreplicationandmeta-analysisdiagramconsortiumLargescaleAssociationAnalysis2012,t2dknowledgeportalTypeDiabetesKnowledge2019}). Each genome-wide significant region (above the horizontal red line) can then be explored using a locus-zoom plot (B), which shows one of the type 2 diabetes-associated loci (overlapping the gene \textit{WFS1}) as an example \cite{t2dknowledgeportalTypeDiabetesKnowledge2019}. In the locus zoom plot, each dot represents a variant associated with type 2 diabetes, and its colour represents the level of LD, with the lead variant (reference variant [Ref Var]) highlighted in purple. Most SNPs occur in non- coding regions, where chromatin state analyses (C) help identify locations of tissue-specific regulatory regions. While some enhancer regions may be shared across tissues, there are others that are unique. Reprinted from \cite{kyonoGenomicAnnotationDiseaseassociated2019}}
        \label{fig:ci_f3}
\end{figure}


\subsection{Profiling accessible chromatin to identify regulatory elements in high resolution}
To identify regulatory segments at a higher resolution, it is imperative to locate the binding sites of TFs. TFs are known to bind in regions of accessible or ‘open’ chromatin regions, or, conversely, TF binding can create focal changes to chromatin architecture such that nucleosomes are displaced and the surrounding DNA becomes more accessible. Consequently, profiling accessible regions of the genome can help close in to the TF bound regulatory elements. Early genome-wide maps of open chromatin regions in human pancreatic islets used formaldehyde-assisted isolation of regulatory elements (FAIRE-seq) \cite{gaultonMapOpenChromatin2010} or DNase I digestion coupled to DNA sequencing (DNase-seq) \cite{stitzelGlobalEpigenomicAnalysis2010}. By comparing these data to maps from other cell types, these studies identified islet-specific open chromatin regions that coincided with evolutionarily conserved binding sites for key islet transcription factors near to genes of critical importance in pancreatic islets (e.g. PDX1 and NKX6-1). A more recent open chromatin profiling method, the assay for transposase-accessible chromatin sequencing (ATAC-seq) \cite{buenrostroATACseqMethodAssaying2015}, has enabled more routine analysis of scarce samples, such as human pancreatic islets, because of its lower minimum sample size (Fig. \ref{fig:ci_f4}A). DNA sequence underlying the highly accessible regions (ATAC-seq peaks) can then be interrogated using the vast TF DNA binding sequence motif (os position weight matrix, PWM) information databases \cite{kheradpourSystematicDiscoveryCharacterization2014, mathelierJASPAR2014Extensively2014,sandelinJASPAROpenaccessDatabase2004,jolmaDNABindingSpecificitiesHuman2013} using specifically designed tools \cite{pique-regiAccurateInferenceTranscription2011} to infer binding sites of these TFs (TF footprint motifs) (Fig. \ref{fig:ci_f4}B, adapted from \cite{kyonoGenomicAnnotationDiseaseassociated2019}). Therefore, compared with analysis of histone marks, open chromatin analyses (especially ATAC-seq) have a higher resolution, permitting the identification of specific TF footprint motifs that may be altered by disease risk alleles. 


\begin{figure}
        \centering
        \includegraphics[width=1\textwidth]{1_introduction/figures/fig4.pdf}
        \caption{Pinpointing individual cis-regulatory elements within broad regulatory regions. A. Open chromatin regions can be identified by assays such as ATAC-seq. TF Motif analysis within open chromatin regions may identify bound by TFs. Searching a TF motifs from available databases in the open chromatin region can nominate TF footprint motif(s) associated with specific TFs. B. eQTL analyses, which use statistical associations between genetic variation and gene expression at a population level, can identify variants that influence expression of downstream target genes, for example, by activating or disrupting transcription factor binding sites. In this example, the blue ‘C’ allele disrupts an underlying TF footprint motif and is associated with decreased expression of the hypothetical target ‘gene X’. Adapted from \cite{kyonoGenomicAnnotationDiseaseassociated2019}}
        \label{fig:ci_f4}
\end{figure}


\subsection{Identifying target genes of regulatory variants}
The next step towards attaining a more complete mechanistic insight is identifying the target genes of the regulatory variants. This can be accomplished with expression quantitative trait loci (eQTL) studies, which look at population-level statistical associations between gene expression and genetic variation to assign SNPs to target genes (Fig. \ref{fig:ci_f4}B). Standard eQTL analysis involves a direct association test between markers of genetic variation with gene expression levels typically measured in tens or hundreds of individuals. Several such studies have been conducted across diverse and diabetes-relevant human tissues, such as skeletal muscle \cite{scottGeneticRegulatorySignature2016}, adipose \cite{civelekGeneticRegulationAdipose2017}, liver \cite{gtexconsortiumGeneticEffectsGene2017} , islets \cite{fadistaGlobalGenomicTranscriptomic2014, buntTranscriptExpressionData2015, varshneyGeneticRegulatorySignatures2017} along with other emerging work included as a part of this work. Additional layers of regulatory annotation could reveal additional signatures of convergence.


\subsection{Thesis outline}
In this work, I have analyzed multiple large-scale omics datasets to better understand gene regulatory mechanisms. In chapter 2, I compared gene regulatory annotations defined using diverse epigenomic data across 4 cell types to compare their cell specificities and genetics of gene expression regulation. In chapters 3 and 4, I describe large scale eQTL mapping efforts along with exhaustive integration of epigenomic data to describe molecular regulatory mechanisms. In chapter 5, I describe profiling and analysis of the enhancer transcriptome in islets, which I then integrate with other available epigenomic data to better understand characteristics of gene regulation. I then delineate the exciting future perspectives that stem from my work and could further contribute to the field of human complex disease genetics.