\section{Abstract}
Identifying active regulatory elements and their characteristics is critical to understand gene regulatory mechanisms and subsequently better delineating biological mechanisms of complex disease/trait predisposition. Studies have shown that many active enhancers are transcribed into enhancer RNA (eRNA). Here, we identify actively transcribed regulatory elements in human pancreatic islets in high genomic resolution by generating eRNA profiles using cap analysis of gene expression (CAGE) across 70 islet samples. We identify $>$10,000 clusters of CAGE tag transcription start sites (TSS) or tag clusters (TCs) in islets, ~20\% of which are islet specific when compared to CAGE TCs in other publicly available tissues. Islet TCs are most enriched to overlap GWAS loci for islet-relevant traits such as fasting glucose. We integrated islet CAGE profiles with diverse epigenomic information such as chromatin immunoprecipitation followed by sequencing (ChIP-seq) profiles of five histone modifications and accessible chromatin profiles from the assay for transposase accessible chromatin followed by sequencing (ATAC-seq), to understand how the underlying chromatin landscape affects transcription. As expected, we observe that transcription largely initiates downstream of ATAC-seq peak summits. We identify that ATAC-seq informed transcription factor binding sites (TF ‘footprint’ motifs) for the RFX TF family are highly enriched in transcribed regions occurring in enhancer associated chromatin, whereas footprint motifs for the ETS family TFs are highly enriched in transcribed regions with promoter associated chromatin. Using massively parallel reporter assays in a rat pancreatic islet beta cell line, we tested the activity of 3,240 islet TC elements, ~70\% (2,206) of which show significant regulatory activity (5\% FDR). This work provides a high-resolution transcriptional regulatory map of human pancreatic islets.


\section{Introduction}
T2D is a complex disease that is caused due to an interplay of factors such as pancreatic islet dysfunction and insulin resistance in peripheral tissues such as fat and muscle. GWASs to date have identified $>$240 loci that modulate risk for T2D \cite{mahajanFinemappingTypeDiabetes2018}. However, these SNPs mostly occur in non protein-coding regions and are highly enriched to overlap islet-specific enhancer regions \cite{theencodeprojectconsortiumIntegratedEncyclopediaDNA2012, mauranoSystematicLocalizationCommon2012, trynkaChromatinMarksIdentify2013, parkerChromatinStretchEnhancer2013, pasqualiPancreaticIsletEnhancer2014, quangMotifSignaturesStretch2015}. This suggests the variants likely affect gene expression rather than directly altering protein structure or function. Moreover, due to the correlated structure of common genetic variations across genome, GWAS signals are usually marked by numerous SNPs in high linkage disequilibrium (LD). Therefore, identifying causal SNP(s) is extremely difficult using genetic information alone. These factors have impeded our understanding of the molecular mechanisms by which genetic variants modulate gene expression in orchestrating disease. 


In order to understand gene regulatory mechanisms, it is essential to identify regulatory elements at high genomic resolution, since these are most likely to house causal variant(s). Active regulatory elements can be delineated by profiling covalent modifications of the histone H3 subunit such as H3 lysine 27 acetylation (H3K27ac) which is associated with enhancer activity, H3 lysine 4 trimethylation (H3K4me3) which is associated with promoter activity, among others. However, such chromatin modification based methods identify regions of the genome that typically span hundreds of base pairs. Since TF binding can affect gene expression, TF accessible regions of the chromatin within these active enhancer and promoter elements can enable identifying the regulatory element at a higher resolution. Numerous studies have utilized this diverse information in islets to nominate causal gene regulatory mechanisms \cite{varshneyGeneticRegulatorySignatures2017,buntTranscriptExpressionData2015, fadistaGlobalGenomicTranscriptomic2014, thurnerIntegrationHumanPancreatic2018, romanTypeDiabetesAssociated2017, thurnerIntegrationHumanPancreatic2018}. 


In the light of identifying active regulatory elements, studies have shown that a subset of enhancers are also transcribed into enhancer RNA (eRNA), and that transcription is a robust predictor of enhancer activity \cite{anderssonAtlasActiveEnhancers2014, mikhaylichenkoDegreeEnhancerPromoter2018}. eRNAs are nuclear, short, mostly-unspliced, 5’ capped and non-polyadenylated \cite{anderssonAtlasActiveEnhancers2014}. eRNAs have generally shown to be bidirectionally transcribed with respect to the regulatory element \cite{kimWidespreadTranscriptionNeuronal2010, melgarDiscoveryActiveEnhancers2011, anderssonAtlasActiveEnhancers2014}, however, unidirectional transcription at enhancers has also been reported. Previous studies have indicated that these transcripts could be stochastic output of Pol2 and TF machinery at active regions, whereas in some cases, the transcripts could serve important functions such as sequestering TFs or potentially assisting in chromatin looping  \cite{kaikkonenRemodelingEnhancerLandscape2013, hsiehEnhancerRNAsParticipate2014, liFunctionalRolesEnhancer2013, yangEnhancerRNAdrivenLooping2016a}. Therefore, identifying active sites where transcription initiates can pinpoint active regulatory elements at a higher genomic resolution.
                 
Genome-wide sequencing of 5' capped RNAs using Cap Analysis of Gene Expression (CAGE) can detect transcription start sites (TSSs) and thereby profile transcribed promoter and enhancer regions \cite{kimWidespreadTranscriptionNeuronal2010, anderssonAtlasActiveEnhancers2014}. CAGE-identified enhancers are two to three times more likely to validate in vitro than non-transcribed enhancers detected by chromatin-based methods \cite{anderssonAtlasActiveEnhancers2014}. An advantage of CAGE is that it can be applied on RNA samples from hard to acquire biological tissue such as islets and does not require live cells that are imperative for other TSS profiling techniques such as GRO-cap seq \cite{coreAnalysisNascentRNA2014, coreNascentRNASequencing2008, lopesGROseqToolIdentification2017}. The functional annotation of the mammalian genome (FANTOM5) project \cite{thefantomconsortiumPromoterlevelMammalianExpression2014} has generated an exhaustive CAGE expression atlas across 573 primary cell types and tissues, including the pancreas. However, islets, that secrete insulin and are relevant for T2D and related traits, constitute only ~1\% of the pancreas tissue. Therefore, pancreas transcriptome cannot accurately represent the islet enhancer transcription landscape. Motivated by these reasons, we profiled the islet transcriptomes using (CAGE). Here, we present the islet CAGE TSS atlas in pancreatic islets and complement the omics compendium for the tissue.


\section{Results}
\subsection{The CAGE landscape in human pancreatic islets}
We analyzed transcriptomes in 70 human pancreatic islet samples obtained from unrelated organ donors by employing CAGE on total RNA from each sample. To enrich for the non poly-adenylated and short in size ($<$1kb) eRNA transcripts \cite{anderssonAtlasActiveEnhancers2014}, we performed polyA depletion and small fragment size selection ($<$1kb, methods CAGE library preparation) to enrich for the eRNA transcript fraction. CAGE libraries were prepared according to the \ac{nAnTiCAGE} protocol \cite{murataDetectingExpressedGenes2014}, and an 8 bp unique molecular identifier was added to identify PCR duplicates. We sequenced CAGE libraries, performed quality control (QC) and mapped to the hg19 genome and identified transcription start sites (TSSs) or CAGE tags. We selected 51 samples that passed our QC measures (see methods) for all further analyses. To identify regions with high density of transcription initiation events, we called clusters of CAGE tags or tag clusters (TCs) using the paraclu \cite{frithCodeTranscriptionInitiation2008} method in each islet sample. We then identified a consensus set of aggregated islets TCs by merging TCs across samples in a strand-specific manner and retaining TC segments that were supported by at least 10 individual samples (Methods, Fig. \ref{fig:c5_sf_islet_threshold}). We identified 10,373 tag clusters with median length of 191 bp (Fig. \ref{fig:c5_sf_tc_lengths}), spanning a total genomic territory of ~2.5 Mb. To analyze characteristics of islet TCs and explore the chromatin landscape underlying these regions, we utilized publicly available ChIP-seq data for five histone modifications along with ATAC-seq data in islets \cite{varshneyGeneticRegulatorySignatures2017}. We integrated the datasets for histone modifications, namely, promoter associated H3K4me3, enhancer associated H3K4me1, active promoter and enhancer associated H3K27ac, transcribed gene-associated H3K36me3 and repressed chromatin associated H3K27me3 across islets and analyzed the data jointly with corresponding publicly available ChIP-seq datasets for Skeletal Muscle, Adipose and Liver (included for other ongoing projects) using ChromHMM \cite{ernstDiscoveryCharacterizationChromatin2010, ernstMappingAnalysisChromatin2011, ernstChromHMMAutomatingChromatin2012}. This analysis produced 11 unique and recurrent chromatin states (Fig. \ref{fig:c5_sf_chromstate}), including promoter, enhancer, transcribed, and repressed states. Fig. \ref{fig:c5_f1}A shows an example locus in the intronic region of the \textit{ST18} gene where a TC identified in islets overlaps an active TSS chromatin state and an ATAC-seq peak. The regulatory activity of this element was validated by the VISTA project in an \textit{in vivo} reporter assay in mouse embryos \cite{viselVISTAEnhancerBrowser2007}. 


\begin{figure}
        \centering
        \includegraphics[width=.5\textwidth]{5_cage/figures/M1_S1_sample_islet_only.pdf}
        \caption[Islet TC identification using CAGE data across multiple samples]{TC segments called using the paraclu in each of the 51 islet samples were merged in a strand specific manner. Shown here is the number of merged TC segments that overlap TC territory in x or more islet samples. We required support in a minimum of 10 islet samples to include a TC segment in the aggregate list of islet TCs.}
        \label{fig:c5_sf_islet_threshold}
\end{figure}


\begin{figure}
        \centering
        \includegraphics[width=.5\textwidth]{5_cage/figures/M1_S3_islet_len_distribution.pdf}
        \caption[Islet TC length distribution]{Distribution of islet TC lengths.}
        \label{fig:c5_sf_tc_lengths}
\end{figure}


\begin{figure}
        \centering
        \includegraphics[width=.85\textwidth]{5_cage/figures/M1_S3_chromStates.pdf}
        \caption[11 chromatin state model]{11 chromatin state model. Shown are the emission probabilities of each of the five histone marks, chromatin state annotation and the percent genome coverage of each state}
        \label{fig:c5_sf_chromstate}
\end{figure}


\begin{figure}
        \centering
        \includegraphics[width=1\textwidth]{5_cage/figures/f1.png}
        \caption[Islet CAGE tag cluster identification]{Islet CAGE tag cluster identification. A: Genome browser view of the intronic region of the \textit{ST18} gene as an example locus where a TC is identified in islets that overlaps an islet ATAC-seq peak and an active TSS chromatin state. This TC also overlaps an enhancer element validated by the VISTA project \cite{viselVISTAEnhancerBrowser2007}. B: Base-pair level overlap between islet CAGE TC territory and FANTOM robust CAGE peaks. C: Distribution of the number of tissues in which TCs identified by the FANTOM project overlap each islet TC segment. D: Genome browser view of an example locus near the \textit{AP1G2} gene that highlights an islet TC that is also identified in FANTOM tissues (FANTOM TCs track is a dense depiction of TCs called across $>$120 tissues) (green box), occurs in a ATAC-seq peak region in both islets and lymphoblastoid cell line GM12878 (ATAC-seq track) and overlap active TSS chromatin states across numerous other tissues. Another islet TC ~34 kb distal to the \textit{AP1G2} gene is not identified as a TC in other FANTOM tissues, occurs in an islet ATAC-seq peak and a more islet-specific active enhancer chromatin state region (blue box). E: Enrichment of islet TCs to overlap islet chromatin state annotations and other common annotations. F: Enrichment of islet TCs to overlap GWAS loci of various disease/traits. Number of loci for each trait are noted in parentheses.}
        \label{fig:c5_f1}
\end{figure}


We next compared the islet TCs with CAGE peaks identified across across diverse cell and tissue types by FANTOM project. Using CAGE profiles across hundreds of cell/tissues, the FANTOM project identified peaks using a decomposition-based peak identification (DPI) method \cite{thefantomconsortiumPromoterlevelMammalianExpression2014}, following which a set of ‘robust’ peaks were defined that included a CAGE TSS with more than 10 read counts and 1 TPM (tags per million) in at least one sample. We observed that 77.8\% of Islet TCs segments overlapped (at least 1bp) with FANTOM robust peaks, and the total overlapping region comprised 24\% of the total Islet TC territory (Fig. \ref{fig:c5_f1}B). To compare islet TCs with individual FANTOM tissues, we identified TCs in each FANTOM human tissue using the paraclu method similarly as islets. For each Islet TC segment, we then calculated the number of FANTOM tissues in which TCs overlapped the islet segment. We observed that ~20\% of Islet TCs were unique, whereas about ~60\% of segments were shared across 60 or more  tissues (Fig. \ref{fig:c5_f1}C). We highlight an example locus where an islet TC in the \textit{AP1G2} gene occurs in active TSS chromatin states across multiple tissues, and overlaps shared ATAC-seq peaks in islet and the lymphoblastoid cell line GM12878 \cite{buenrostroTranspositionNativeChromatin2013} (Fig. \ref{fig:c5_f1}D). This region was also identified as a TC in FANTOM tissues (Fig. \ref{fig:c5_f1}D, green box). Another islet TC ~34kb away however occurs in a region lacking gene annotations, and overlaps a more islet-specific active enhancer chromatin state and ATAC-seq peak (Fig. \ref{fig:c5_f1}D, blue box). This region was not identified as a TC in the 120 FANTOM tissues that were analyzed. These data highlight that islet TCs comprise both shared and also islet-specific sites of active transcription initiation.   


We next asked if islet TCs preferentially occurred in certain genomic annotations. We computed the enrichment of islet TCs to overlap islet annotations such as active TSS and other chromatin states and islet ATAC-seq peaks. We also included ‘static’ annotations such as known gene promoters, coding, untranslated (UTR) regions, or annotations such as super enhancers, or histone ChIP-seq peaks that were aggregated across multiple cell types. We observed that Islet TCs were highly enriched to overlap Islet active TSS states ($>$ 60 fold, P=0.0001, Fig. \ref{fig:c5_f1}E). This result is largely expected since CAGE profiles transcription start sites where the underlying chromatin is more likely to look like ‘active TSS’. TCs were also enriched to overlap islet ATAC-seq peaks, which signifies that these regions of transcription initiation are bound by TFs, and 5’ untranslated regions (UTRs). 


To gauge if these transcribed elements could be relevant for diverse disease/traits, we computed enrichment for TCs to overlap GWAS loci for $>$100 traits from the NHGRI catalog \cite{bunielloNHGRIEBIGWASCatalog2019}. We observed that traits such as Fasting Glucose (FGlu) (fold enrichment = 7.05, P value = 3.30$\times$10\textsuperscript{-4}), metabolic traits (fold enrichment = 6.44, P value = 2.09$\times$10\textsuperscript{-4}) were the among the most highly enriched, highlighting the relevance and of these transcribed elements for islets (Fig. \ref{fig:c5_f1}E). GWAS loci for T2D were also enriched in islet TCs (fold enrichment = 2.45, P value = 0.02). Since T2D is orchestrated through a complex interplay between islet beta cell dysfunction and insulin resistance in peripheral tissues, we reasoned that some underlying pathways in T2D might be more relevant to islets than others. To explore this rationale, we utilized results from a previous study that analyzed GWAS data for T2D along with 47 other diabetes related traits and identified clusters of related loci at the  T2D GWAS signals \cite{udlerTypeDiabetesGenetic2018}. Interestingly, we observe that signals in the islet beta cell and proinsulin cluster were highly enriched to overlap islet TCs (fold enrichment = 5.59, P value = 0.004), whereas signals in the insulin resistance cluster were depleted (fold enrichment = 0.91). These results suggest that islet TCs comprise active regulatory elements relevant for traits specifically related to islet function.
    
\subsection{Integrating CAGE TCs with epigenomic information}
We further explored CAGE profiles relative to the underlying chromatin landscape to identify characteristics of transcription initiation. We first overlayed CAGE profiles over accessible chromatin (ATAC-seq) profiles. Aggregated CAGE signal over ATAC-seq narrow peak summits highlighted the characteristic divergent pattern of transcription (Fig. \ref{fig:c5_f2}A). Conversely, we anchored ATAC-seq signal over islet TC centers and observed that the summit of the ATAC-seq signal lies upstream of the TC center (Fig. \ref{fig:c5_f2}B). We next asked if TF binding sites were more enriched to occur upstream of TCs vs downstream. We utilized TF footprint motifs previously identified using islet ATAC-seq data and TF DNA binding position weight matrices (PWMs) \cite{varshneyGeneticRegulatorySignatures2017}. These footprint motifs represent putative TF binding sites that are also supported by accessible chromatin profiles, as opposed to TF motif matches that are only informed by DNA sequence. We observe that most TFs were more enriched to occur in TCs and the 500bp upstream region as compared to TCs and 500 bp downstream region (Fig. \ref{fig:c5_f2}C). These observations highlight that the region upstream of the TC is most accessible and more TF binding events occur. 


\begin{figure}
        \centering
        \includegraphics[width=1\textwidth]{5_cage/figures/f2.png}
        \caption[Integrating Islet CAGE TCs with other epigenomic information reveals characteristics of transcription initiation]{Integrating Islet CAGE TCs with other epigenomic information reveals characteristics of transcription initiation. A: Aggregate CAGE profiles over ATAC-seq peak summits. B: Aggregate ATAC-seq profile over TC midpoints. C: Enrichment of TF footprint motifs to overlap TC and 500 bp upstream region (y axis) vs TC and 500 bp downstream region (x axis). D: Chromatin state annotations across 98 Roadmap Epigenomics cell types (using the 18 state ‘extended model’, \cite{theroadmapepigenomicsconsortiumIntegrativeAnalysis1112015}) for TC segments that occur in islet active TSS chromatin state and overlap ATAC-seq peaks. These segments were segregated into those occurring 5kb proximal (left) and distal (right) to known protein coding gene TSS (Gencode V19). E: Chromatin state annotations across 98 Roadmap Epigenomics cell types for TC segments that occur in islet active enhancer chromatin state and overlap ATAC-seq peaks, segregated into those occurring 5kb proximal (left) and distal (right) to known gene TSS. F: Aggregate CAGE profiles centered and oriented relative to RFX5\_known8 footprint motifs occurring in 5kb TSS distal regions. G: Aggregate CAGE profiles centered and oriented relative to ELK4\_1 footprint motifs.}
        \label{fig:c5_f2}
\end{figure}


We next explored the characteristics of TCs that occurred in the two main regulatory classes - promoter and enhancers, relative to each other. We focussed on transcribed, accessible regions in promoter and in enhancer states (TCs overlapping ATAC-seq peaks within promoter or enhancer segments). We considered the proximity of these elements to know gene TSSs and further classified the segments as TSS proximal or distal using a 5kb distance threshold from the nearest protein coding genes (Gencode V19). We then interrogated the chromatin landscape at these regions across 98 Roadmap Epigenomics cell types for which chromatin state annotations are publicly available (18 state ‘extended model’, see methods) \cite{theroadmapepigenomicsconsortiumIntegrativeAnalysis1112015}. We observed that TSS proximal islet TCs in accessible islet TSS chromatin states were nearly ubiquitously identified as TSS chromatin states across roadmap cell types (Fig. \ref{fig:c5_f2}D, left). A fraction of TSS distal islet TCs in accessible islet TSS chromatin states however were more specific for pancreatic islets (Fig. \ref{fig:c5_f2}D, right). In contrast, we observed that islet TCs in accessible islet enhancer chromatin states, both proximal and distal to known gene TSSs were more specifically identified as enhancer states in pancreatic islets (Fig. \ref{fig:c5_f2}E). This pattern was more clear for pancreatic islets than whole pancreas (Fig. \ref{fig:c5_f2}D and E, labelled) which further emphasizes the differences between epigenomic profiles for islets vs pancreas tissue. 


Having observed differences in cell-type specificities in islet TCs in TSS vs enhancer states, we next asked if transcription factors displayed preferences to bind in these regions. We observed that footprint motifs for regulatory factor X (RFX) TF family were highly enriched ($>$3 fold, P value = 0.0001) in TCs in accessible enhancers. On the other hand, TCs in accessible TSS regions were highly enriched to overlap footprint motifs of the E26 transformation-specific (ETS) TF family. 


We observe divergent aggregate CAGE profiles over TF footprint motifs enriched in enhancers for example RFX5\_known8 footprint motifs in 5kb TSS distal regions and ELK4\_1 motif (Fig. \ref{fig:c5_f2}G and H).


\subsection{Experimental validation of transcribed regions}
We next sought to experimentally validate the regulatory activity of islet TCs. We utilized the massively parallel reporter assay platform wherein thousands of elements can be simultaneously tested by including unique barcode sequences for each element and determining the transcriptional regulatory activity using sequencing based barcode quantification. This approach is also known as the self-transcribing active regulatory region sequencing (STARR-seq) assay. We generated libraries of TC sequences (198 bp elements) and cloned these along with unique 16 bp barcode sequences into the STARR-seq vector, downstream of the GFP gene which was in control with the SCP1 promoter. We transfected the STARR-seq libraries into rat beta cell insulinoma (INS1 832/13) cell line, extracted RNA and sequenced the barcodes. We added 8 bp unique molecular identifier (UMI) sequences before the PCR amplification of the RNA libraries to enable accounting for and removing PCR duplicates while quantifying true biological RNA copies. We then modeled the RNA and DNA barcode counts in generalized linear models (GLMs) to model RNA and DNA counts for each barcode using MPRAnalyze \cite{ashuachMPRAnalyzeStatisticalFramework2019} to quantify transcriptional activity of the TC element inserts. Our STARR-seq library included 6,798 insert elements (198 bp each) that overlapped 5,898 TCs. We selected barcodes that each had at least 10 DNA counts and non zero RNA counts in at least one technical replicate, and selected insert elements that had at least two of such qualifying barcodes. We had 3,240 such insert elements which we then tested for significant activity in the STARR-seq assay. We observed that 68.1\% (N = 2,260) of these elements showed significant regulatory activity (5\% FDR) (Fig. \ref{fig:c5_f3}A (top)). On classifying insert elements based on active TSS, enhancer or other chromatin state overlap in islets, we observe that a larger fraction of TC elements overlapping the active TSS state had significant transcriptional activity compared to elements overlapping enhancer states, which was in turn higher than TCs in other chromatin states (Fig. \ref{fig:c5_f3}A (bottom)). We also observed that the STARR-seq activity Z scores for TC elements in active TSS states were higher than those in enhancer states (Wilcoxon rank sum test P = 2.99$\times$10\textsuperscript{-9}) (Fig. \ref{fig:c5_f3}B). Z scores for TCs that overlapped ATAC-seq peaks were also significantly higher than those that did not occur in peaks (Wilcoxon rank sum test P = 4.01$\times$10\textsuperscript{-15}) (Fig. \ref{fig:c5_f3}C). Also, Z scores for TCs proximal to gene TSSs were higher than TCs that were distal to gene TSS locations (Wilcoxon rank sum test P = 4.21$\times$10\textsuperscript{-9}) (Fig. \ref{fig:c5_f3}D). In Fig. \ref{fig:c5_f3}E, we highlight an islet TC for which we tested three insert elements (Fig. \ref{fig:c5_f3}E, STARR-seq elements track), which occurred in active TSS and enhancer states and overlapped ATAC-seq peak. All three of the elements showed significant transcriptional activity in our assay (Z score $>$ 2.94, Z score P values $<$ 0.001). Interestingly, while there are no known gene TSS annotations in this region, clear islet polyA+ mRNA-seq profiles that overlap the CAGE signal can be observed here. Another example TC locus that occurred in the intronic region of the \textit{ABCC8} gene marked a region of islet-specific enhancer chromatin state and overlapped an ATAC-seq peak (Fig. \ref{fig:c5_f3}F). The regulatory activity of this region was previously validated in the pancreatic bud in mouse embryos from a LacZ assay \cite{parkerChromatinStretchEnhancer2013}. We included 39 insert elements that tiled this region (50 bp offset) (Fig. \ref{fig:c5_f3}F, ‘STARR-seq element’ track, which is a dense depiction of these tiles) in the STARR-seq assay and observed significant activity in multiple tiles within and neighboring the TC and the ATAC-seq peak (Fig. \ref{fig:c5_f3}F, ‘STARR-seq Z scores’ track). Through these analyses we could experimentally validate a considerable proportion of TCs for transcriptional regulatory activity in a STARR-seq assay in a rodent beta cell model system.   
   
\begin{figure}
        \centering
        \includegraphics[width=1\textwidth]{5_cage/figures/f3.png}
        \caption[Experimental validation of TCs using STARR-seq MPRA]{Experimental validation of TCs using STARR-seq MPRA: A: (Top) Number and fraction of TCs identified as significantly active (5\% FDR), nominally active (P $<$ 0.05) or non-significant in STARR-seq assay performed in rat beta cell insulinoma (INS1 832/13) cell line model. (Bottom) panel indicates the proportion of TCs that overlapped active TSS, enhancer or other chromatin states that were identified as active in STARR-seq assay. B. STARR-seq activity Z scores for TCs occurring in active TSS, enhancer or other chromatins states. C: STARR-seq activity Z scores for TCs that overlap ATAC-seq peak vs those that do not overlap peaks. D: STARR-seq activity Z scores for TCs based on relative position (5kb TSS proximal or distal) to known protein coding gene TSSs (Gencode V19). E: Example locus were TC elements that occur in active TSS and enhancer chromatin state and overlap ATAC-seq peak that were tested in the STARR-seq assay. The CAGE profile coincides with islet mRNA profile that is detected despite no known gene annotation in the region and the nearest protein coding gene is ~6kb away. F: The intronic locus of the \textit{ABCC8} gene, where an islet TC overlaps an ATAC-seq peak and active enhancer chromatin states. 198 bp tiles spanning the region shown in the ‘STARR-seq elements’ track were included in the assay.}
        \label{fig:c5_f3}
\end{figure}


\subsection{CAGE profiles augment functional genomic annotations to better understand GWAS and eQTL associations}
Observing characteristics of TCs in islets in different epigenomic contexts and validating the activities of these elements, we next asked if islet TCs taken as an additional layer of functional genomic information could supplement fine mapping efforts in understanding GWAS or eQTL associations. We classified genomic annotations utilizing different layers of epigenomic data such as histone modification based chromatin states, accessible regions within these states and transcribed accessible regions within these states. We then computed enrichment for T2D GWAS loci to overlap these annotations using full GWAS summary statistics \cite{mahajanFinemappingTypeDiabetes2018} using a Bayesian hierarchical model implemented in the fGWAS tool \cite{pickrellJointAnalysisFunctional2014}. This method allows using not only the genome wide significant loci, instead, leveraging genome wide association statistics such that marginal associations can also be accounted for. The prior probabilities of a region of the genome containing an association and a SNP being causal are then allowed to vary based on overlap with annotations. We observed that TCs in accessible enhancer regions were the most highly enriched for T2D GWAS loci (Fig. \ref{fig:c5_f4}A, left). We performed similar analysis using islet eQTL summary data \cite{varshneyGeneticRegulatorySignatures2017}, where we observed that TCs in accessible regions in both enhancers and promoters were most highly enriched (Fig. \ref{fig:c5_f4}A, right). These data suggest that including TC information with other functional genomics data help delineate more relevant regions for gene expression and trait association signals.     


\begin{figure}
            \centering
            \includegraphics[width=1\textwidth]{5_cage/figures/f4.png}
            \caption[Islet TCs along with ATAC-seq and chromatin state information supplement GWAS finemapping efforts]{Islet TCs along with ATAC-seq and chromatin state information supplement GWAS finemapping efforts: Enrichment of T2D GWAS (A) or islet eQTL (B) loci in annotations that comprise different levels of epigenomic information, including including chromatin state, ATAC-seq and TCs. Annotations were defined using combinations of these datasets such as accessible enhancers (ATAC-seq peaks in enhancer states) transcribed accessible enhancers (TCs that overlap ATAC-seq peaks in enhancer states) etc. Enrichment was calculated using fGWAS \cite{pickrellJointAnalysisFunctional2014} using summary statistics from GWAS (in A) \cite{mahajanFinemappingTypeDiabetes2018} or eQTL (in B) \cite{varshneyGeneticRegulatorySignatures2017}. C: fGWAS conditional enrichment analysis testing the contribution of annotations such as islet TCs or ATAC-seq peaks after conditioning on histone-only based annotations such as promoter and enhancer chromatin states in islets. D: Maximum SNP PPA per FGlu GWAS locus when using a model including ATAC-seq (x axis) or TCs (y axis)  another annotation such a STARR-seq activity Z scores for TCs occurring in active TSS, enhancer or other chromatins states. Continued on the next page.}
            \label{fig:c5_f4}
\end{figure}


\addtocounter{figure}{-1}
\begin{figure} [t!]
  \caption[Figure \ref{fig:c5_f4} continued]{Continued - E: Genome browser view of the \textit{STARD10} gene locus where T2D and Fasting Glucose GWAS SNPs and eQTL SNPs for the \textit{STARD10} gene occur (left). \textit{STARD10} eQTL Lead and LD r\textsuperscript{2}$>$0.8 proxy SNPs are shown in the eQTL SNP track. Genome browser view on the right shows the region zooming in on the lead eQTL SNP rs11603334 and another SNP rs1552225 (LD  r\textsuperscript{2}=1 with the lead SNP) which overlaps an islet TC. Functional reweighting of Fasting Glucose GWAS data using chromatin state, ATAC-seq and TC data resulted in the PPA of the SNP rs1552225 = 0.772.}
\end{figure}


We also asked if TCs or ATAC-seq data can be more informative in pinpointing active elements within enhancers or promoters. We performed GWAS and islet eQTL enrichment analyses for TCs and ATAC-seq peaks while conditioning on active TSS or active enhancer chromatin states. We observed that TCs had a higher conditional enrichment over enhancer states for T2D (\ref{fig:c5_f4}B) and ATAC-seq peaks. TCs also had a higher conditional enrichment over enhancer and promoter states for islet eQTL loci as compared to ATAC-seq peaks  (\ref{fig:c5_f4}B). We then sought to reweight the GWAS posterior probabilities of association (PPAs) by including these functional annotations in order to fine-map Fasting glucose GWAS loci and compared the maximal SNP PPA at each locus (Fig. \ref{fig:c5_f4}C). We highlight one such region within the \textit{ARAP1} gene that includes many variants in high LD. Variants at this T2D and FGlu GWAS locus are identified as eQTL for the \textit{STARD10} gene \cite{voightTwelveTypeDiabetes2010} but not for ARAP1. The GWAS and eQTL index SNP rs11603334 is in high LD (r\textsuperscript{2}=1) with rs1552224. Including TC information results in increased PPA of rs1552224 to 0.772. Without TC data the PPA for both rs11603334 and rs1552224 and were 0.446. We observed significant activity of the TC element that overlaps rs1552224 in our STARR-seq assay (Z score = 4.90, Z score P value = 4.78$\times$10\textsuperscript{-7}). A previous study showed stronger evidence for rs11603334 to be the causal variant \cite{kulzerCommonFunctionalRegulatory2014}, whereas another study pointed towards another variant (rs140130268) as more likely causal \cite{carratDecreasedSTARD10Expression2017} which highlights the complexity at this locus. These analyses demonstrate the utility of transcription initiation information to demarcate active regulatory elements at higher genomic resolution.  


\section{Discussion}
We profiled transcription start sites in human pancreatic islets using CAGE. We observe high enrichment of CAGE TCs in TSS chromatin states and ATAC-seq peaks in islets, which expectedly reflects the chromatin landscape at regions where transcription initiation occurs. Comparison of islet CAGE TCs with those identified across multiple tissues revealed that 20\% of islet TCs were islet specific. Furthermore, comparing the chromatin states underlying these TCs across multiple cell types and tissues indicated that TCs that occur distal to known TSSs of protein coding genes comprised of more islet specific promoter and enhancer chromatin states. These analyses also highlighted the differences in TCs and their underlying chromatin contexts between islets and pancreas tissues, which further demonstrate the need for molecular profiling in the islet tissue to better understand islet mechanisms. Islet TCs were also enriched to overlap GWAS loci of fasting glucose and specifically the islet beta cell related components of T2D loci, while being depleted for the insulin resistance related components of T2D GWAS loci. These analyses demonstrate that islet TCs mark active, specific and relevant Islet regulatory elements. 

Our work revealed that transcribed and accessible enhancer regions were most enriched to overlap TF footprint motifs for the RFX family of TFs. We previously showed that RFX footprint motifs are confluently disrupted by T2D GWAS risk alleles \cite{varshneyGeneticRegulatorySignatures2017}, which are enriched to occur in islet specific enhancer regions. These observations together highlight the role of islet specific enhancer regions, and the potential of ATAC-seq and CAGE profiling to identify the active regulatory elements within these large enhancer elements at high genomic resolution.

Utilizing the STARR-seq enhancer MPRA approach, we observed that ~68\% of Islet TCs induce significant transcriptional activity which highlights how CAGE identifies active regulatory elements. A larger fraction of TCs that occurred in active TSS chromatin states were significantly active than those occurring in active enhancer or other chromatin states. The transcriptional activities of these active TSS state overlapping TCs were also higher than the latter. We note here that only a small fraction of TCs (0.4\%) were identified to overlap the active enhancer state. Studies have shown that gene distal transcripts are more unstable, which would therefore be difficult to profile from a total RNA sample. Of course, given the relative instability of enhancer RNAs some chromatin-defined sites may be active but fall below the limits of detection of CAGE. Therefore, it is understandable that islet CAGE profiling from total RNA samples would comprise more stable promoter-associated RNA transcripts and have lesser representation of weaker transcripts originating from enhancer regions. In our previous work \cite{varshneyCellSpecificityHuman2018}, we showed that genetic variants in more cell type-specific enhancer regions have lower effects on gene expression (measured as eQTL effect sizes) than the variants occurring in more ubiquitous promoter regions, in the un-stimulated or basal cell state. This is consistent with our observation of lower transcriptional activities and even low representation of transcription initiation identified in enhancer state regions.
   
To better understand the mechanisms underlying GWAS loci, we interrogated the potential of TC information in identifying the causal SNP(s) using functional fine mapping approach (fGWAS). While most islet TCs overlapped islet ATAC-seq peaks ($>$70\%), we observed that regions supported by TCs, ATAC-seq peaks and enhancer chromatin states (transcribed, accessible enhancer regions) were most enriched to overlap T2D GWAS loci. This enrichment was higher than in regions only informed by ATAC-seq peaks and enhancer chromatin states, indicating that the small set of TCs in enhancer regions actually delineate highly relevant elements. Our work demonstrates that transcription start site information profiled using CAGE in islets can be used in addition to other relevant epigenomic information such as histone mark informed chromatin states and chromatin accessibility in nominating relevant variants and biological mechanisms.


         
\section{Materials and Methods}


\subsection{Islet Procurement and Processing}
Islet samples from organ donors were received from the Integrated Islet Distribution Program, the National Disease Research Interchange (NDRI), and Prodo- Labs. Islets were shipped overnight from the distribution centers. On receipt, we prewarmed islets to 37 °C in shipping media for 1–2 h before harvest; ∼2,500–5,000 islet equivalents (IEQs) from each organ donor were harvested for RNA isolation. We transferred 500–1,000 IEQs to tissue culture-treated flasks and cultured them as in the work in \cite{gershengornEpithelialtoMesenchymalTransitionGenerates2004}.
\subsection{RNA isolation, CAGE-seq library preparation and sequencing}
Total RNA from 2000-3000 islet equivalents (IEQ) was extracted and purified using Trizol (Life Technologies). RNA quality was confirmed with Bioanalyzer 2100 (Agilent); samples with RNA integrity number (RIN) $>$ 6.5 were prepared for CAGE sequencing. 1ug Total RNA samples were sent to DNAFORM, Japan, where polyA negative selection and size selection ($<$1000bp) was performed. Stranded CAGE-sequencing libraries were generated for each islet sample using the () kit according to manufacturer’s protocol (Illumina). Each islet CAGE-seq library was barcoded, pooled into 24-sample batches, and sequenced over multiple lanes of HiSeq 2000 to obtain an average depth of 100 million 2 x 101 bp sequences. All procedures followed ethical guidelines at the National Institutes of Health (NIH).


\subsection{CAGE data mapping and processing}
Because read lengths differed across libraries, we trimmed all reads to 51 bp using fastx\_trimmer (FASTX Toolkit v. 0.0.14). Adapters and technical sequences were trimmed using trimmomatic (v. 0.38; paired-end mode, with options ILLUMINACLIP:adapters.fa:1:30:7:1:true). To remove potential E. coli contamination, we mapped to the E. coli chromosome (genome assembly GCA\_000005845.2) with bwa mem (v. 0.7.15; options: -M). We then removed read pairs that mapped in a proper pair (with mapq $\geq$ 10) to E. coli. We mapped the remaining reads to hg19 using STAR (v. 2.5.4b; default parameters). We pruned the mapped reads to high quality autosomal read pairs (using samtools view v. 1.3.1; options -f 3 -F 4 -F 8 -F 256 -F 2048 -q 255). We then performed UMI-based deduplication using umitools dedup (v. 0.5.5; --method directional).


We selected Islet samples with strandedness measures $>$0.85 calculated from QoRTS \cite{hartleyQoRTsComprehensiveToolset2015a} for all downstream analyses. 50 Islet samples passed this threshold.


\subsection{Tag cluster calling} 
We used the paralu method to identify clusters of CAGE start sites (CAGE tag clusters) \cite{frithCodeTranscriptionInitiation2008}. The algorithm uses a density parameter d and identifies segments that maximize the value of \( Number\:of\:events - d * size\:of\:the\:segment\:(bp) \). Here, large values of d would favor small, dense clusters and small values of d would favor larger more rarefied clusters. The method identifies segments over all values of d beginning at the largest scale, where d = 0, where all of the events are merged into one big cluster. It then calculates the density (events per nucleotide) of every prefix and suffix of the big cluster. The lowest value among all of these densities is the maximum value of d for the big cluster because at higher values of d the big cluster will no longer be a maximal-scoring segment (because zero-scoring prefixes or suffixes are not allowed). 


We called TCs in each individual sample using raw tag counts, requiring at least 2 tags at each included start site and allowing single base-pair tag clusters (‘singletons’) if supported by $>$2 tags. We then merged the tag clusters on each strand across samples. For each resulting segment, we calculated the number of islet samples in which TCs overlapped the segment. We included the segment in the consensus TCs set if it was supported by independent TCs in at least 10 individual islet samples. This threshold was selected based on comparing the number of tag clusters with the number of samples across which support was required to consider the segment (Fig. \ref{fig:c5_sf_islet_threshold}). 
  
\subsection{Chromatin state analysis}
We collected publicly available cell/tissue ChIP-seq data for H3K27ac, H3K27me3, H3K36me3, H3K4me1, and H3K4me3 and input for Islets, Adipose and Skeletal Muscle and Liver. Data for Adipose, Skeletal Muscle and Liver tissues were included in the joint model for other ongoing projects. We performed read mapping and integrative chromatin-state analyses in a manner similar to that of our previous reports  and followed quality control procedures reported by the Roadmap Epigenomics Study \cite{theroadmapepigenomicsconsortiumIntegrativeAnalysis1112015}. Briefly, we trimmed reads across datasets to 36bp and overrepresented adapter sequences as shown by FASTQC (version v0.11.5, \url{https://www.bioinformatics.babraham.ac.uk/projects/fastqc/}) using cutadapt (version 1.12) \cite{martinCutadaptRemovesAdapter2011}. We then mapped reads using BWA (version 0.5.8c), removed duplicates using samtools \cite{liSequenceAlignmentMap2009}, and filtered for mapping quality score of at least 30. To assess the quality of each dataset, we performed strand cross-correlation analysis using phantompeakqualtools (v2.0; \url{code.google.com/p/phantompeakqualtools}) \cite{landtChIPseqGuidelinesPractices2012}. We converted bam files for each dataset to bed using the bamToBed tool. To more uniformly represent datasets with different sequencing depths across histone marks and tissues, we randomly subsampled each dataset bed file to the mean depth for that mark across the four included tissues. This allowed comparable chromatin state territories across tissues and ensured that chromatin state territories were not heavily driven by high sequencing depth. Chromatin states were learned jointly for the three cell types using the ChromHMM (version 1.10) hidden Markov model algorithm at 200-bp resolution to five chromatin marks and input \cite{ernstDiscoveryCharacterizationChromatin2010, ernstMappingAnalysisChromatin2011, ernstChromHMMAutomatingChromatin2012}. We ran ChromHMM with a range of possible states and selected a 11-state model, because it most accurately captured information from higher-state models and provided sufficient resolution to identify biologically meaningful patterns in a reproducible way. We have used this state selection procedure in previous analyses \cite{scottGeneticRegulatorySignature2016, varshneyGeneticRegulatorySignatures2017}. To assign biological function names to our states that are consistent with previously published states, we performed enrichment analyses in ChromHMM comparing our states with the states reported previously \cite{varshneyGeneticRegulatorySignatures2017} for the four matched tissues. We assigned the name of the state that was most strongly enriched in each of our states (Fig. \ref{fig:c5_sf_chromstate}).                 
                
\subsection{ATAC-seq data analysis}
We used previously published data for chromatin accessibility profiled using ATAC-seq in islets from two human organ donor samples \cite{varshneyGeneticRegulatorySignatures2017}. For each sample, we trimmed reads to 36 bp (to uniformly process ATAC-seq from other tissues for ongoing projects) and removed adapter sequences using Cutadapt (version 1.12) \cite{martinCutadaptRemovesAdapter2011}, mapped to hg19 used bwa-mem (version 0.7.15-r1140) \cite{liAligningSequenceReads2013}, removed duplicates using Picard (\url{http://broadinstitute.github.io/picard}) and filtered out regions blacklisted by the ENCODE consortium due to poor mappability (wgEncodeDacMapabilityConsensusExcludable.bed and wgEncodeDukeMapabilityRegionsExcludable.bed). For each tissue we subsampled both samples to the same depth so that each tissue had overall similar genomic region called as peaks. We used MACS2 (\url{https://github.com/taoliu/MACS}), version 2.1.0, with flags “-g hs–nomodel–shift -100–extsize 200 -B–broad–keep-dup all,” to call peaks and retained all broad-peaks that satisfied a 1\% FDR.
        
\subsection{Overlap enrichment between TCs and annotations}
Enrichment for overlap between each pair of regulatory annotations in Figure S1 was calculated using the Genomic Association Tester (GAT) tool \cite{hegerGATSimulationFramework2013}. To ask if two sets of regulatory annotations overlap more than that expected by chance, GAT randomly samples segments of one regulatory annotation set from the genomic workspace (hg19 chromosomes) and computes the expected overlaps with the second regulatory annotation set. We used 10,000 GAT samplings for each enrichment run. GAT outputs the observed overlap between segments and annotation along with the expected overlap and an empirical p-value.


\subsection{GWAS data collection and LD pruning} 
We downloaded the GWAS data for various traits from the NHGRI website on June 12, 2018 (file gwas\_catalog\_v1.0.2-associations\_e92\_r2018-05-29.tsv from \url{https://www.ebi.ac.uk/gwas/docs/file-downloads}). We selected genome-wide significant GWAS SNPs (P $<$5$\times$10\textsuperscript{-8}) for traits for which the study included European samples. To retain independent signals, we linkage diequilibrium (LD) pruned the list of SNPs to retain SNPs with the most significant P values that had LD r\textsuperscript{2} $<$0.2 between each pair. This procedure was performed using the PLINK (v1.9) tool \cite{purcellPLINKToolSet2007, changSecondgenerationPLINKRising2015}  –clump option and 1000 genomes phase 3 vcf files (downloaded from ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502 ), subsetted to the European samples as reference. We selected traits that had $>$30 independent signals for following analyses.        


\subsection{Enrichment of genetic variants in genomic features}
Enrichment for genome wide association study (GWAS) variants for different traits in Islet TCs was calculated using the Genomic Regulatory Elements and Gwas Overlap algoRithm (GREGOR) tool (version 1.2.1) \cite{schmidtGREGOREvaluatingGlobal2015}. Since the causal SNP(s) for the traits are not known, GREGOR allows considering the input lead SNP along with SNPs in high linkage disequilibrium (LD) (based on the provided R2THRESHOLD parameter) while computing overlaps with genomic features (such as islet TCs). Therefore, as input to GREGOR, we supplied SNPs that were not in high linkage disequilibrium with each other. For each input SNP, GREGOR selects ~500 control SNPs that match the input SNP for minor allele frequency (MAF), distance to the nearest gene, and number of SNPs in LD. Fold enrichment is calculated as the number of loci at which an input SNP (either lead SNP or SNP in high LD) overlaps the feature over the mean number of loci at which the matched control SNPs (or SNPs in high LD) overlap the same features. This process accounts for the length of the features, as longer features will have more overlap by chance with control SNP sets.                 


Specific parameters for the GWAS enrichment were: GREGOR: r2 threshold = 0.8. LD window size = 1Mb; minimum neighbor number = 500, population = European.
         
\subsection{Comparison of features with Roadmap chromatin states}
We downloaded the chromatin state annotations identified in 127 human cell types and tissues by the Roadmap epigenomics project \cite{theroadmapepigenomicsconsortiumIntegrativeAnalysis1112015} after integrating ChIP-seq data for five histone 3 lysine modifications (H3K4me3, H3K4me1, H3K36me3, H3K9me3 and H3K27me3) that are associated with promoter, enhancer, transcribed and repressed activities, across each cell type. For each TC feature, for example, TCs in ATAC-seq peaks within islet enhancer chromatin states, we identified segments occurring proximal to (within 5kb) and distal from (further than 5kb) known protein coding gene TSS (gencode V19 \cite{harrowGENCODEReferenceHuman2012}). For each such segment, we identified the maximally overlapping chromatin state across 98 cell types publicly available from the Roadmap Epigenomics project in their 18 state ‘extended’ model using BEDtools intersect. We then ordered the segments using clustering (hclust function in R) based on the gower distance metric (daisy function in R) for the roadmap state assignments across 127 cell types.


\subsection{Aggregate signal}
We generated the ATAC-seq density plot over islet TC midpoints using the Agplus tool (version 1.0) \cite{maeharaAgplusRapidFlexible2015}. We used the ATAC-seq signal track for reads per 10 Million to aggregate over stranded TCs. 


To obtain CAGE tracks, we merged CAGE bam files for islet samples that passed QC (see CAGE data processing section) and obtained the read 1 start sites or TSSs. To better visualise the CAGE signal, we then flanked each TSS 10bp upstream and downstream and normalized the TSS counts to 10M mapped reads.   We generated CAGE density plots over ATAC-seq narrow peak summits by using the agplus tool.  


To obtain aggregate CAGE signal over TF footprint motifs, we oriented the CAGE signal with respect to the footprint taken on the plus strand. We used HTSeq GenomicPosition method \cite{andersHTSeqPythonFramework2015} to obtain the sum of CAGE signal at each base pair relative to the footprint motif mid point.


\subsection{fGWAS analyses and finemapping}        
fGWAS (version 0.3.6) \cite{pickrellJointAnalysisFunctional2014} employs a Bayesian hierarchical model to determine shared properties of loci affecting a trait. The model uses association summary level data, divides the genome into windows generally larger than the expected LD patterns in the population. The model estimates the probabilities that an association lies in a window and that a SNP is causal. These probabilities are then allowed to depend on genomic annotations, and are estimated based on enrichment patterns of annotations across the genome using a Bayes approach. We used fGWAS with default parameters for enrichment analyses for individual annotations in Fig. \ref{fig:c5_f4} A and B. For each individual annotation, the model provided maximum likelihood enrichment parameters and annotations were considered as significantly enriched if the parameter estimate and 95\% CI was above zero. 


We performed conditional analyses using the ‘-cond’ option. 


To reweight GWAS summary data based on functional annotation overlap, we used the ‘-print’ option in an fGWAS model run after including multiple annotations that were individually significantly enriched. We included Active TSS, active enhancer, stretch enhancer, quiescent and polycomb repressed annotations along with ATAC-seq or TCs in a model to derive enrichment priors which can then be used to evaluate both the significance and functional impact of associated variants in GWAS regions; such that variants overlapping more enriched annotations carry extra weight.

\section{Acknowledgements}
I'd like to thank several people who contributed to this project. I thank Dr. Mike Erdos, Narisu Narisu Dr. Nandini Manickam for preparing the RNA samples, coordinating shipping to DNAFORM, Japan for the CAGE library preparation and subsequent sequencing performed at the National Institutes of Health (NIH), USA. I especially thank Dr. Yasuhiro Kyono, Dr. Venkateswaran Ramamoorthy Elangovan and Prof. Jacob Kitzman for all their help in designing, performing and the subsequent analysis of the STARR-seq assay. I thank Prof. Steve Parker conceptualizing and organizing this project and for his insight and support all throughout.